<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">


<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    
    <title>** &mdash; tranSMART User Guide 0.1 documentation</title>
    
    <link rel="stylesheet" href="_static/alabaster.css" type="text/css" />
    <link rel="stylesheet" href="_static/pygments.css" type="text/css" />
    
    <script type="text/javascript">
      var DOCUMENTATION_OPTIONS = {
        URL_ROOT:    './',
        VERSION:     '0.1',
        COLLAPSE_INDEX: false,
        FILE_SUFFIX: '.html',
        HAS_SOURCE:  true
      };
    </script>
    <script type="text/javascript" src="_static/jquery.js"></script>
    <script type="text/javascript" src="_static/underscore.js"></script>
    <script type="text/javascript" src="_static/doctools.js"></script>
    <script type="text/javascript" src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
    <link rel="top" title="tranSMART User Guide 0.1 documentation" href="index.html" />
    <link rel="prev" title="Welcome to tranSMART User Guide’s documentation!" href="index.html" />
   
  <link rel="stylesheet" href="_static/custom.css" type="text/css" />
  
  <meta name="viewport" content="width=device-width, initial-scale=0.9, maximum-scale=0.9" />

  </head>
  <body role="document">
  

    <div class="document">
      <div class="documentwrapper">
        <div class="bodywrapper">
          <div class="body" role="main">
            
  <p><a class="reference internal" href="_images/image2.png"><img alt="image0" src="_images/image2.png" style="width: 8.70972in; height: 11.73958in;" /></a></p>
<blockquote>
<div><p><a class="reference internal" href="_images/image3.png"><img alt="image1" src="_images/image3.png" style="width: 1.50000in; height: 0.52000in;" /></a></p>
<p>This version 1.1 of the tranSMART documentation was created by
tranSMART Foundation and is made available under a <a class="reference external" href="http://creativecommons.org/licenses/by/3.0/">Creative Commons
Attribution 3.0 Unported
License.</a> Earlier
versions of this tranSMART documentation were created by Johnson &amp;
Johnson and Recombinant Data and used under a <a class="reference external" href="http://creativecommons.org/licenses/by/3.0/">Creative Commons
Attribution 3.0 Unported
License.</a></p>
<p>Recombinant® is a registered trademark of Recombinant Data Corp. in
the United States and other countries.</p>
<p>Other company, product, and service names referenced in the
documentation may be trademarks or service marks of others.</p>
<p>This document is licensed under the Creative Commons Attribution 3.0
Unported License.</p>
<p><strong>You are free:</strong></p>
<blockquote>
<div><p><strong>to share</strong> – to copy, distribute and transmit the work</p>
<p><strong>to remix</strong> – to adapt the work</p>
</div></blockquote>
<p>to make commercial use of the work</p>
<p><strong>Under the following conditions:</strong></p>
<p><strong>Attribution</strong> – You must attribute your use of the work in the
following manner:</p>
<blockquote>
<div><p>“This version 1.1 of the tranSMART documentation was created by
tranSMART Foundation and is made available under a <a class="reference external" href="http://creativecommons.org/licenses/by/3.0/">Creative
Commons Attribution 3.0 Unported
License.</a>
Earlier versions of this tranSMART documentation were created by
Johnson &amp; Johnson and Recombinant Data and used under a
<a class="reference external" href="http://creativecommons.org/licenses/by/3.0/">Creative Commons Attribution 3.0 Unported
License.</a> The
URL for the work is
<a class="reference external" href="http://transmartfoundation.org/site/OpenSource_tranSMART_User_Guide_v11.pdf">http://transmartfoundation.org/site/OpenSource_tranSMART_User_Guide_v11.pdf</a>.</p>
<p>Recombinant® is a registered trademark of Recombinant Data Corp.
in the United States and other countries.</p>
<p>Other company, product, and service names referenced in the
documentation may be trademarks or service marks of others.”</p>
</div></blockquote>
</div></blockquote>
<div class="section" id="id3">
<h1><a href="#id1"><span class="problematic" id="id2">**</span></a><a class="headerlink" href="#id3" title="Permalink to this headline">¶</a></h1>
<p>Chapter&nbsp;1</p>
<div class="section" id="chapter-1-getting-started-with-transmart">
<h2>Chapter 1: Getting Started with tranSMART<a class="headerlink" href="#chapter-1-getting-started-with-transmart" title="Permalink to this headline">¶</a></h2>
<p>The tranSMART application reflects the efforts of various informatics
groups to integrate data from internal and external data sources within
a single data warehouse, and to provide scientific end users the tools
to search for, view, and analyze the data in the warehouse.</p>
<p>The core internal data is a historical base of biomarker data from gene
expression, RBM, and SNP experiments, including both raw and analyzed
data.</p>
<p>External data sources include publicly available resources such as the
Gene Expression Omnibus repository and MeSH Ontology.</p>
<p>The tranSMART application presents scientists with a search tool to
query this vast ocean of disparate data through a Google-like user
interface.</p>
<p>As users become more sophisticated in developing patterns of search
terms, they can save and share their efforts with fellow researchers. A
second tool, called the Dataset Explorer, allows the properly authorized
user to create and study cohorts of patients that have been involved in
completed clinical research efforts. Dataset Explorer also provides the
user with tools to compare an individual (or group) in one study against
a person or cohort in another study.</p>
<table border="1" class="docutils">
<colgroup>
<col width="9%" />
<col width="91%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image2" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">There may be some minor differences between the UI objects illustrated in this guide and the ones you see on your screen.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<div class="section" id="feature-overview">
<h3>Feature Overview<a class="headerlink" href="#feature-overview" title="Permalink to this headline">¶</a></h3>
<p>tranSMART contains the following major features:</p>
<ul class="simple">
<li>Search tool</li>
<li>Dataset Explorer</li>
<li>Gene Signature Wizard</li>
</ul>
<div class="section" id="search-tool">
<h4>Search Tool<a class="headerlink" href="#search-tool" title="Permalink to this headline">¶</a></h4>
<p>tranSMART provides a Google-like search tool that lets you search across
multiple data sources for information related to items of interest, such
as biomarkers, diseases, genes, and gene signatures.</p>
<p>The scope of a search can include clinical studies, externally conducted
experiments, and in vivo/in vitro studies.</p>
<p>Search tool functionality includes:</p>
<ul class="simple">
<li>Searching within a particular category, such as diseases, genes, or
pathways, or searching across all categories.</li>
<li>Building complex search criteria that let you precisely define what
to search for.</li>
<li>Saving search criteria for easy recall and re-execution.</li>
<li>Emailing search criteria to colleagues.</li>
</ul>
<div class="section" id="search-results">
<h5>Search Results<a class="headerlink" href="#search-results" title="Permalink to this headline">¶</a></h5>
<p>In searches of experiments, tranSMART displays complete listings of all
analyses related to the experiments that are found.</p>
<p>tranSMART flags “meaningful” results in the analysis lists. Meaningful
analyses are those where the signature genes are differentially
modulated in a statistically significant way, indicating that the
associated target is probably affected by the treatment, disease or
other topic examined in the experiment.</p>
<p>Search result functionality includes:</p>
<ul class="simple">
<li>Displaying details of a particular experiment by clicking the name of
the trial or experiment in the results list.</li>
<li>Accessing a number of gene-related sites – Entrez Gene, Entrez
Global, GeneCards, and Google Scholar – by clicking the name of a
gene in the results list.</li>
<li>Viewing the technical report or protocol used for an analysis.</li>
<li>Exporting the complete results list to a Microsoft Excel file.</li>
<li>Exporting details of a particular study, experiment, or other result
to a Microsoft Excel file.</li>
</ul>
</div>
</div>
<div class="section" id="dataset-explorer">
<h4>Dataset Explorer<a class="headerlink" href="#dataset-explorer" title="Permalink to this headline">¶</a></h4>
<p>The Dataset Explorer, originally based on an i2b2 design, lets you
compare two sets of study groups based on one or more points of
comparison. You define both the criteria that populate the study groups
and the points of comparison between the study groups.</p>
<p>Dataset Explorer uses a standard navigation tree interface to display
data from clinical trials, and also employs an intuitive drag-and-drop
functionality to help you build the criteria for populating the study
groups and to add the points of comparison.</p>
<p>Dataset Explorer functionality includes:</p>
<ul class="simple">
<li>Saving the criteria used to populate the study groups.</li>
<li>Emailing the study group criteria to colleagues.</li>
<li>Using a heatmap to visualize the change in the expression of a
specific protein from one sample to another.</li>
<li>Using principal component analysis (PCA) to reduce the dimensionality
of the dataset and to identify new, meaningful variables in the
dataset.</li>
<li>Performing advanced analyses and displaying results in various
formats (scatter plot with linear regression, box plot with analysis
of variance, etc.)</li>
<li>Exporting a study or subset of a study to analyze in an external
tool.</li>
</ul>
</div>
<div class="section" id="gene-signature-wizard">
<h4>Gene Signature Wizard<a class="headerlink" href="#gene-signature-wizard" title="Permalink to this headline">¶</a></h4>
<p>tranSMART provides a wizard to help you create and define gene
signatures and gene lists.</p>
<p>You can use your gene signature or gene list in tranSMART searches to
find studies where the differentially regulated genes match those in
your gene signature or list. This can help you develop hypotheses about
diseases or treatments that may have similar genes deregulated.</p>
<p>Stored gene signatures can also be used in the analyses functionality of
Dataset Explorer.</p>
<p>Gene signature functionality includes:</p>
<ul class="simple">
<li>Keeping the gene signature or list private so that only you can
access it and use it in searches, or making it publicly available to
all tranSMART users.</li>
<li>Cloning an existing gene signature or list – either yours or a public
one – as the starting point for creating and defining a new gene
signature or list.</li>
<li>Exporting all details of a gene signature or list to a Microsoft
Excel file.</li>
</ul>
</div>
</div>
<div class="section" id="logging-in">
<h3>Logging In<a class="headerlink" href="#logging-in" title="Permalink to this headline">¶</a></h3>
<ol class="arabic simple">
<li>To log into tranSMART:<ol class="arabic">
<li>Type the address of the tranSMART software into your browser’s URL
field:</li>
</ol>
</li>
</ol>
<p><em>http://transmart.host.com/transmart</em></p>
<p>The login screen appears:</p>
<p><a class="reference internal" href="_images/image5.png"><img alt="image3" src="_images/image5.png" style="width: 3.09872in; height: 1.12987in;" /></a></p>
<ol class="arabic simple">
<li>Type your tranSMART login credentials, then click <strong>Login</strong>.</li>
<li><a class="reference internal" href="_images/image4.png"><img alt="image4" src="_images/image4.png" style="width: 0.37282in; height: 0.33333in;" /></a>Your tranSMART software may also be configured to login
automatically as a guest user.</li>
</ol>
</div>
<div class="section" id="tools">
<h3>Tools<a class="headerlink" href="#tools" title="Permalink to this headline">¶</a></h3>
<p>tranSMART provides the following tools:</p>
<ul class="simple">
<li><strong>Search</strong> – Search across internal and external data sources for
research data and literature related to search terms that you
provide.</li>
<li><strong>Dataset</strong> <strong>Explorer</strong> – View study data for subjects that you
select, based on criteria that you specify. Also, compare data
generated for subjects in two different study groups, based on
criteria and points of comparison that you specify.</li>
<li><strong>Gene</strong> <strong>Signature/Lists</strong> – View definitions of existing gene
signatures and add new gene signature definitions.</li>
<li><strong>Utilities</strong> – contains the following submenus:<ul>
<li><strong>Help</strong> – Display links to the tranSMART documentation set.</li>
<li><strong>Contact Us –</strong> Email questions, problem reports, enhancement
requests, or any other feedback about the tranSMART application.</li>
<li><strong>About</strong> – Displays the version of tranSMART.</li>
</ul>
</li>
</ul>
<p>Select the tranSMART tool to use by clicking one of the tool tabs at the
top of the tranSMART window:</p>
<p><img alt="image5" src="_images/image6.png" /></p>
</div>
<div class="section" id="opening-a-particular-tool-at-login">
<h3>Opening a Particular Tool at Login<a class="headerlink" href="#opening-a-particular-tool-at-login" title="Permalink to this headline">¶</a></h3>
<p>By default, tranSMART opens the Search tool after you log in. However,
you can specify the tool for tranSMART to open immediately after login
by including the tool name in the address you type into your browser’s
URL field.</p>
<p>To automatically open a particular tranSMART tool immediately after
login, use an address listed below:</p>
<table border="1" class="docutils">
<colgroup>
<col width="22%" />
<col width="78%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image6" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">The addresses below are case-sensitive.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<ul class="simple">
<li>Search tool – either of the following:</li>
</ul>
<p><a class="reference external" href="https://transmart.host.com/transmart">https://transmart.host.com/transmart</a></p>
<p><a class="reference external" href="https://transmart.host.com/transmart/search">https://transmart.host.com/transmart/search</a></p>
<ul class="simple">
<li>Dataset Explorer tool</li>
</ul>
<p><a class="reference external" href="https://transmart.host.com/transmart/datasetExplorer">https://transmart.host.com/transmart/datasetExplorer</a></p>
<ul class="simple">
<li>Gene Signature/Lists tool</li>
</ul>
<p><a class="reference external" href="https://transmart.host.com/transmart/geneSignature">https://transmart.host.com/transmart/geneSignature</a></p>
<p>Chapter&nbsp;2</p>
</div>
</div>
<div class="section" id="chapter-2-search-tool">
<h2>Chapter 2: Search Tool<a class="headerlink" href="#chapter-2-search-tool" title="Permalink to this headline">¶</a></h2>
<p>tranSMART provides a Google-like interface for searching across internal
data sources as well as external data sources with a single query, based
on one or more search filters that you define.</p>
<div class="section" id="search-tasks">
<h3>Search Tasks<a class="headerlink" href="#search-tasks" title="Permalink to this headline">¶</a></h3>
<p>A search filter is the name of a biomedical concept such as a gene,
pathway, disease, or other item of medical interest. These filter names
are pre-defined in tranSMART. You can browse lists of these filter names
and select the filter you want, or type part of a filter name in the
<strong>Search</strong> field, causing tranSMART to display a list of filters that
begin with the text you type.</p>
<p>You can base your search on a single search filter or on a multi-filter
search string.</p>
<div class="section" id="defining-a-search-filter">
<h4>Defining a Search Filter<a class="headerlink" href="#defining-a-search-filter" title="Permalink to this headline">¶</a></h4>
<p>There are several ways to define a search filter:</p>
<ul class="simple">
<li>Type all or part of the filter name directly into the <strong>Search</strong>
field.</li>
<li>Browse all the pre-defined filters within filter categories (such as
diseases).</li>
<li>Use a saved search filter or search string.</li>
</ul>
<div class="section" id="type-the-filter-name">
<h5>Type the Filter Name<a class="headerlink" href="#type-the-filter-name" title="Permalink to this headline">¶</a></h5>
<ol class="arabic">
<li><p class="first">To search the internal and external data sources for information
related to a filter name:</p>
<ol class="arabic simple">
<li>Click the tab for the Search tool at the top of the tranSMART
window.</li>
</ol>
<ol class="arabic simple">
<li>Click the search filter category to search within (for example,
search only diseases, or search only genes).</li>
</ol>
</li>
</ol>
<p><img alt="image7" src="_images/image6.png" />The search engine first filters by the filter category you
select, and then filters by the name you type. To search across all
filter categories, click <strong>all</strong>:</p>
<table border="1" class="docutils">
<colgroup>
<col width="6%" />
<col width="94%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image8" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">You can only specify one search filter in the <strong>Search</strong> field shown above. For instructions on creating a multi-filter search string, see <em>Building a Search String</em> on page 11.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<ol class="arabic simple">
<li>Type part or all of the filter name into the <strong>Search</strong> field.</li>
</ol>
<p>Up to 20 matches that begin with the text you type are displayed in a
dropdown list below the <strong>Search</strong> field. For example, the following
list appears for the search filter <strong>bra</strong> when searching across all
filter categories:</p>
<p><a class="reference internal" href="_images/image7.png"><img alt="image9" src="_images/image7.png" style="width: 4.53646in; height: 2.39727in;" /></a></p>
<table border="1" class="docutils">
<colgroup>
<col width="9%" />
<col width="91%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image10" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">You can also search for aliases. For example, to find the gene PTK7, you can type part or all of the name PTK7 or its alias, CCK4.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<ol class="arabic simple">
<li>Do one of the following:</li>
</ol>
<ul class="simple">
<li>If the name of the filter you want appears in the list, click the
filter name. The search begins immediately.</li>
<li>If the filter name you want does not appear in the list, type a more
complete name in the <strong>Search</strong> field. For example, if you typed only
<strong>br</strong> in the <strong>Search</strong> field, no entries for “brain diseases”
appear in the list. Typing an <strong>a</strong> after the text you already typed
displays a list like the one shown above.</li>
<li>If no list appears after you type a complete filter name, click the
<strong>Search</strong> button.</li>
</ul>
<ol class="arabic">
<li><p class="first">To start another search using a new search filter, click <strong>clear
all</strong> above the search result:</p>
<blockquote>
<div><p><a class="reference internal" href="_images/image8.png"><img alt="image11" src="_images/image8.png" style="width: 4.09375in; height: 0.82292in;" /></a></p>
</div></blockquote>
</li>
</ol>
<p>Alternatively, you can click the tranSMART logo, or simply type a new
filter in the <strong>Search</strong> field.</p>
<p>See <a class="reference external" href="#Working_With_Search_Results">*Working with Search Results*</a> on
page 16 for information on viewing and refining search results.</p>
</div>
<div class="section" id="browse-for-a-filter-name">
<h5>Browse for a Filter Name<a class="headerlink" href="#browse-for-a-filter-name" title="Permalink to this headline">¶</a></h5>
<p>You can browse through all the pre-defined filters in each of the
following areas:</p>
<ul class="simple">
<li>Disease</li>
<li>Gene Signature/Lists</li>
<li>Geo/ebi</li>
<li>Pathway</li>
</ul>
<ol class="arabic">
<li><p class="first">To browse the pre-defined filters:</p>
<ol class="arabic simple">
<li>Click the tab for the Search tool at the top of the tranSMART
window.</li>
<li>Click the <strong>browse</strong> link to the right of the <strong>Search</strong> button. A
window similar to the following appears:</li>
</ol>
<blockquote>
<div><p><a class="reference internal" href="_images/image9.png"><img alt="image12" src="_images/image9.png" style="width: 3.72917in; height: 2.84530in;" /></a></p>
</div></blockquote>
</li>
</ol>
<table border="1" class="docutils">
<colgroup>
<col width="9%" />
<col width="91%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image13" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">The search engine ignores any filter category you may have selected and any filter text you may have entered in the <strong>Search</strong> field.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<ol class="arabic simple">
<li>Click the tab for the area in which you want to browse for filters.</li>
<li>To initiate a search for information related to a filter, click the
filter name or the green arrow (<img alt="image14" src="_images/image10.png" />) after the name.</li>
</ol>
<p>After you click a filter, the search begins immediately.</p>
<ol class="arabic simple">
<li>To browse for another filter, click <strong>browse</strong> again. There is no
need to clear the previous result.</li>
<li>To start another search using a new search filter, click <strong>clear
all</strong> above the search result:</li>
</ol>
<p><a class="reference internal" href="_images/image11.png"><img alt="image15" src="_images/image11.png" style="width: 4.09375in; height: 0.66667in;" /></a></p>
<p>Alternatively, you can click the tranSMART logo, or simply type a new
filter in the <strong>Search</strong> field.</p>
<p>See <a class="reference external" href="#Working_With_Search_Results">*Working with Search Results*</a> on
page <a class="reference external" href="#Working_With_Search_Results">16</a> for information on viewing
and refining search results.</p>
</div>
<div class="section" id="use-a-saved-search-filter">
<h5>Use a Saved Search Filter<a class="headerlink" href="#use-a-saved-search-filter" title="Permalink to this headline">¶</a></h5>
<p>There are two ways to access a saved search filter:</p>
<ul class="simple">
<li>Retrieve the saved filter from a list of filters that you created and
saved. The instructions in this section describe this method.</li>
<li>Click a link to a saved filter that someone else has created, saved,
and emailed to you.</li>
</ul>
<p>See <em>Saving a Search Filter or Search String</em> on page 14 for more
information, including instructions on saving search filters and search
strings.</p>
<ol class="arabic">
<li><p class="first">To search against a filter that you created and saved:</p>
<ol class="arabic simple">
<li>Click the tab for the Search tool at the top of the tranSMART
window.</li>
<li>Click the <strong>saved filters</strong> link to the right of the <strong>Search</strong>
button. A list of filters that you created and saved appears:</li>
</ol>
<blockquote>
<div><p><a class="reference internal" href="_images/image12.png"><img alt="image16" src="_images/image12.png" style="width: 6.00000in; height: 0.99487in;" /></a></p>
</div></blockquote>
</li>
</ol>
<ol class="arabic simple">
<li>To search against a saved filter in the list, click the <strong>select</strong>
link to the right of the saved filter name. The search begins
immediately.</li>
<li>To start another search using a new search filter, click <strong>clear</strong>
<strong>all</strong> above the search result.</li>
</ol>
<p>Alternatively, you can click the tranSMART logo, or simply type a new
filter in the <strong>Search</strong> field.</p>
<p>See <em>Working with Search Results</em> on page 16 for information on viewing
and refining search results.</p>
</div>
</div>
<div class="section" id="building-a-search-string">
<h4>Building a Search String<a class="headerlink" href="#building-a-search-string" title="Permalink to this headline">¶</a></h4>
<p>You can make the scope of your search more precise by building a
multi-filter search string. The filters in a search string are joined by
the logical operators AND and OR.</p>
<div class="section" id="rules-for-building-a-search-string">
<h5>Rules for Building a Search String<a class="headerlink" href="#rules-for-building-a-search-string" title="Permalink to this headline">¶</a></h5>
<p>The following rules apply to building a multi-filter search string:</p>
<ul class="simple">
<li>Filters within the same filter category (such as diseases or genes)
are joined by the logical operator OR.</li>
</ul>
<p>For example, if you add the filters Diseases&gt; Melanoma and Diseases&gt;
Melanoma, Experimental to a search string, the search engine evaluates
them as in the following expression:</p>
<blockquote>
<div>(Diseases&gt; Melanoma <strong>OR</strong> Diseases&gt; Melanoma, Experimental)</div></blockquote>
<ul class="simple">
<li>Filters within different filter categories are joined by the logical
operator AND.</li>
</ul>
<p>For example, if you add the filters Diseases&gt; Anemia, Diseases&gt; Anemia,
Hemolytic, and Gene&gt; HBB to a search string, the search engine evaluates
them as in the following expression:</p>
<blockquote>
<div>(Diseases&gt; Anemia <strong>OR</strong> Diseases&gt; Anemia, Hemolytic) <strong>AND</strong> Gene&gt;
HBB</div></blockquote>
<ul class="simple">
<li>Filters that are not among the pre-defined filters are assigned to
the filter category <strong>Text&gt;</strong>.</li>
</ul>
</div>
<div class="section" id="instructions-for-building-a-search-string">
<h5>Instructions for Building a Search String<a class="headerlink" href="#instructions-for-building-a-search-string" title="Permalink to this headline">¶</a></h5>
<ol class="arabic">
<li><p class="first">To build a multi-filter search string:</p>
<ol class="arabic simple">
<li>Define a search filter using any of the methods described in
<em>Defining a Search Filter</em> on page 7.</li>
<li>When the results window appears, click <strong>advanced</strong>:</li>
</ol>
<blockquote>
<div><p><a class="reference internal" href="_images/image13.png"><img alt="image17" src="_images/image13.png" style="width: 4.10365in; height: 0.73949in;" /></a></p>
</div></blockquote>
</li>
</ol>
<p>The Edit Filters dialog appears, displaying the filter you just created:</p>
<blockquote>
<div><a class="reference internal" href="_images/image14.png"><img alt="image18" src="_images/image14.png" style="width: 5.41667in; height: 2.27537in;" /></a></div></blockquote>
<ol class="arabic simple">
<li>To add another filter, type part or all of a filter name into the
<strong>Search</strong> field.</li>
</ol>
<p>Up to 20 matches for the text you type are displayed in a dropdown list
below the <strong>Search</strong> field. For example, the following list appears for
the search filter <strong>dis</strong>:</p>
<p><a class="reference internal" href="_images/image15.png"><img alt="image19" src="_images/image15.png" style="width: 5.32225in; height: 3.13503in;" /></a></p>
<p>Do one of the following:</p>
<ul class="simple">
<li>If the name of the filter you want appears in the list, click the
filter name. The tranSMART software inserts the filter into the
<strong>Filters Box</strong>.</li>
<li>If the filter you want does not appear in the list, type a more
complete name in the <strong>Search</strong> field.</li>
<li>If no list appears after you type a complete filter name, click the
plus-sign button ( <a class="reference internal" href="_images/image16.png"><img alt="image20" src="_images/image16.png" style="width: 0.16875in; height: 0.21563in;" /></a> ) to the right of the <strong>Search</strong> field.</li>
</ul>
<ol class="arabic">
<li><p class="first">Repeat the previous step for each new filter to add to the search
string.</p>
</li>
<li><p class="first">Optionally, to delete a filter from the search string, click the red
<strong>X</strong> (<img alt="image21" src="_images/image17.png" />) to the right of the filter name:</p>
<blockquote>
<div><p><a class="reference internal" href="_images/image18.png"><img alt="image22" src="_images/image18.png" style="width: 4.35362in; height: 2.41636in;" /></a></p>
</div></blockquote>
</li>
</ol>
<ol class="arabic simple">
<li>When finished defining the search string, click <strong>Apply</strong> to begin
the search.</li>
<li>When the results window appears, you can continue editing the search
string or save it, as follows:</li>
</ol>
<ul>
<li><p class="first">To continue editing the search string, click <strong>advanced</strong>.</p>
</li>
<li><p class="first">To save the search string, click <strong>save</strong>.</p>
<p><a class="reference internal" href="_images/image19.png"><img alt="image23" src="_images/image19.png" style="width: 4.90625in; height: 1.23866in;" /></a></p>
</li>
</ul>
<p>The search engine evaluates this search string as in the following
expression:</p>
<p>(Disease&gt; Brain Diseases <strong>OR</strong> Dementia)</p>
<p>See <em>Saving a Search Filter or Search String</em> on page 14 for more
information about saving search filters and search strings.</p>
</div>
</div>
<div class="section" id="saving-a-search-filter-or-search-string">
<h4>Saving a Search Filter or Search String<a class="headerlink" href="#saving-a-search-filter-or-search-string" title="Permalink to this headline">¶</a></h4>
<ol class="arabic simple">
<li>To save a search filter or search string:<ol class="arabic">
<li>After defining the search filter or search string, run the search
and click <strong>save</strong> in the results window.</li>
</ol>
</li>
</ol>
<p>The Create Filter window appears:</p>
<p><a class="reference internal" href="_images/image20.png"><img alt="image24" src="_images/image20.png" style="width: 4.89011in; height: 1.54679in;" /></a></p>
<ol class="arabic simple">
<li>In the <strong>Name</strong> field, type a name for the search filter or search
string.</li>
<li>Optionally, in the <strong>Description</strong> field, type a description of the
search filter or search string. In the saved filters list, the
description appears immediately below the name of the search filter
or search string.</li>
<li>Check the <strong>Private</strong> <strong>Flag</strong> checkbox to prevent others from using
this search filter or search string, or clear the checkbox to allow
others to use the search filter or search string.</li>
</ol>
<p>If a filter is public, a shortcut (link) to the filter is displayed in
the <strong>saved filters</strong> list, and an <strong>email</strong> link is provided, allowing
you to email the shortcut to others. If a filter is private, the saved
filter is marked “Private,” and the filter shortcut and <strong>email</strong> link
are not displayed.</p>
<table border="1" class="docutils">
<colgroup>
<col width="4%" />
<col width="96%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image25" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">Only the person who created and saved a search filter can see that filter in the saved filter list. To let a colleague use a search filter you saved, you must (1) mark the filter as Public, and (2) click the <strong>email</strong> link to send the shortcut for the search filter to the colleague.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<p>In the following <strong>Saved Filters</strong> list, the first two entries are
private and the third is public:</p>
<p><a class="reference internal" href="_images/image21.png"><img alt="image26" src="_images/image21.png" style="width: 4.85417in; height: 1.42407in;" /></a></p>
<ol class="arabic simple">
<li>When finished, click <strong>Create</strong> to save the new search filter or
search string, or click <strong>Cancel</strong> to abandon it.</li>
</ol>
<div class="section" id="editing-and-deleting-saved-filters">
<h5>Editing and Deleting Saved Filters<a class="headerlink" href="#editing-and-deleting-saved-filters" title="Permalink to this headline">¶</a></h5>
<ol class="arabic">
<li><p class="first">To edit a saved filter:</p>
<ol class="arabic simple">
<li>Click the tab for the Search tool at the top of the tranSMART
window.</li>
<li>Click the <strong>saved filters</strong> link to the right of the <strong>Search</strong>
button. A list of your saved search filters appears.</li>
<li>Click the <strong>edit</strong> link to the right of the saved filter name. The
Edit Filter window appears:</li>
</ol>
<blockquote>
<div><p><a class="reference internal" href="_images/image22.png"><img alt="image27" src="_images/image22.png" style="width: 5.68499in; height: 1.42500in;" /></a></p>
</div></blockquote>
</li>
</ol>
<ol class="arabic simple">
<li>Make one or more of the following changes:</li>
</ol>
<ul class="simple">
<li>In the <strong>Name</strong> field, modify the name of the saved filter.</li>
<li>In the <strong>Description</strong> field, add or modify an optional description
of the saved filter. In the <strong>saved filters</strong> list, the description
appears immediately below the saved filter name.</li>
<li>Check the <strong>Private Flag</strong> checkbox to prevent others from using this
saved filter, or clear the checkbox to allow others to use the saved
filter.</li>
</ul>
<p>Another user can use a filter you created and saved only (1) if the
filter is public, and (2) you email the user the shortcut (link) to the
filter.</p>
<ul class="simple">
<li>To delete the filter you are editing, click the <strong>Delete</strong> button
(<a class="reference internal" href="_images/image23.png"><img alt="image28" src="_images/image23.png" style="width: 0.77500in; height: 0.28125in;" /></a>).</li>
</ul>
<table border="1" class="docutils">
<colgroup>
<col width="4%" />
<col width="96%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image29" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">These are the only changes you can make to a saved filter. To make changes to the filter itself, run the search against the filter, then click <strong>advanced</strong> to define a new search filter based on the existing one. For details, see Instructions for <em>Building a Search String</em> on page 11.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<ol class="arabic simple">
<li>When finished making changes, click the <strong>Update</strong> button to save
your changes, or click the <strong>Cancel</strong> button to abandon them.</li>
<li>To delete a saved filter from the saved filters list:<ol class="arabic">
<li>Click the tab for the Search tool at the top of the tranSMART
window.</li>
<li>Click the <strong>saved filters</strong> link to the right of the <strong>Search</strong>
button. A list of saved search filters appears.</li>
<li>Click the <strong>delete</strong> link to the right of the saved filter name.</li>
</ol>
</li>
</ol>
</div>
</div>
<div class="section" id="working-with-search-results">
<h4>Working with Search Results<a class="headerlink" href="#working-with-search-results" title="Permalink to this headline">¶</a></h4>
<p>The results window displays all the clinical, documentary, and other
information found in the data warehouse that relates to the search
filter or search string.</p>
<p>The content of the results window varies, depending on the result
category you select (for example, Clinical Trials or mRNA Analysis) and
the type of view you want to use to display the results (for example,
Heatmap or Study View). Some result categories also let you further
refine the results by adding more filters to the search.</p>
<p>To select a result category to view, click the tab that contains the
result category name.</p>
<p>The following figure shows the sections of the results window:</p>
<p><a class="reference internal" href="_images/image24.png"><img alt="image30" src="_images/image24.png" style="width: 6.56250in; height: 1.59375in;" /></a></p>
<p>The tabs for the result categories Clinical Trials and mRNA Analysis
display pairs of numbers. These numbers represent the following results:</p>
<ul class="simple">
<li><strong>Clinical Trials ( x, y )</strong><ul>
<li>x = the number of statistically significant analyses. These hits
can be viewed in the Analysis View.</li>
<li>y = the total number of analyses. These hits can be viewed in the
Study View.</li>
</ul>
</li>
</ul>
<p>For example, in the preceding figure, 1 statistically significant
analysis was returned, and a total of 45 analyses were returned.</p>
<ul class="simple">
<li><strong>mRNA Analysis (x, y)</strong><ul>
<li>x = the number of statistically significant analyses. These hits
can be viewed in the Analysis View.</li>
<li>y = the total number of analyses. These hits can be viewed in the
Study View.</li>
</ul>
</li>
</ul>
<p>For example, in the preceding figure, 1 statistically significant
analysis was returned, and a total of 3 analyses were returned.</p>
<p>A <em>statistically significant analysis</em> is one in which the genes in a
gene signature, gene list, or pathway are differentially modulated in a
statistically significant way, indicating that the associated target or
pathway is probably affected by the treatment, disease or other topic
examined in the study.</p>
<p>To qualify as a statistically significant analysis, certain data points
(such as p-value) must be evaluated and attain an aggregate score that
meets or exceeds a particular threshold, based on an internal algorithm.
For information on the rules that determine how analysis results are
ranked, see <em>TEA Analyses</em> on page 23.</p>
<p>The following sections describe the views and operations available for
each result category:</p>
<ul class="simple">
<li><em>Clinical Trials Tab</em> (Page 17)</li>
<li><em>mRNA Analysis Tab</em> (page 18)</li>
<li><em>Literature Tab</em> (page 23)</li>
<li><em>Documents Tab</em> (page 23)</li>
</ul>
<div class="section" id="clinical-trials-tab">
<h5>Clinical Trials Tab<a class="headerlink" href="#clinical-trials-tab" title="Permalink to this headline">¶</a></h5>
<p>This result category contains data from internal clinical trials.</p>
<p>Click the <strong>Clinical Trials</strong> tab to display the results in this
category. The buttons in the following figure appear at the top of the
results list. You may see fewer buttons, depending on the results of
your particular search:</p>
<p><a class="reference internal" href="_images/image25.png"><img alt="image31" src="_images/image25.png" style="width: 4.29167in; height: 0.28125in;" /></a></p>
<p>These buttons give you access to the following views and operations:</p>
<ul class="simple">
<li><strong>Show Filters</strong> – Define additional filters to further refine the
search results.</li>
<li><strong>Heatmap</strong> – View the results as a heat map.</li>
<li><strong>Analysis View</strong> – View a list of the statistically significant
analyses of the clinical trials.</li>
</ul>
<p>Results are sorted from the highest-scoring analysis down to the lowest.</p>
<ul class="simple">
<li><strong>Study View</strong> – View a list of the clinical trials and, optionally,
<em>all</em> the analyses for each clinical trial – that is, those analyses
that are considered statistically significant and those that are not.</li>
</ul>
<p>Results are sorted from the clinical trial with the most matches with
the search filter or search string, down to the one with the least
matches.</p>
<ul class="simple">
<li><strong>Export Results</strong> – Export descriptions of each clinical trial, and
also all the analysis data from each of the clinical trials, to a
Microsoft Excel file. All clinical trial descriptions are written to
one worksheet in the file, and all analysis data is written to a
second worksheet in the file.</li>
</ul>
</div>
<div class="section" id="mrna-analysis-tab">
<h5>mRNA Analysis Tab<a class="headerlink" href="#mrna-analysis-tab" title="Permalink to this headline">¶</a></h5>
<p>The mRNA Analysis tab contains gene expression data derived largely from
external experiments and from some internal experiments.</p>
<p>Click the <strong>mRNA Analysis</strong> tab to display the results in this category.
The buttons in the following figure appear at the top of the results
list. You may see fewer buttons, depending on the results of your
particular search:</p>
<p><a class="reference internal" href="_images/image26.png"><img alt="image32" src="_images/image26.png" style="width: 3.57743in; height: 0.29008in;" /></a></p>
<p>These buttons give you access to the following views and operations:</p>
<ul class="simple">
<li><strong>Show Filters</strong> – Define additional filters to further refine the
search results.</li>
<li><strong>Analysis View</strong> – View the analyses of the experiments that are
ranked as statistically significant analyses.</li>
<li><strong>Study View</strong> – View the details of the experiments and, optionally,
<em>all</em> the analyses for each experiment – that is, those analyses that
are considered statistically significant and those that are not.</li>
<li><strong>Export Results</strong> – Export descriptions of each experiment, and also
all the analysis data from each of the experiments, to a Microsoft
Excel file. All descriptions of experiments are written to one
worksheet in the file, and all analysis data is written to a second
worksheet in the file.</li>
</ul>
<p>The following sections describe the results of experiments for the
disease<strong>Brain Diseases</strong>. Click the <strong>mRNA Analysis</strong> tab to see the
results.</p>
<div class="section" id="show-filters">
<h6>Show Filters<a class="headerlink" href="#show-filters" title="Permalink to this headline">¶</a></h6>
<p>Click the <strong>Show Filters</strong> button to further refine the search results.
When you click the button, a window containing filter fields appears
(shown below), and the <strong>Show Filters</strong> button is replaced by the <strong>Hide
Filters</strong> button.</p>
<p>In the figure below, filter selections are set for the broadest possible
search.</p>
<ol class="arabic simple">
<li>To narrow the search:<ol class="arabic">
<li>Specify one or more filters – for example, specify a particular
p-value to search against, and/or select a particular species from
the dropdown list.</li>
<li>Click <strong>Filter Results</strong> to start the search.</li>
</ol>
</li>
</ol>
<p><a class="reference internal" href="_images/image27.png"><img alt="image33" src="_images/image27.png" style="width: 4.63403in; height: 1.59375in;" /></a></p>
</div>
<div class="section" id="analysis-view">
<h6>Analysis View<a class="headerlink" href="#analysis-view" title="Permalink to this headline">¶</a></h6>
<p>Click the <strong>Analysis View</strong> button to view the statistically significant
analyses associated with mRNA experiments.</p>
<p>For information on the rules that determine how analysis results are
ranked, see <em>TEA Analyses</em> on page 23.</p>
<p><a class="reference internal" href="_images/image28.png"><img alt="image34" src="_images/image28.png" style="width: 6.08750in; height: 3.55347in;" /></a></p>
<p>When you click the <strong>+</strong> icon (<a class="reference internal" href="_images/image29.png"><img alt="image35" src="_images/image29.png" style="width: 0.11761in;" /></a>) to pull down the list of
biomarkers, you see two arrows next to each biomarker name. The arrows
have the following meanings:</p>
<ul class="simple">
<li>The leftmost arrow indicates whether the gene in the signature or
list is up-regulated (up arrow) or down-regulated (down arrow).</li>
<li>The rightmost arrow (not shown above) indicates whether the gene in
the comparison set is up-regulated (up arrow) or down-regulated (down
arrow).</li>
</ul>
<table border="1" class="docutils">
<colgroup>
<col width="9%" />
<col width="91%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image36" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">The leftmost arrow has meaning only for searches involving gene signatures or lists. The arrow is not shown for other searches.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<p>Each analysis also includes the following download option:</p>
<ul class="simple">
<li><strong>Excel</strong> – Download detailed analysis data (such as probe set, fold
change ratio, p‑value) to a Microsoft Excel spreadsheet.</li>
</ul>
</div>
<div class="section" id="study-view">
<h6>Study View<a class="headerlink" href="#study-view" title="Permalink to this headline">¶</a></h6>
<p>Click the <strong>Study View</strong> button to view the mRNA experiments that are
returned and, optionally, <em>all</em> the analyses for each experiment – that
is, those analyses that are considered statistically significant and
those that are not.</p>
<p><a class="reference internal" href="_images/image30.png"><img alt="image37" src="_images/image30.png" style="width: 6.07292in; height: 2.40625in;" /></a></p>
<ol class="arabic simple">
<li>To drill down from the list of experiments:<ol class="arabic">
<li>Click the <strong>+</strong> icon (<a class="reference internal" href="_images/image29.png"><img alt="image38" src="_images/image29.png" style="width: 0.11761in;" /></a>) to the left of the experiment
name to pull down a list of all the analyses done for the
experiment.</li>
</ol>
</li>
</ol>
<p>The analysis list is similar to the list of the statistically
significant analyses in the Analysis View. However, because Study View
includes analyses ranked as not statistically significant, TEA scores
and the designations co-regulated and anti-regulated are not specified
for the analyses in Study View.</p>
<ol class="arabic simple">
<li>Click the <strong>+</strong> icon (<a class="reference internal" href="_images/image29.png"><img alt="image39" src="_images/image29.png" style="width: 0.11761in;" /></a>) to the left of the <strong>BioMarker</strong>
label to pull down a list of applicable biomarkers for an analysis.
Note that the same export options for biomarkers are available in
Study View as in Analysis View.</li>
</ol>
</div>
<div class="section" id="export-results-in-analysis-view-or-study-view">
<h6>Export Results in Analysis View or Study View<a class="headerlink" href="#export-results-in-analysis-view-or-study-view" title="Permalink to this headline">¶</a></h6>
<p>While in either Analysis View or Study View, click the <strong>Export
Results</strong> button to export the results data in the view to a Microsoft
Excel spreadsheet:</p>
<p><a class="reference internal" href="_images/image31.png"><img alt="image40" src="_images/image31.png" style="width: 5.84103in; height: 1.07292in;" /></a></p>
<p>The Export function writes the following information to the spreadsheet:</p>
<ul class="simple">
<li>Descriptions of each experiment returned from the search. This is the
same information that appears in a details box for an experiment. In
addition, associated diseases are exported to the Excel file.</li>
<li>Information about the analyses associated with each experiment
returned from the search. Information includes:<ul>
<li>Analysis information displayed in the search results – for
example, analysis description, TEA score, the list of matching
biomarkers, and the probe set, fold change value, p-value, and TEA
p-value associated with each biomarker.</li>
<li>Additional information about an analysis, such as QA criteria,
analysis platform, descriptions of the biomarkers, biomarker type
(such as gene expression), and associated diseases involved in the
experiment.</li>
</ul>
</li>
</ul>
<p>All descriptions of experiments are written to one worksheet in the
file, and all analysis data is written to a second worksheet in the
file.</p>
</div>
<div class="section" id="export-information-about-a-particular-analysis">
<h6>Export Information about a Particular Analysis<a class="headerlink" href="#export-information-about-a-particular-analysis" title="Permalink to this headline">¶</a></h6>
<p>To export details about all the biomarkers in a particular analysis,
click the <strong>Excel</strong> button to the right of the analysis name – for
example:</p>
<p><a class="reference internal" href="_images/image32.png"><img alt="image41" src="_images/image32.png" style="width: 5.90625in; height: 1.94804in;" /></a></p>
<p>Note that the number of genes shown in parentheses after the
<strong>BioMarkers</strong> label (16995 in the above example), which specifies the
number of genes included in the analysis, may be less than the number of
rows written to the spreadsheet. The Export function writes one row of
data for each <em>probe set</em>, not each gene, and the same gene may be
associated with multiple probe sets.</p>
</div>
<div class="section" id="mouse-gene-homology-in-search-results">
<h6>Mouse Gene Homology in Search Results<a class="headerlink" href="#mouse-gene-homology-in-search-results" title="Permalink to this headline">¶</a></h6>
<p>Searches can return experiment results involving mouse genes. If
experiment data is collected on a human gene and the corresponding mouse
gene, a search against a human gene may potentially return results
containing both human and mouse gene expression experiments.</p>
<p>For example, information on both can be found by clicking the <strong>Export
Results</strong> button in the search results. The <strong>Organism</strong> column in the
Excel worksheet indicates whether a particular measurement was made on a
human gene or a mouse gene.</p>
<p>The following figure shows part of an Excel worksheet containing the
results of a search against the MET gene:</p>
<p><a class="reference internal" href="_images/image33.png"><img alt="image42" src="_images/image33.png" style="width: 6.49606in; height: 1.83071in;" /></a></p>
</div>
<div class="section" id="additional-resources">
<h6>Additional Resources<a class="headerlink" href="#additional-resources" title="Permalink to this headline">¶</a></h6>
<p>An mRNA Analysis search result contains links to the following
resources:</p>
<table border="1" class="docutils">
<colgroup>
<col width="12%" />
<col width="88%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head">Resource Link</th>
<th class="head">Description</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td><p class="first">Experiment name</p>
<p>Example:</p>
<p class="last"><a class="reference internal" href="_images/image34.png"><img alt="image43" src="_images/image34.png" style="width: 0.83323in; height: 0.21872in;" /></a></p>
</td>
<td><p class="first">View information about the experiment, including title, description, and primary investigator.</p>
<p class="last">The display may contain links to additional information, such as NCBI GEO and ArrayExpress data.</p>
</td>
</tr>
<tr class="row-odd"><td><p class="first">QA criteria</p>
<p>Example:</p>
<p class="last"><a class="reference internal" href="_images/image35.png"><img alt="image44" src="_images/image35.png" style="width: 1.84375in; height: 0.25000in;" /></a></p>
</td>
<td>View key parameters of the experiment, such as p-Value and fold-change cutoffs, analysis platform, and methodology.</td>
</tr>
<tr class="row-even"><td><p class="first">Gene</p>
<p>Example:</p>
<p class="last"><a class="reference internal" href="_images/image36.png"><img alt="image45" src="_images/image36.png" style="width: 0.49653in; height: 0.15625in;" /></a></p>
</td>
<td><p class="first">Search the following sites for information about the gene:</p>
<p class="last"><a class="reference internal" href="_images/image37.png"><img alt="image46" src="_images/image37.png" style="width: 3.38499in; height: 0.29163in;" /></a></p>
</td>
</tr>
<tr class="row-odd"><td><a class="reference internal" href="_images/image38.png"><img alt="image47" src="_images/image38.png" style="width: 0.47917in; height: 0.15625in;" /></a></td>
<td>Export data (such as gene, probe set, and fold-change ratio) for the matching biomarkers in a particular analysis to Microsoft Excel.</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="section" id="literature-tab">
<h5>Literature Tab<a class="headerlink" href="#literature-tab" title="Permalink to this headline">¶</a></h5>
<p>This result category contains hits from a set of curated articles within
a set of selected diseases.</p>
<p>Click the <strong>Literature</strong> tab to display the results in this category.
The following figure shows the controls that appear at the top of the
results list.</p>
<p>These controls give you access to the following views and operations:</p>
<ul class="simple">
<li><strong>Show Filters</strong> button – Define additional filters to further refine
the search results.</li>
<li><strong>Export Results</strong> button – Write curation data to Microsoft Excel.</li>
<li><strong>Results for</strong> dropdown – Specify the type of literature results you
want to see in the categories.</li>
</ul>
</div>
<div class="section" id="documents-tab">
<h5>Documents Tab<a class="headerlink" href="#documents-tab" title="Permalink to this headline">¶</a></h5>
<p>The search results in this category are based on internal text-indexed
document repositories.</p>
</div>
</div>
</div>
<div class="section" id="tea-analyses">
<h3>TEA Analyses<a class="headerlink" href="#tea-analyses" title="Permalink to this headline">¶</a></h3>
<p>Target Enrichment Analysis (TEA) measures the enrichment of a gene
signature, gene list, or pathway in a microarray expression experiment.</p>
<table border="1" class="docutils">
<colgroup>
<col width="10%" />
<col width="90%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image48" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">For information on how TEA scores are calculated, see <em>Appendix A: How TEA Scores Are Calculated</em> on page 95.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<div class="section" id="tea-indicators-applied-to-individual-biomarkers">
<h4>TEA Indicators Applied to Individual Biomarkers<a class="headerlink" href="#tea-indicators-applied-to-individual-biomarkers" title="Permalink to this headline">¶</a></h4>
<p>The Study View of mRNA Analysis search result lists all experiments that
satisfy the search criteria. Further, in Study View, you can list:</p>
<ul class="simple">
<li>All of an experiment’s analyses that satisfy the search criteria</li>
<li>All of an analysis’ biomarkers that satisfy the search criteria</li>
</ul>
<p>To drill down to the matching analyses in an experiment, click the <strong>+</strong>
icon (<a class="reference internal" href="_images/image29.png"><img alt="image49" src="_images/image29.png" style="width: 0.11761in;" /></a>) next to the experiment name. To drill down to the
matching biomarkers in an analysis, click the <strong>+&nbsp;</strong>icon next to the
label <strong>BioMarkers</strong> under the analysis name.</p>
<p>The following example shows the experiment <strong>GSE4226</strong>in Study View.
The biomarkers for the analysis <strong>DiseaseState =&gt; Sporadic Alzheimer_s
Disease vs Normal elderly control</strong> are displayed:</p>
<p><a class="reference internal" href="_images/image39.png"><img alt="image50" src="_images/image39.png" style="width: 6.07571in; height: 2.00901in;" /></a></p>
<p>Notice the rightmost column of biomarker values: <strong>TEA p-Value</strong>. These
normalized p‑values are intermediate values in the TEA calculation. To
be considered a statistically significant analysis, an analysis must
have at least one matching biomarker with a TEA p-Value of less than
0.05.</p>
<p>The following figure shows the same experiment and analysis from the
figure above, but in Analysis View:</p>
<p><a class="reference internal" href="_images/image40.png"><img alt="image51" src="_images/image40.png" style="width: 6.11363in; height: 1.48649in;" /></a></p>
<p>Statistically significant analyses are candidates for display in the
Analysis View, after further TEA calculations are performed to determine
whether the analysis is a <strong>significant TEA analysis</strong> or an
<strong>insignificant TEA analysis</strong>.</p>
</div>
<div class="section" id="tea-indicators-applied-to-an-analysis">
<h4>TEA Indicators Applied to an Analysis<a class="headerlink" href="#tea-indicators-applied-to-an-analysis" title="Permalink to this headline">¶</a></h4>
<p>The TEA algorithm assigns an aggregate score to each analysis within an
experiment. A TEA score is a binomial distribution of normalized
p-values, calculated in the context of the following factors:</p>
<ul class="simple">
<li><strong>With gene signatures and gene lists</strong> – The level of co-regulation
or anti-regulation of the genes within the gene signature or gene
list, as compared with the experiment.</li>
<li><strong>With pathways –</strong> The level of up-regulation or down-regulation of
the genes within the pathway, as compared with the experiment.</li>
</ul>
<table border="1" class="docutils">
<colgroup>
<col width="12%" />
<col width="88%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image52" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">For details on the TEA algorithm, see <em>Appendix A: How TEA Scores Are Calculated</em> on page 95.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<p>TEA identifies experiments where the genes in the signature, list, or
pathway are <em>differentially modulated</em>, indicating that the target is
affected by the treatment, disease or other topic examined in the
experiment.</p>
<div class="section" id="what-the-tea-score-means">
<h5>What the TEA Score Means<a class="headerlink" href="#what-the-tea-score-means" title="Permalink to this headline">¶</a></h5>
<p>The TEA score displayed for an analysis of an experiment is not the
actual TEA score calculated by the TEA algorithm. TEA scores are
typically very small decimal numbers that are not easily human-readable.
To aid users in interpreting the relative value of TEA scores, scores
are converted to a larger number, as follows:</p>
<p>Displayed_TEA_Score = -log(Actual_TEA_Score)</p>
<p>The larger the displayed TEA score, the more statistically significant
is the analysis.</p>
<p>Typically, displayed TEA scores for statistically significant analyses
of experiments range from 3 to 30 or 40.</p>
<p>Analyses of experiments are grouped into the categories <strong>Significant
TEA Analyses</strong> and <strong>Insignificant TEA Analyses</strong>, as follows:</p>
<ul class="simple">
<li>Significant TEA analyses have a displayed TEA score of &gt;= 2.9957.</li>
<li>Insignificant TEA analyses have a displayed TEA score of &lt; 2.9957.</li>
</ul>
</div>
<div class="section" id="what-co-anti-regulation-and-up-down-regulation-mean">
<h5>What Co-/Anti-Regulation and Up-/Down-Regulation Mean<a class="headerlink" href="#what-co-anti-regulation-and-up-down-regulation-mean" title="Permalink to this headline">¶</a></h5>
<p>An analysis of a statistically significant experiment returned from a
search against a gene signature or list is designated as <em>co-regulated</em>
or <em>anti-regulated</em>. An analysis of a statistically significant
experiment returned from a search against a pathway is designated as
<em>up-regulated</em> or <em>down-regulated</em>.</p>
<p>The following table describes what these terms imply in the context of
an analysis of a statistically significant experiment:</p>
<table border="1" class="docutils">
<colgroup>
<col width="11%" />
<col width="57%" />
<col width="32%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head">&nbsp;</th>
<th class="head">Gene Signature/List</th>
<th class="head">Pathway</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>Co-Regulated</td>
<td><p class="first">Genes that are up-regulated in the signature or list are predominantly up-regulated in the experiment.</p>
<p class="last">Genes that are down-regulated in the signature or list are predominantly down-regulated in the experiment.</p>
</td>
<td>n/a</td>
</tr>
<tr class="row-odd"><td>Anti-Regulated</td>
<td><p class="first">Genes that are up-regulated in the signature or list are predominantly down-regulated in the experiment.</p>
<p class="last">Genes that are down-regulated in the signature or list are predominantly up-regulated in the experiment.</p>
</td>
<td>n/a</td>
</tr>
<tr class="row-even"><td>Up-Regulated</td>
<td>n/a</td>
<td>Genes in the experiment are predominantly up-regulated.</td>
</tr>
<tr class="row-odd"><td><strong>Down-Regulated</strong></td>
<td>n/a</td>
<td>Genes in the experiment are predominantly down-regulated.</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="section" id="tea-indicators-applied-to-an-individual-gene">
<h4>TEA Indicators Applied to an Individual Gene<a class="headerlink" href="#tea-indicators-applied-to-an-individual-gene" title="Permalink to this headline">¶</a></h4>
<p>In an analysis list, TEA indicators for a gene appear as arrows, as
shown in the figure below. The leftmost arrow represents the gene
expression in the gene signature or list. The rightmost arrow represents
the gene expression in the experiment:</p>
<p><a class="reference internal" href="_images/image41.png"><img alt="image53" src="_images/image41.png" style="width: 6.00000in; height: 0.79359in;" /></a></p>
<table border="1" class="docutils">
<colgroup>
<col width="15%" />
<col width="85%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image54" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">The leftmost arrow appears only for gene signatures and gene lists.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<p>The direction of the arrows indicates the following:</p>
<ul class="simple">
<li><strong>Up-arrow</strong> – An upward-pointing arrow alongside a gene indicates
that the gene is up-regulated in the gene signature/list (leftmost
arrow) or in the experiment (rightmost arrow).</li>
</ul>
<p>If both arrows point in the same direction, the gene is co-regulated in
the signature/list and the experiment. If the arrows point in opposite
directions, the gene is anti-regulated.</p>
<ul class="simple">
<li><strong>Down-arrow</strong> – A downward-pointing arrow alongside a gene indicates
that the gene is down-regulated in the gene signature/list (leftmost
arrow) or in the experiment (rightmost arrow).</li>
</ul>
<p>If both arrows point in the same direction, the gene is co-regulated in
the signature/list and the experiment. If the arrows point in opposite
directions, the gene is anti-regulated.</p>
<p>The relationships between TEA indicators for genes and TEA indicators
for an experiment are as follows:</p>
<ul class="simple">
<li><strong>Co-regulated genes</strong> – Up- or down-regulated genes in the
signature/list are similarly up- or down-regulated in the experiment.</li>
<li><strong>Anti-regulated</strong> <strong>genes</strong> – Up- or down-regulated genes in the
signature/list are conversely down- or up-regulated in the
experiment.</li>
</ul>
<p>Chapter&nbsp;3</p>
</div>
</div>
</div>
<div class="section" id="chapter-3-dataset-explorer">
<h2>Chapter 3: Dataset Explorer<a class="headerlink" href="#chapter-3-dataset-explorer" title="Permalink to this headline">¶</a></h2>
<p>Dataset Explorer lets you compare data generated for test subjects in
two different study groups, based on criteria and points of comparison
that you specify. Dataset Explorer is useful to help you test a
hypothesis that involves the criteria and points of comparison that you
select.</p>
<div class="section" id="overview-of-the-ui">
<h3>Overview of the UI<a class="headerlink" href="#overview-of-the-ui" title="Permalink to this headline">¶</a></h3>
<p>The figure below shows the Dataset Explorer interface. It is divided
into two panes:</p>
<p><strong>Left pane</strong></p>
<ul class="simple">
<li>Lets you select the study of interest.</li>
<li>Provides a navigation tree where you select the criteria for
membership in the study groups and the points of comparison between
the study groups.</li>
</ul>
<p><a href="#id4"><span class="problematic" id="id5">**</span></a>Right pane **</p>
<ul class="simple">
<li>Lets you define the criteria that test subjects must satisfy to
become members of one of the two groups being compared. Each of these
groups is called a <em>subset</em> because it typically contains only some
of the subjects in the actual study group involved in the study.</li>
</ul>
<p>You define the criteria for the subsets in the subset definition boxes
shown below. Subjects who do not satisfy the criteria you define are
excluded from the subsets.</p>
<ul>
<li><p class="first">Provides summary data about the subjects being compared, and several
different views of the comparison data.</p>
<blockquote>
<div><p><a class="reference internal" href="_images/image42.png"><img alt="image55" src="_images/image42.png" style="width: 5.99097in; height: 2.15347in;" /></a></p>
</div></blockquote>
</li>
</ul>
<p>The following table describes the buttons and tabs in the right pane of
Dataset Explorer:</p>
<table border="1" class="docutils">
<colgroup>
<col width="16%" />
<col width="84%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head">Button or Tab</th>
<th class="head">Description</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>Generate Summary Statistics button</td>
<td><p class="first">Displays tables and charts that describe demographic information about the subjects in the subsets, and also analyses of criteria included in the subset definitions.</p>
<p class="last">The tables and charts are displayed in the Results/Analysis view.</p>
</td>
</tr>
<tr class="row-odd"><td>Summary button</td>
<td>Displays a summary of the query criteria you specified. Dataset Explorer uses these criteria to select the subjects for the subsets.</td>
</tr>
<tr class="row-even"><td>Clear button</td>
<td>Clears all the criteria in the subset definition boxes.</td>
</tr>
<tr class="row-odd"><td>Save button</td>
<td><p class="first">Saves the criteria definition. This allows you to re-generate the comparison at a later time without having to reconstruct the criteria that select the subjects for the subsets.</p>
<p class="last">For more information, see <em>Saving Comparison Definitions</em> on page 39.</p>
</td>
</tr>
<tr class="row-even"><td>Export button</td>
<td>Exports summary statistics data or expression data to Microsoft Excel.</td>
</tr>
<tr class="row-odd"><td>Print button</td>
<td>Prints the tables and charts in the Results/Analysis view.</td>
</tr>
<tr class="row-even"><td>Comparison tab</td>
<td>Removes the currently displayed view (that is, the Results/Analysis view, or Grid view) and re-displays the subset definition boxes. This allows you to further refine the subjects for the comparison.</td>
</tr>
<tr class="row-odd"><td>Advanced Workflow tab</td>
<td>Displays advanced analyses and visualizations that you can perform on data.</td>
</tr>
<tr class="row-even"><td>Results/Analysis tab</td>
<td>Displays tables and charts containing comparison and analysis data generated from the “Summary Statistics” workflow.</td>
</tr>
<tr class="row-odd"><td>Grid View tab</td>
<td>Displays the comparison and analysis data in grid format.</td>
</tr>
<tr class="row-even"><td>Jobs tab</td>
<td>Displays previously run analyses.</td>
</tr>
<tr class="row-odd"><td>Data Export tab</td>
<td>Allows you to select data to export for further analysis in an external tool.</td>
</tr>
<tr class="row-even"><td>Export Jobs tab</td>
<td>Displays previously exported jobs.</td>
</tr>
</tbody>
</table>
</div>
<div class="section" id="using-dataset-explorer">
<h3>Using Dataset Explorer<a class="headerlink" href="#using-dataset-explorer" title="Permalink to this headline">¶</a></h3>
<p>Four basic tasks are involved in using Dataset Explorer:</p>
<ul class="simple">
<li>Select the study (clinical trial or experiment) to use in the
comparison.</li>
<li>Specify the criteria for membership in the two study groups. Note
some analyses only allow for the specification of one group at this
time.</li>
<li>Generate summary statistics for the two study groups.</li>
<li>Specify the points of comparison to apply to the study groups.</li>
</ul>
<table border="1" class="docutils">
<colgroup>
<col width="8%" />
<col width="92%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image56" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">You may see the notations <strong>NA</strong> and <strong>Unknown</strong> in the study data. <strong>NA</strong> indicates not applicable, and <strong>Unknown</strong> indicates not available.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<div class="section" id="public-and-private-studies">
<h4>Public and Private Studies<a class="headerlink" href="#public-and-private-studies" title="Permalink to this headline">¶</a></h4>
<p>Dataset Explorer studies can be either public or private. Public studies
can be found both in the <strong>Public Studies</strong> folder of the Dataset
Explorer navigation tree, as well as in the research-specific folders.</p>
<p>You can perform all the operations described in this chapter on public
studies. No special privileges are required.</p>
<p>To perform operations described in this chapter on a private study, a
tranSMART Administrator must assign you access rights to the study.
Access rights are based on the following access levels:</p>
<table border="1" class="docutils">
<colgroup>
<col width="9%" />
<col width="91%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head">Access Level</th>
<th class="head">Privileges</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>VIEW</td>
<td>Define the criteria for the study groups to be compared, generate summary statistics for the study groups, and specify points of comparison for the study groups.</td>
</tr>
<tr class="row-odd"><td>EXPORT</td>
<td>All privileges of the VIEW access level, plus the ability to export comparison data or expression data to a Microsoft Excel spreadsheet.</td>
</tr>
<tr class="row-even"><td>OWN</td>
<td><p class="first">All VIEW and EXPORT privileges.</p>
<p class="last">This access level can only be assigned to the owner of the study.</p>
</td>
</tr>
</tbody>
</table>
<p>If you do not have access rights to the study you want (that is, if the
study name is grayed out), contact a tranSMART Administrator. The
administrator will contact the study owner to find out if you should be
granted VIEW access, EXPORT access, or no access.</p>
<table border="1" class="docutils">
<colgroup>
<col width="8%" />
<col width="92%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image57" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">Even if you have no access rights to a private study, you can read a description of the study. For information, see <em>Viewing a Study</em> on page 45.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
</div>
<div class="section" id="selecting-the-study">
<h4>Selecting the Study<a class="headerlink" href="#selecting-the-study" title="Permalink to this headline">¶</a></h4>
<p>You select the study in the left pane of Dataset Explorer. You have
several ways to select the study, based on the tab you choose – Search
Terms or Navigate Terms.</p>
<div class="section" id="search-terms-tab">
<h5>Search Terms Tab<a class="headerlink" href="#search-terms-tab" title="Permalink to this headline">¶</a></h5>
<p>Use this tab to search for studies using one or a combination of the
following fields:</p>
<ul class="simple">
<li><strong>Search</strong> field. It lets you specify part or all of a term from a
study that is stored in the tranSMART database. Search terms may
include part or all of a study name, the text in a branch of the
Dataset Explorer navigation tree, or some attribute of a study, such
as a disease or an area of clinical interest.</li>
</ul>
<p>Example:</p>
<p><a class="reference internal" href="_images/image43.png"><img alt="image58" src="_images/image43.png" style="width: 3.45347in; height: 2.35347in;" /></a></p>
<p>If you want to base your search on a study name, note the following
naming conventions for Public Studies in Dataset Explorer:</p>
<table border="1" class="docutils">
<colgroup>
<col width="8%" />
<col width="92%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head">Study Type</th>
<th class="head">Naming Convention</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>Public Studies</td>
<td><p class="first">Name segments in the following typical format:</p>
<p><em>Condition_StudyFirstAuthor</em>_<em>GEOid</em></p>
<p>Example: ProstateCancer_Ambs_GSE6956</p>
<p class="last">If you prefer, you can rearrange the order of the segments (for example, author segment first). The name structure is determined by the ETL process that loads the data into the i2b2 database.</p>
</td>
</tr>
</tbody>
</table>
<div class="section" id="selecting-and-opening-a-study-in-a-search-result">
<h6>Selecting and Opening a Study in a Search Result<a class="headerlink" href="#selecting-and-opening-a-study-in-a-search-result" title="Permalink to this headline">¶</a></h6>
<p>A search result may include multiple entries. Further, an entry may not
indicate the study it is from. To see the name of the study that an
entry represents, hover the mouse pointer over the entry – for example:</p>
<p><a class="reference internal" href="_images/image44.png"><img alt="image59" src="_images/image44.png" style="width: 3.26887in; height: 2.54955in;" /></a></p>
<p>If you want more information about the study represented by an entry,
right-click the entry, then click <strong>Show</strong> <strong>Definition</strong> to open the
details box for the study:</p>
<p><a class="reference internal" href="_images/image45.png"><img alt="image60" src="_images/image45.png" style="width: 3.31528in; height: 1.14444in;" /></a></p>
<p>To open a study from an entry in a search result, right-click the entry,
then click <strong>Show</strong> <strong>Node</strong>. The study appears in the Dataset Explorer
navigation tree, where you can open any of the branches (nodes) in the
study.</p>
<table border="1" class="docutils">
<colgroup>
<col width="14%" />
<col width="86%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image61" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">You may need to scroll down slightly in the navigation tree to see the study.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="section" id="navigate-terms-tab">
<h5>Navigate Terms Tab<a class="headerlink" href="#navigate-terms-tab" title="Permalink to this headline">¶</a></h5>
<p>Use this tab to browse through all the experiments in the navigation
tree to select and open the study you want.</p>
<p>Studies that are grayed out are private studies that you are not
authorized to access.</p>
<p>To display the details box for a study, right-click the study name and
click <strong>Show</strong> <strong>Definition</strong>. You can display the details box for a
study whether or not the study is grayed out.</p>
</div>
<div class="section" id="branches-and-leaves-of-the-navigation-tree">
<h5>Branches and Leaves of the Navigation Tree<a class="headerlink" href="#branches-and-leaves-of-the-navigation-tree" title="Permalink to this headline">¶</a></h5>
<p>The Dataset Explorer navigation tree looks and works much likeyour
operating system’s file explorer: it is a an expandable hierarchy of
folders, sub-folders, and files. Dataset Explorer is a hierarchy of
folders and sub-folders (the branches) and values (the leaves) that
reflect aspects of the trial, such as research metrics, compounds used,
and patient demographics.</p>
<p>In Dataset Explorer, all levels of the tree, both branches and leaves,
are referred to as nodes.</p>
<p>The following figure shows typical top-level nodes of a study. Some
studies may not require all of these nodes, and others may require
additional nodes (such as Published Conclusions):</p>
<p><a class="reference internal" href="_images/image46.png"><img alt="image62" src="_images/image46.png" style="width: 2.51010in; height: 1.71854in;" /></a></p>
<p>The following table describes possible top-level nodes of a study:</p>
<table border="1" class="docutils">
<colgroup>
<col width="18%" />
<col width="82%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head">Node</th>
<th class="head">Description</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>Biomarker Data</td>
<td>Measurements of biomarkers such as RBM antigens, gene expressions, antibodies and antigens in ELISA tests, and SNPs.</td>
</tr>
<tr class="row-odd"><td>Clinical Data</td>
<td>Primary and secondary endpoints, and other measurements from the study.</td>
</tr>
<tr class="row-even"><td>Samples and Timepoints</td>
<td>Tested samples (such as tissue or blood) and time periods when the samples were taken.</td>
</tr>
<tr class="row-odd"><td>Scheduled Visits</td>
<td>Periodic stages of the trial during which patients are seen.</td>
</tr>
<tr class="row-even"><td>Design Factors</td>
<td><p class="first">Compounds involved in the study, dosages, and regularity with which the compounds were administered.</p>
<p class="last"><strong>Note:</strong> With clinical trials, this node is typically named Treatment Groups.</p>
</td>
</tr>
<tr class="row-odd"><td>Sample Factors</td>
<td>Patient information, such as demographics and medical history.</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="section" id="populating-the-study-groups">
<h4>Populating the Study Groups<a class="headerlink" href="#populating-the-study-groups" title="Permalink to this headline">¶</a></h4>
<p>You populate the study groups by defining criteria that members of each
group must satisfy. For example, members of study groups might be
required to satisfy a weight or age requirement. Dataset Explorer lets
you build a set of criteria for each study group that can be as simple
or as complex as you need.</p>
<p>The study groups you define are called <em>subsets</em>, because typically,
after your criteria are applied, the members of a resulting study group
are a subset of the full study group that participated in the study.</p>
<div class="section" id="selecting-criteria-for-the-study-groups">
<h5>Selecting Criteria for the Study Groups<a class="headerlink" href="#selecting-criteria-for-the-study-groups" title="Permalink to this headline">¶</a></h5>
<p>You define the study groups by selecting criteria (called concepts) from
the navigation tree and dragging them into the subset definition boxes.</p>
<div class="section" id="visual-aids-to-help-you-select-the-criteria">
<h6>Visual Aids to Help You Select the Criteria<a class="headerlink" href="#visual-aids-to-help-you-select-the-criteria" title="Permalink to this headline">¶</a></h6>
<p>Each concept node in the navigation tree displays the following
information about the concept:</p>
<ul>
<li><p class="first">The numbers in parentheses at each node of the tree indicate the
number of subjects to whom that node applies. For example, in the
figure below, there are a total of 327 subjects in the study.</p>
</li>
<li><p class="first">In tranSMART, data values are represented in one of three ways: by
number, by text, or by high dimensional data (SNP, gene expression,
etc.) stored as <em>arrays</em>.</p>
<p>The three types of data values are described in the illustration
below:</p>
<p><a class="reference internal" href="_images/image47.png"><img alt="image63" src="_images/image47.png" style="width: 6.00000in; height: 1.78846in;" /></a></p>
</li>
</ul>
</div>
</div>
<div class="section" id="specifying-a-numeric-value">
<h5>Specifying a Numeric Value<a class="headerlink" href="#specifying-a-numeric-value" title="Permalink to this headline">¶</a></h5>
<p>When you drag a numeric concept into a subset definition box, the Set
Value dialog appears:</p>
<p><a class="reference internal" href="_images/image48.png"><img alt="image64" src="_images/image48.png" style="width: 4.17500in; height: 1.67500in;" /></a></p>
<p>Use the Set Value dialog to specify how you want to constrain the
numeric values to use in the subset definition. To do so, first select
one of the following choices:</p>
<table border="1" class="docutils">
<colgroup>
<col width="10%" />
<col width="90%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head">Selection</th>
<th class="head">Description</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>No Value</td>
<td><p class="first">Values are not constrained. All the numeric data associated with the concept are factored into the subset definition.</p>
<p class="last">If you select <strong>No Value</strong>, no other information is required. Click <strong>OK</strong> to add the concept with all its associated numeric data to the subset.</p>
</td>
</tr>
<tr class="row-odd"><td>By high/low flag</td>
<td><p class="first">If the testing laboratory has grouped the numeric values into high/low/normal ranges, select the range to factor into the subset definition.</p>
<p class="last">When you select <strong>By high/low flag</strong>, the <strong>Please select range</strong> field appears. Select the range you want and click <strong>OK</strong>.</p>
</td>
</tr>
<tr class="row-even"><td>By numeric value</td>
<td><p class="first">Values are constrained by an exact value or a range of values.</p>
<p>After you select <strong>By numeric value</strong>:</p>
<ul class="simple">
<li>Select one of the following numeric operators in the <strong>Please select operator dropdown</strong>:</li>
</ul>
<p><a class="reference internal" href="_images/image49.png"><img alt="image65" src="_images/image49.png" style="width: 2.00279in; height: 1.12222in;" /></a></p>
<ul class="simple">
<li>In <strong>Please enter value</strong>, type the numeric value that the operator applies to.</li>
</ul>
<p>For example, to constrain the ages of subjects to 50 years or younger, select LESS THAN OR EQUAL TO(&lt;=) in the dropdown, then type 50 in the <strong>Please enter value</strong> field.</p>
<ul class="simple">
<li>Click <strong>OK.</strong></li>
</ul>
<p class="last">See the next section for information on viewing the numeric values associated with the concept and that you can select from.</p>
</td>
</tr>
</tbody>
</table>
<table border="1" class="docutils">
<colgroup>
<col width="5%" />
<col width="95%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image66" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">When finished defining the numeric constraint on the Set Value dialog, be sure to click <strong>OK</strong> and not press the <strong>Enter</strong> key. Pressing <strong>Enter</strong> will activate the subset button that has focus – the <strong>Exclude</strong> button in the example below:</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<p><a class="reference internal" href="_images/image50.png"><img alt="image67" src="_images/image50.png" style="width: 2.83333in; height: 0.73958in;" /></a></p>
<div class="section" id="viewing-the-numeric-values-associated-with-a-concept">
<h6>Viewing the Numeric Values Associated with a Concept<a class="headerlink" href="#viewing-the-numeric-values-associated-with-a-concept" title="Permalink to this headline">¶</a></h6>
<p>Note the buttons <strong>Show Histogram</strong> and <strong>Show Histogram for subset</strong> in
the Set Value dialog. The histograms show how the numeric values
associated with the concept that you placed in the subset box are
distributed among the subjects across both subsets, or in the particular
subset you are currently defining, respectively.</p>
<p>A histogram may be helpful in determining the number to set as the
constraining factor for a concept. For example, suppose you drag a
Weight concept into a subset box, then click <strong>Show Histogram for
subset</strong>. In the following histogram of the weights of test subjects,
the weights range from about 25 kg to just under 125 kg. The largest bin
represents just under 50 subjects. You may want to use these weight
parameters to help you determine the value to set for the weight
concept.</p>
<p><a class="reference internal" href="_images/image51.png"><img alt="image68" src="_images/image51.png" style="width: 2.67708in; height: 1.94097in;" /></a></p>
<p>You can get more specific information about the number of subjects
represented by a particular bin and the average of the values in the bin
by hovering the mouse cursor over the bin you are interested in. For
example, in the following figure, the largest bin represents 49 subjects
with an average weight of 68.7 kg:</p>
<p><a class="reference internal" href="_images/image52.png"><img alt="image69" src="_images/image52.png" style="width: 2.67708in; height: 1.94097in;" /></a></p>
</div>
</div>
<div class="section" id="saving-comparison-definitions">
<h5>Saving Comparison Definitions<a class="headerlink" href="#saving-comparison-definitions" title="Permalink to this headline">¶</a></h5>
<p>You may save your search criteria in order to regenerate the comparison
at a later time without having to redefine the subsets.</p>
<ol class="arabic">
<li><p class="first">To save search criteria:</p>
<ol class="arabic">
<li><p class="first">Run tranSMART, then click the <strong>Dataset Explorer</strong> tab.</p>
</li>
<li><p class="first">Select the study of interest.</p>
</li>
<li><p class="first">Define the cohorts whose data points will be represented.</p>
</li>
<li><p class="first">Click <strong>Save</strong>:</p>
<p><a class="reference internal" href="_images/image53.png"><img alt="image70" src="_images/image53.png" style="width: 4.12014in; height: 0.52639in;" /></a></p>
</li>
<li><p class="first">Click <strong>Email this comparison</strong>:</p>
<p><a class="reference internal" href="_images/image54.png"><img alt="image71" src="_images/image54.png" style="width: 2.08125in; height: 1.09028in;" /></a></p>
</li>
</ol>
</li>
</ol>
<p>Your email application will open with a link to the saved comparison.</p>
<ol class="arabic simple">
<li>Send the email to yourself so that you can retrieve the comparison
later. Optionally, send it to colleagues who might be interested in
the comparison.</li>
</ol>
<p>When you or someone else clicks the link in the email, Dataset Explorer
opens with the subset boxes pre-defined.</p>
</div>
<div class="section" id="joining-multiple-criteria-for-a-subset-definition">
<h5>Joining Multiple Criteria for a Subset Definition<a class="headerlink" href="#joining-multiple-criteria-for-a-subset-definition" title="Permalink to this headline">¶</a></h5>
<p>Multiple criteria for a subset definition are joined by one of the
following logical operators: AND, OR, or AND NOT.</p>
<p>The rules for joining multiple criteria are as follows:</p>
<ul class="simple">
<li>Criteria in separate subset definition boxes are joined by an AND
operator.</li>
</ul>
<p>For example, the following definition boxes select only male subjects,
AND males whose weights are between 65 kg and 90 kg:</p>
<p><a class="reference internal" href="_images/image55.png"><img alt="image72" src="_images/image55.png" style="width: 3.26560in; height: 1.47188in;" /></a></p>
<ul class="simple">
<li>Criteria within the same subset definition box are joined by an OR
operator.</li>
</ul>
<p>For example, to use the extreme ends of the weight scale for your weight
criterion, you might add the following to a definition box:</p>
<p><a class="reference internal" href="_images/image56.png"><img alt="image73" src="_images/image56.png" style="width: 3.24375in; height: 0.79688in;" /></a></p>
<p>This criterion selects subjects whose weight is either 50 kg or less, OR
100 kg or greater.</p>
<ul class="simple">
<li>To join a definition box with an AND NOT operator, click the
<strong>Exclude</strong> button above the definition box.</li>
</ul>
<p>The figure below selects only male subjects, but not those who weigh
between 50&nbsp;kg and 100 kg:</p>
<p><a class="reference internal" href="_images/image57.png"><img alt="image74" src="_images/image57.png" style="width: 3.27188in; height: 1.50000in;" /></a></p>
<p>Note that when you click the <strong>Exclude</strong> button, the button label
changes to <strong>Include</strong>, allowing you to join the criteria in the box
with an AND operator later if you choose.</p>
</div>
<div class="section" id="modifying-or-deleting-criteria">
<h5>Modifying or Deleting Criteria<a class="headerlink" href="#modifying-or-deleting-criteria" title="Permalink to this headline">¶</a></h5>
<p>To delete or modify a criterion in a subset definition box, right-click
the criterion and select either <strong>Delete</strong> or <strong>Set Value</strong>.</p>
<p>To remove the entire contents of a subset definition box from the subset
definition, click the <strong>X</strong> icon (<a class="reference internal" href="_images/image58.png"><img alt="image75" src="_images/image58.png" style="width: 0.14801in; height: 0.17708in;" /></a>) above the box:</p>
<p><a class="reference internal" href="_images/image59.png"><img alt="image76" src="_images/image59.png" style="width: 3.39422in; height: 0.88114in;" /></a></p>
</div>
</div>
<div class="section" id="generating-summary-statistics">
<h4>Generating Summary Statistics<a class="headerlink" href="#generating-summary-statistics" title="Permalink to this headline">¶</a></h4>
<p>When you finish defining criteria for the groups to compare – the
subsets – click the <strong>Generate Summary Statistics</strong> button.</p>
<p>tranSMART displays tables and charts of information that describe the
subsets. The information is displayed in the Results/Analysis view in
the following sections:</p>
<ul class="simple">
<li>A summary of the criteria used to define subsets to compare. Example:</li>
</ul>
<p><a class="reference internal" href="_images/image60.png"><img alt="image77" src="_images/image60.png" style="width: 6.00000in; height: 0.91795in;" /></a></p>
<ul>
<li><p class="first">A table showing the number of subjects in each subset who match the
subset criteria. Example:</p>
<blockquote>
<div><p><a class="reference internal" href="_images/image61.png"><img alt="image78" src="_images/image61.png" style="width: 2.08333in; height: 0.83333in;" /></a></p>
<p>In this example, 52 subjects matched the criteria for Subset 1, and
48 matched the criteria for Subset 2. Further, 25 subjects matched
the criteria for both subsets (and thus, were included in both).</p>
</div></blockquote>
</li>
<li><p class="first">Tables and charts that show how the subjects who match the criteria
fit into age, sex, and race demographics. Example (showing the age
portion only):</p>
</li>
</ul>
<p><a class="reference internal" href="_images/image62.png"><img alt="image79" src="_images/image62.png" style="width: 6.26250in; height: 2.82188in;" /></a></p>
<ul class="simple">
<li>Analyses of the concepts you added to the subsets from the navigation
tree. Example (showing the weight concept):</li>
</ul>
<p><a class="reference internal" href="_images/image63.png"><img alt="image80" src="_images/image63.png" style="width: 6.00000in; height: 2.42756in;" /></a></p>
<div class="section" id="significance-tests">
<h5>Significance Tests<a class="headerlink" href="#significance-tests" title="Permalink to this headline">¶</a></h5>
<p>The above figure includes the results of significance testing that
Dataset Explorer performs:</p>
<p><a class="reference internal" href="_images/image64.png"><img alt="image81" src="_images/image64.png" style="width: 2.82292in; height: 0.73958in;" /></a></p>
<p>Significance testing is designed to indicate whether the reliability of
the statistics is 95% or greater, based on p-value.</p>
<p>Dataset Explorer calculates the significance result using either t-test
or chi-squared statistics to determine the p-value:</p>
<ul class="simple">
<li>For continuous variables (for example, subject weight or age), a
t-test compares the observed values in the two subsets.</li>
</ul>
<p>tranSMART uses the following Java method to calculate the t-test
statistic:</p>
<blockquote>
<div><a class="reference external" href="http://commons.apache.org/proper/commons-math/apidocs/org/apache/commons/math3/stat/inference/TTest.html#tTest(double[],double[])">http://commons.apache.org/proper/commons-math/apidocs/org/apache/commons/math3/stat/inference/TTest.html#tTest(double[],
double[])</a></div></blockquote>
<ul class="simple">
<li>For categorical values (for example, diagnoses), a chi-squared test
compares the counts in the two subsets.</li>
</ul>
<p>tranSMART uses the following Java method to calculate the chi-squared
statistic:</p>
<blockquote>
<div><a class="reference external" href="http://commons.apache.org/proper/commons-math/apidocs/org/apache/commons/math3/stat/inference/ChiSquareTest.html#chiSquareTest(long[][])">http://commons.apache.org/proper/commons-math/apidocs/org/apache/commons/math3/stat/inference/ChiSquareTest.html#chiSquareTest(long[][])</a></div></blockquote>
<p>If there is not enough data to calculate a test, Dataset Explorer
displays a message indicating the insufficient quantity data. Also,
significance test results are not displayed in the following
circumstances:</p>
<ul class="simple">
<li>If two identical subsets are defined. In this case, the significance
test results are not meaningful.</li>
<li>If all subjects in the first subset have one set of values for the
categorical value, and all subjects in the second subset have other
categorical values. For example, suppose you set Subset 1 to contain
only males and Subset 2 to contain only females. Also, suppose that
Subset 1 has 15 subjects and Subset 2 has 20. If you then try to show
statistics by gender, a table like the following would result:</li>
</ul>
<table border="1" class="docutils">
<colgroup>
<col width="29%" />
<col width="35%" />
<col width="35%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head">&nbsp;</th>
<th class="head">Subset 1</th>
<th class="head">Subset 2</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>Female</td>
<td>0</td>
<td>20</td>
</tr>
<tr class="row-odd"><td>Male</td>
<td>15</td>
<td>0</td>
</tr>
</tbody>
</table>
<p>In this case, the chi-squared function doesn’t return meaningful
results.</p>
</div>
</div>
<div class="section" id="defining-points-of-comparison">
<h4>Defining Points of Comparison<a class="headerlink" href="#defining-points-of-comparison" title="Permalink to this headline">¶</a></h4>
<p>Once you establish the subsets of subjects that you want to compare, you
can apply one or more points of comparison to the subsets.</p>
<p>A point of comparison is a concept in the navigation tree.</p>
<ol class="arabic simple">
<li>To apply a point of comparison to the subsets:<ol class="arabic">
<li>You must already have defined the subsets and have generated
summary statistics for the subsets, as described in the previous
section.</li>
<li>Drag the concept that you want to introduce as the point of
comparison from the navigation tree, and drop it anywhere in the
Results/Analysis view.</li>
</ol>
</li>
</ol>
<p>As soon as you drop the point of comparison into the Results/Analysis
view, tranSMART begins to compare the subsets based on that point of
comparison. When finished, tranSMART displays a side-by-side summary of
how the subjects in each subset match or respond to the point of
comparison.</p>
<div class="section" id="results-of-a-comparison">
<h5>Results of a Comparison<a class="headerlink" href="#results-of-a-comparison" title="Permalink to this headline">¶</a></h5>
<p>In a comparison of subjects in a psychological study, suppose Subset 1
contains subjects with a substance abuse problem, and Subset 2 contains
subjects with no substance abuse assessment.</p>
<p>After the subsets are defined and summary statistics are generated, a
diagnosis of depression is dropped into the Results/Analysis view as a
point of comparison. tranSMART displays a side-by-side comparison of the
subjects in each subset, indicating that almost all the subjects with a
substance abuse problem have been diagnosed with depression, while that
diagnosis for those with no substance abuse problem is more evenly
split.</p>
<p>The comparison is placed at the top of the Results/Analysis view, above
the demographic definitions plus any other earlier comparisons:</p>
<p><a class="reference internal" href="_images/image65.png"><img alt="image82" src="_images/image65.png" style="width: 6.00000in; height: 1.92051in;" /></a></p>
<table border="1" class="docutils">
<colgroup>
<col width="8%" />
<col width="92%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image83" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">To keep the size of the preceding figure within production limits, the demographics (age, sex, and race) portions of the figure have been excluded.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="section" id="printing-or-saving-the-contents-of-the-results-analysis-view">
<h4>Printing or Saving the Contents of the Results/Analysis View<a class="headerlink" href="#printing-or-saving-the-contents-of-the-results-analysis-view" title="Permalink to this headline">¶</a></h4>
<ol class="arabic simple">
<li><ol class="first arabic">
<li>With the Results/Analysis view displayed, click <strong>Print</strong>:</li>
</ol>
</li>
</ol>
<p><a class="reference internal" href="_images/image66.png"><img alt="image84" src="_images/image66.png" style="width: 1.47917in; height: 0.38542in;" /></a></p>
<p>The entire contents of the Results/Analysis view appear in a separate
browser window.</p>
<ol class="arabic simple">
<li>Click one of the following buttons at the top of the browser window:</li>
</ol>
<p><a class="reference internal" href="_images/image67.png"><img alt="image85" src="_images/image67.png" style="width: 2.01042in; height: 0.39375in;" /></a></p>
</div>
<div class="section" id="copying-individual-charts-in-the-results-analysis-view">
<h4>Copying Individual Charts in the Results/Analysis View<a class="headerlink" href="#copying-individual-charts-in-the-results-analysis-view" title="Permalink to this headline">¶</a></h4>
<p>If you are interested in a particular chart in the Results/Analysis
View, you can copy the chart to a file, as follows:</p>
<ol class="arabic simple">
<li><ol class="first arabic">
<li>With the Results/Analysis view displayed, click <strong>Print</strong>.</li>
</ol>
</li>
</ol>
<p>The entire contents of the Results/Analysis view appear in a separate
browser window.</p>
<ol class="arabic simple">
<li>Right-click the chart to copy.</li>
<li>In the browser popup menu, click <strong>Save Image As</strong>.</li>
<li>In the Save Picture dialog, specify the name, location, and the file
type for the chart.</li>
<li>Click <strong>Save</strong>.</li>
</ol>
</div>
</div>
<div class="section" id="viewing-a-study">
<h3>Viewing a Study<a class="headerlink" href="#viewing-a-study" title="Permalink to this headline">¶</a></h3>
<p>You can view a description of any Dataset Explorer study, whether or not
you have access rights to the study.</p>
<ol class="arabic simple">
<li>To view a description of a study:<ol class="arabic">
<li>In Dataset Explorer, click the <strong>Navigate Terms</strong> tab.</li>
<li>Open the top-level node for the list of studies you are interested
in – for example, click the <strong>+</strong> icon (<a class="reference internal" href="_images/image29.png"><img alt="image86" src="_images/image29.png" style="width: 0.11761in;" /></a>) next to Public
Studies to open the list of experiments:</li>
</ol>
</li>
</ol>
<p><a class="reference internal" href="_images/image68.png"><img alt="image87" src="_images/image68.png" style="width: 1.81679in; height: 0.65333in;" /></a></p>
<ol class="arabic simple">
<li>Right-click the particular study you are interested in.</li>
<li>Click the <strong>Show Definition</strong> popup:</li>
</ol>
<p><a class="reference internal" href="_images/image69.png"><img alt="image88" src="_images/image69.png" style="width: 2.68702in; height: 1.51483in;" /></a></p>
<p>The Show Concept Definition dialog appears, showing the title,
description, and other information about the study.</p>
</div>
<div class="section" id="exporting-dataset-explorer-findings">
<h3>Exporting Dataset Explorer Findings<a class="headerlink" href="#exporting-dataset-explorer-findings" title="Permalink to this headline">¶</a></h3>
<p>The Data Export tab allows you to export your data locally for further
analysis in several different formats. Exporting data using this tool
involves the following high-level tasks:</p>
<p><a class="reference internal" href="_images/image70.png"><img alt="image89" src="_images/image70.png" style="width: 4.48958in; height: 1.85071in;" /></a></p>
<p>Supported file formats include:</p>
<ul class="simple">
<li>Clinical and low dimensional biomarker data</li>
<li>Gene expression data</li>
<li>SNP data</li>
<li>Gene set enrichment analysis (GSEA)</li>
</ul>
<table border="1" class="docutils">
<colgroup>
<col width="9%" />
<col width="91%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image90" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">For more information on GSEA data files, visit the following site: <em>http://www.broadinstitute.org/cancer/software/gsea/wiki/index.php/
Data_formats</em></th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<ol class="arabic">
<li><p class="first">To export Dataset Explorer findings to your local machine:</p>
<ol class="arabic">
<li><p class="first">Click the tranSMART <strong>Dataset Explorer</strong> tab to display the
Dataset Explorer window.</p>
</li>
<li><p class="first">In the left pane of the Dataset Explorer window, click the
<strong>Navigate Terms</strong> tab.</p>
<p>The navigation tree appears, showing the categories of available
studies:</p>
<p><a class="reference internal" href="_images/image71.png"><img alt="image91" src="_images/image71.png" style="width: 2.42537in; height: 0.68559in;" /></a></p>
</li>
<li><p class="first">Select the study of interest.</p>
</li>
<li><p class="first">Define the cohorts whose data points are of interest.</p>
<p>Now that the subsets are defined, you are ready to export data
from the study that applies to the subsets.</p>
</li>
<li><p class="first">Click the <strong>Data Export</strong> tab:</p>
<p><a class="reference internal" href="_images/image72.png"><img alt="image92" src="_images/image72.png" style="width: 5.59305in; height: 0.34371in;" /></a></p>
<p>The Data Export page appears with your selected cohorts:</p>
<p><a class="reference internal" href="_images/image73.png"><img alt="image93" src="_images/image73.png" style="width: 6.00000in; height: 2.95513in;" /></a></p>
</li>
<li><p class="first">Select the check boxes to indicate the data types and file formats
that are desired for export.</p>
</li>
<li><p class="first">Click <strong>Export Data</strong> at the bottom of the tranSMART browser
window.</p>
<p>The command will now start a job. You may choose to have the job
run in the background in order to continue with other analyses and
cohort selection while the job completes. The job could take
several minutes depending on the amount of data selected.</p>
</li>
</ol>
<ol class="arabic">
<li><p class="first">Click the <strong>Export Jobs</strong> tab to access completed jobs or to check
the status of a pending job.</p>
<p>Jobs follow the naming convention <em>User - Type of Job Run - Job
ID.</em></p>
</li>
</ol>
<ol class="arabic">
<li><p class="first">Click the name of the job you processed:</p>
<p><a class="reference internal" href="_images/image74.png"><img alt="image94" src="_images/image74.png" style="width: 6.00000in; height: 1.13194in;" /></a></p>
<p>The Open File dialog box appears:</p>
<p><a class="reference internal" href="_images/image75.png"><img alt="image95" src="_images/image75.png" style="width: 2.16000in; height: 1.55453in;" /></a></p>
</li>
<li><p class="first">Select <strong>Save File</strong>, then click <strong>OK</strong>.</p>
<p>Your file will be sent to the <strong>Downloads</strong> folder on your local
machine in a .zip file. The .zip file contains separate folders
for subsets, clinical data, gene expression data, and other
factors you may have specified during cohort selection.</p>
</li>
</ol>
</li>
</ol>
</div>
<div class="section" id="generating-advanced-analyses-and-visualizations">
<h3>Generating Advanced Analyses and Visualizations<a class="headerlink" href="#generating-advanced-analyses-and-visualizations" title="Permalink to this headline">¶</a></h3>
<p>Advanced analyses and visualizations offered with tranSMART allow a user
to produce the following within Dataset Explorer:</p>
<ul class="simple">
<li>Heatmaps<ul>
<li><em>Standard Heatmap</em> (page 49)</li>
<li><em>Hierarchical Clustering</em> (page 51)</li>
<li><em>K-Means Clustering</em> (page 54)</li>
<li><em>Marker Selection</em> (page 56)</li>
</ul>
</li>
<li>Advanced Analyses<ul>
<li><em>Box Plot with ANOVA</em> (page 59)</li>
<li><em>Principal Component Analysis</em> (page 61)</li>
<li><em>Scatter Plot with Linear Regression</em> (page 64)</li>
<li><em>Survival Analysis</em> (page 66)</li>
<li><em>Table with Fisher Test Analysis</em> (page 69)</li>
</ul>
</li>
</ul>
<p>Dataset Explorer uses the R software environment for statistical
computing and to generate analyses and visualizations. For more
information, visit <a class="reference external" href="http://www.r-project.org">http://www.r-project.org</a>.</p>
<div class="section" id="generating-heatmaps">
<h4>Generating Heatmaps<a class="headerlink" href="#generating-heatmaps" title="Permalink to this headline">¶</a></h4>
<p>In Dataset Explorer, a heatmap is a matrix of data points for a
particular set of biomarkers, such as genes, at a particular point in
time and/or for a particular tissue sample in the study, as measured for
each subject in the study.</p>
<p>In a Dataset Explorer heatmap, the biomarkers appear in the y axis, and
the subjects appear in the x axis.</p>
<table border="1" class="docutils">
<colgroup>
<col width="18%" />
<col width="82%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image96" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">A heatmap can display data points for up to 1000 samples.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<p>Dataset Explorer uses the R software environment for statistical
computing and to generate analyses and visualizations. For more
information, visit <a class="reference external" href="http://www.r-project.org">http://www.r-project.org</a>.</p>
<p>You can generate the following types of heatmaps:</p>
<ul class="simple">
<li><em>Standard Heatmap</em> (below)</li>
<li><em>Hierarchical Clustering</em> (page 51)</li>
<li><em>K-Means Clustering</em> (page 54)</li>
<li><em>Marker Selection</em> (page 56)</li>
</ul>
<div class="section" id="standard-heatmap">
<h5>Standard Heatmap<a class="headerlink" href="#standard-heatmap" title="Permalink to this headline">¶</a></h5>
<p>A standard heatmap is a visualization of biomarker data points with no
indication of patterns, groupings, or differentiation among the data
points.</p>
<ol class="arabic simple">
<li>To generate a standard heatmap:<ol class="arabic">
<li>Run tranSMART, then click the <strong>Dataset Explorer</strong> tab.</li>
<li>Define the cohorts you wish to analyze by dragging one or more
concepts from a study into empty subset definition boxes. For more
information, see <em>Populating the Study Groups</em> on page 35.</li>
</ol>
</li>
</ol>
<table border="1" class="docutils">
<colgroup>
<col width="12%" />
<col width="88%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image97" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">To compare two subsets, you may drag an additional concept into the Subset 2 comparison box.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<ol class="arabic">
<li><p class="first">Click the <strong>Advanced Workflow</strong> tab:</p>
<p><a class="reference internal" href="_images/image76.png"><img alt="image98" src="_images/image76.png" style="width: 5.60000in; height: 0.61597in;" /></a></p>
</li>
<li><p class="first">Select <strong>Heatmap</strong> from the <strong>Analysis</strong> dropdown menu:</p>
<p><a class="reference internal" href="_images/image77.png"><img alt="image99" src="_images/image77.png" style="width: 1.67568in; height: 2.20080in;" /></a></p>
<p>The Variable Selection section appears.</p>
</li>
<li><p class="first">Define the heatmap variable by selecting a high dimensional data node
from the Dataset Explorer tree and dragging it into the Heatmap
Variable definition box:</p>
<p><a class="reference internal" href="_images/image78.png"><img alt="image100" src="_images/image78.png" style="width: 3.53153in; height: 2.70086in;" /></a></p>
</li>
</ol>
<table border="1" class="docutils">
<colgroup>
<col width="13%" />
<col width="87%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image101" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">High dimensional data nodes are indicated by the icon (<a class="reference internal" href="_images/image79.png"><img alt="image102" src="_images/image79.png" style="width: 0.13540in;" /></a>) to the left of study data.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<ol class="arabic">
<li><p class="first">Click the <strong>High Dimensional Data</strong> button.</p>
<p>The Compare Subsets-Pathway Selection dialog appears.</p>
</li>
<li><p class="first">Specify the platform and other factors of interest.</p>
<p>For more information, see <em>High Dimensional Data</em> on page 75.</p>
</li>
<li><p class="first">Click <strong>Apply Selections</strong>.</p>
</li>
<li><p class="first">Click <strong>Run</strong>.</p>
<p>Your analysis appears below:</p>
<p><a class="reference internal" href="_images/image80.png"><img alt="image103" src="_images/image80.png" style="width: 6.54167in; height: 5.06845in;" /></a></p>
</li>
</ol>
</div>
<div class="section" id="hierarchical-clustering">
<h5>Hierarchical Clustering<a class="headerlink" href="#hierarchical-clustering" title="Permalink to this headline">¶</a></h5>
<p>Hierarchical clustering is a visualization of patterns of related data
points in gene expression data.</p>
<ol class="arabic simple">
<li>To generate a hierarchical clustering heatmap:<ol class="arabic">
<li>Run tranSMART, then click the <strong>Dataset Explorer</strong> tab.</li>
<li>Define the cohorts you wish to analyze by dragging one or more
concepts from a study into empty subset definition boxes. For more
information, see <em>Populating the Study Groups</em> on page 35.</li>
</ol>
</li>
</ol>
<table border="1" class="docutils">
<colgroup>
<col width="13%" />
<col width="87%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image104" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">To compare two subsets, you may drag an additional concept into the Subset 2 comparison box.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<ol class="arabic">
<li><p class="first">Click the <strong>Advanced Workflow</strong> tab:</p>
<p><a class="reference internal" href="_images/image76.png"><img alt="image105" src="_images/image76.png" style="width: 5.60000in; height: 0.61597in;" /></a></p>
</li>
<li><p class="first">Select <strong>Hierarchical Clustering</strong> from the <strong>Analysis</strong> dropdown
menu:</p>
<p><a class="reference internal" href="_images/image81.png"><img alt="image106" src="_images/image81.png" style="width: 1.73874in; height: 2.34523in;" /></a></p>
<p>The Variable Selection section appears.</p>
</li>
<li><p class="first">Define the heatmap variable by selecting a high dimensional data node
from the Dataset Explorer tree and dragging it into the Heatmap
Variable definition box:</p>
<p><a class="reference internal" href="_images/image82.png"><img alt="image107" src="_images/image82.png" style="width: 2.65484in; height: 2.22608in;" /></a></p>
</li>
</ol>
<table border="1" class="docutils">
<colgroup>
<col width="13%" />
<col width="87%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image108" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">High dimensional data nodes are indicated by the icon (<a class="reference internal" href="_images/image79.png"><img alt="image109" src="_images/image79.png" style="width: 0.13540in;" /></a>) to the left of study data.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<ol class="arabic">
<li><p class="first">Click the <strong>High Dimensional Data</strong> button.</p>
<p>The Compare Subsets-Pathway Selection dialog appears.</p>
</li>
<li><p class="first">Specify the platform and other factors of interest.</p>
<p>For more information, see <em>High Dimensional Data</em> on page 75.</p>
</li>
<li><p class="first">Click <strong>Apply Selections</strong>.</p>
</li>
<li><p class="first">Click <strong>Run</strong>.</p>
<p>Your analysis appears below:</p>
<p><a class="reference internal" href="_images/image83.png"><img alt="image110" src="_images/image83.png" style="width: 5.98958in; height: 4.66667in;" /></a></p>
</li>
</ol>
<table border="1" class="docutils">
<colgroup>
<col width="12%" />
<col width="88%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image111" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">To read more about Hierarchical Clustering, visit: <a class="reference external" href="http://www.ics.uci.edu/~eppstein/280/cluster.html">http://www.ics.uci.edu/~eppstein/280/cluster.html</a></th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
</div>
<div class="section" id="k-means-clustering">
<h5>K-Means Clustering<a class="headerlink" href="#k-means-clustering" title="Permalink to this headline">¶</a></h5>
<p>K-Means clustering is a visualization of groupings of the most closely
related data points, based on the number of groupings you specify.</p>
<table border="1" class="docutils">
<colgroup>
<col width="16%" />
<col width="84%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image112" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">The K-Means analysis clusters columns together – rows are not clustered.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<ol class="arabic simple">
<li>To generate a k-means clustering heatmap:<ol class="arabic">
<li>Run tranSMART, then click the <strong>Dataset Explorer</strong> tab.</li>
<li>Define the cohorts you wish to analyze by dragging one or more
concepts from a study into empty subset definition boxes. For more
information, see <em>Populating the Study Groups</em> on page 35.</li>
</ol>
</li>
</ol>
<table border="1" class="docutils">
<colgroup>
<col width="13%" />
<col width="87%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image113" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">To compare two subsets, you may drag an additional concept into the Subset 2 comparison box.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<ol class="arabic">
<li><p class="first">Click the <strong>Advanced Workflow</strong> tab:</p>
<p><a class="reference internal" href="_images/image76.png"><img alt="image114" src="_images/image76.png" style="width: 5.25600in; height: 0.57813in;" /></a></p>
</li>
<li><p class="first">Select <strong>K-Means Clustering</strong> from the <strong>Analysis</strong> dropdown menu:</p>
<p><a class="reference internal" href="_images/image84.png"><img alt="image115" src="_images/image84.png" style="width: 1.52252in; height: 2.00103in;" /></a></p>
<p>The Variable Selection section appears.</p>
</li>
<li><p class="first">Define the heatmap variable by selecting a high dimensional data node
from the Dataset Explorer tree and dragging it into the Heatmap
Variable definition box:</p>
<p><a class="reference internal" href="_images/image85.png"><img alt="image116" src="_images/image85.png" style="width: 2.87247in; height: 2.77477in;" /></a></p>
</li>
</ol>
<table border="1" class="docutils">
<colgroup>
<col width="13%" />
<col width="87%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image117" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">High dimensional data nodes are indicated by the icon (<a class="reference internal" href="_images/image79.png"><img alt="image118" src="_images/image79.png" style="width: 0.13540in;" /></a>) to the left of study data.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<ol class="arabic">
<li><p class="first">Click the <strong>High Dimensional Data</strong> button.</p>
<p>The Compare Subsets-Pathway Selection dialog appears.</p>
</li>
<li><p class="first">Specify the platform and other factors of interest.</p>
<p>For more information, see <em>High Dimensional Data</em> on page 75.</p>
</li>
<li><p class="first">Click <strong>Apply Selections</strong>.</p>
</li>
<li><p class="first">In the <strong>Number of clusters</strong> field, type a numerical value.</p>
</li>
<li><p class="first">Click <strong>Run</strong>.</p>
<p>Your analysis appears below:</p>
<p><a class="reference internal" href="_images/image86.png"><img alt="image119" src="_images/image86.png" style="width: 5.98958in; height: 4.60417in;" /></a></p>
</li>
</ol>
<table border="1" class="docutils">
<colgroup>
<col width="12%" />
<col width="88%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image120" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">To read more about K-Means Clustering, visit: <a class="reference external" href="http://www.ics.uci.edu/~eppstein/280/cluster.html">http://www.ics.uci.edu/~eppstein/280/cluster.html</a></th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
</div>
<div class="section" id="marker-selection">
<h5>Marker Selection<a class="headerlink" href="#marker-selection" title="Permalink to this headline">¶</a></h5>
<p>A marker selection heatmap is a visualization of differentially
expressed genes in distinct phenotypes. Specifically, the algorithm
determines the set of genes which is most differently expressed between
the two subsets. This list of differentially expressed genes is
subsequently presented in a table, along with a variety of accompanying
statistics.</p>
<ol class="arabic simple">
<li>To generate a marker selection heatmap:<ol class="arabic">
<li>Run tranSMART, then click the <strong>Dataset Explorer</strong> tab.</li>
<li>Define the cohorts you wish to analyze by dragging one or more
concepts from a study into empty subset definition boxes. For more
information, see <em>Populating the Study Groups</em> on page 35.</li>
</ol>
</li>
</ol>
<table border="1" class="docutils">
<colgroup>
<col width="16%" />
<col width="84%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image87.png"><img alt="image121" src="_images/image87.png" style="width: 0.31246in; height: 0.31246in;" /></a></th>
<th class="head">Two subsets must be specified when using a Marker Selection heatmap.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<ol class="arabic">
<li><p class="first">Click the <strong>Advanced Workflow</strong> tab:</p>
<p><a class="reference internal" href="_images/image76.png"><img alt="image122" src="_images/image76.png" style="width: 5.60000in; height: 0.61597in;" /></a></p>
</li>
<li><p class="first">Select <strong>Marker Selection</strong> from the <strong>Analysis</strong> dropdown menu:</p>
<p><a class="reference internal" href="_images/image88.png"><img alt="image123" src="_images/image88.png" style="width: 1.66667in; height: 2.15962in;" /></a></p>
<p>The Variable Selection section appears.</p>
</li>
<li><p class="first">Define the required variable by selecting a high dimensional data
node from the Dataset Explorer tree and dragging it into the Marker
Variable definition box:</p>
<p><a class="reference internal" href="_images/image89.png"><img alt="image124" src="_images/image89.png" style="width: 2.85737in; height: 2.53153in;" /></a></p>
</li>
</ol>
<table border="1" class="docutils">
<colgroup>
<col width="13%" />
<col width="87%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image125" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">High dimensional data nodes are indicated by the icon (<a class="reference internal" href="_images/image79.png"><img alt="image126" src="_images/image79.png" style="width: 0.13540in;" /></a>) to the left of study data.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<ol class="arabic">
<li><p class="first">Click the <strong>High Dimensional Data</strong> button.</p>
<p>The Compare Subsets-Pathway Selection dialog appears.</p>
</li>
<li><p class="first">Specify the platform and other factors of interest.</p>
<p>For more information, see <em>High Dimensional Data</em> on page 75.</p>
</li>
<li><p class="first">Click <strong>Apply Selections</strong>.</p>
<p>In the <strong>Number of Markers</strong> field, type a numerical value. This will
determine the number of differentially expressed genes that are
returned.</p>
</li>
<li><p class="first">Click <strong>Run</strong>.</p>
<p>Your analysis appears below:</p>
<p><a class="reference internal" href="_images/image90.png"><img alt="image127" src="_images/image90.png" style="width: 6.00000in; height: 4.09375in;" /></a></p>
</li>
</ol>
<table border="1" class="docutils">
<colgroup>
<col width="10%" />
<col width="90%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image128" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">For more information on the analyses used in Marker Selection, visit: <a class="reference external" href="http://mathworld.wolfram.com/BonferroniCorrection.html">http://mathworld.wolfram.com/BonferroniCorrection.html</a></th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="section" id="generating-advanced-analyses">
<h4>Generating Advanced Analyses<a class="headerlink" href="#generating-advanced-analyses" title="Permalink to this headline">¶</a></h4>
<p>Advanced analyses include:</p>
<ul class="simple">
<li><em>Box Plot with ANOVA</em> (page 59)</li>
<li><em>Principal Component Analysis</em> (page 61)</li>
<li><em>Scatter Plot with Linear Regression</em> (page 64)</li>
<li><em>Survival Analysis</em> (page 66)</li>
<li><em>Table with Fisher Test Analysis</em> (page 69)</li>
</ul>
<div class="section" id="box-plot-with-anova">
<h5>Box Plot with ANOVA<a class="headerlink" href="#box-plot-with-anova" title="Permalink to this headline">¶</a></h5>
<p>A box plot with ANOVA analysis displays a box and whisker plot with
corresponding analysis of variance in the sample(s).</p>
<ol class="arabic simple">
<li>To perform a box plot with ANOVA analysis:<ol class="arabic">
<li>Run tranSMART, then click the <strong>Dataset Explorer</strong> tab.</li>
<li>Define the cohorts you wish to analyze by dragging one or more
concepts from a study into empty subset definition boxes. For more
information, see <em>Populating the Study Groups</em> on page 35.</li>
</ol>
</li>
</ol>
<table border="1" class="docutils">
<colgroup>
<col width="13%" />
<col width="87%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image129" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">Only one subset may be specified in this analysis. Information in Subset 2 will be ignored.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<ol class="arabic">
<li><p class="first">Click the <strong>Advanced Workflow</strong> tab:</p>
<p><a class="reference internal" href="_images/image76.png"><img alt="image130" src="_images/image76.png" style="width: 5.60000in; height: 0.61597in;" /></a></p>
</li>
<li><p class="first">Select <strong>Box Plot with ANOVA</strong> from the <strong>Analysis</strong> dropdown menu:</p>
<p><a class="reference internal" href="_images/image91.png"><img alt="image131" src="_images/image91.png" style="width: 1.65766in; height: 2.18061in;" /></a></p>
<p>The Variable Selection section appears. You will need to define what
variables in the study are independent, and what variables are
dependent. At least one of the variables should be continuous (for
example, Age), and one should be a categorical value (for example,
Tissue Type).</p>
</li>
</ol>
<table border="1" class="docutils">
<colgroup>
<col width="8%" />
<col width="92%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image132" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">If the <em>independent variable</em> defines the groups, boxes will be plotted horizontally. If the <em>dependent variable</em> defines the groups, boxes will be plotted vertically.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<ol class="arabic simple">
<li>Define the variables.</li>
</ol>
<table border="1" class="docutils">
<colgroup>
<col width="4%" />
<col width="96%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image133" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">In this example, the data binning feature is not used. For future reference, data binning refers to a pre-processing technique used to reduce minor observation errors. Clusters of data are replaced by a value representative of that cluster (the central value). For information on binning, see <em>Data Binning</em> on page 70.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<ol class="arabic">
<li><p class="first">Click <strong>Run</strong>.</p>
<p>Your analysis appears below:</p>
<p><a class="reference internal" href="_images/image92.png"><img alt="image134" src="_images/image92.png" style="width: 4.23803in; height: 8.37219in;" /></a></p>
</li>
</ol>
</div>
<div class="section" id="principal-component-analysis">
<h5>Principal Component Analysis<a class="headerlink" href="#principal-component-analysis" title="Permalink to this headline">¶</a></h5>
<p>In a principal component analysis (PCA), the total number of variables
in the dataset is reduced to a smaller number of variables – the
principle components of the dataset.</p>
<p>Principal component variables are calculated from correlated variables
in the total dataset. In other words, the principal component analysis
is a workflow used to identify variance in a dataset. The analysis can
be run on an entire microarray chip, or on a pathway.</p>
<ol class="arabic simple">
<li>To perform a principal component analysis:<ol class="arabic">
<li>Run tranSMART, then click the <strong>Dataset Explorer</strong> tab.</li>
<li>Define the cohorts you wish to analyze by dragging one or more
concepts from a study into empty subset definition boxes. For more
information, see <em>Populating the Study Groups</em> on page 35.</li>
</ol>
</li>
</ol>
<table border="1" class="docutils">
<colgroup>
<col width="13%" />
<col width="87%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image135" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">Only one subset may be specified in this analysis. Information in Subset 2 will be ignored.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<ol class="arabic">
<li><p class="first">Click the <strong>Advanced Workflow</strong> tab:</p>
<p><a class="reference internal" href="_images/image76.png"><img alt="image136" src="_images/image76.png" style="width: 5.25600in; height: 0.57813in;" /></a></p>
</li>
<li><p class="first">Select <strong>PCA</strong> from the <strong>Analysis</strong> dropdown menu:</p>
<p><a class="reference internal" href="_images/image93.png"><img alt="image137" src="_images/image93.png" style="width: 1.51351in; height: 2.00651in;" /></a></p>
<p>The Variable Selection section appears.</p>
</li>
<li><p class="first">Define the heatmap variable by selecting a high dimensional data node
from the Dataset Explorer tree and dragging it into the PCA Variable
definition box:</p>
<p><a class="reference internal" href="_images/image94.png"><img alt="image138" src="_images/image94.png" style="width: 2.81700in; height: 2.68468in;" /></a></p>
</li>
</ol>
<table border="1" class="docutils">
<colgroup>
<col width="12%" />
<col width="88%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image139" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">High dimensional data nodes are indicated by the ( <a class="reference internal" href="_images/image79.png"><img alt="image140" src="_images/image79.png" style="width: 0.13540in;" /></a> ) icon to the left of study data.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<ol class="arabic">
<li><p class="first">Click the <strong>High Dimensional Data</strong> button.</p>
<p>The Compare Subsets-Pathway Selection dialog appears.</p>
</li>
<li><p class="first">Specify the platform and other factors of interest.</p>
<p>For more information, see <em>High Dimensional Data</em> on page 75.</p>
</li>
<li><p class="first">Click <strong>Apply Selections</strong>.</p>
</li>
<li><p class="first">Click <strong>Run</strong>.</p>
<p>Your analysis appears below:</p>
<p><a class="reference internal" href="_images/image95.png"><img alt="image141" src="_images/image95.png" style="width: 4.03543in; height: 7.16418in;" /></a></p>
</li>
</ol>
<table border="1" class="docutils">
<colgroup>
<col width="11%" />
<col width="89%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image142" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">For more information regarding PCAs, see: <a class="reference external" href="http://psb.stanford.edu/psb-online/proceedings/psb00/raychaudhuri.pdf">http://psb.stanford.edu/psb-online/proceedings/psb00/raychaudhuri.pdf</a>.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
</div>
<div class="section" id="scatter-plot-with-linear-regression">
<h5>Scatter Plot with Linear Regression<a class="headerlink" href="#scatter-plot-with-linear-regression" title="Permalink to this headline">¶</a></h5>
<p>A scatter plot displays values for two variables within a dataset, with
a line that best fits the slope of the data.</p>
<ol class="arabic simple">
<li>To perform a scatter plot with linear regression analysis:<ol class="arabic">
<li>Run tranSMART, then click the <strong>Dataset Explorer</strong> tab.</li>
<li>Define the cohorts you wish to analyze by dragging one or more
concepts from a study into empty subset definition boxes. For more
information, see <em>Populating the Study Groups</em> on page 35.</li>
</ol>
</li>
</ol>
<table border="1" class="docutils">
<colgroup>
<col width="13%" />
<col width="87%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image143" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">Only one subset may be specified in this analysis. Information in Subset 2 will be ignored.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<ol class="arabic">
<li><p class="first">Click the <strong>Advanced Workflow</strong> tab above Subset 1:</p>
<p><a class="reference internal" href="_images/image76.png"><img alt="image144" src="_images/image76.png" style="width: 5.25600in; height: 0.57813in;" /></a></p>
</li>
<li><p class="first">Select <strong>Scatter Plot with Linear Regression</strong> from the <strong>Analysis</strong>
dropdown menu:</p>
<p><a class="reference internal" href="_images/image96.png"><img alt="image145" src="_images/image96.png" style="width: 1.64865in; height: 2.17430in;" /></a></p>
<p>The Variable Selection section appears. You will need to define what
variables in the study are independent, and what variables are
dependent. Both variables should be continuous (for example, Age).</p>
</li>
<li><p class="first">Define the variables.</p>
</li>
<li><p class="first">Click <strong>Run</strong>.</p>
<p>Your analysis appears below:</p>
<p><a class="reference internal" href="_images/image97.png"><img alt="image146" src="_images/image97.png" style="width: 4.63484in; height: 6.70750in;" /></a></p>
</li>
</ol>
</div>
<div class="section" id="survival-analysis">
<h5>Survival Analysis<a class="headerlink" href="#survival-analysis" title="Permalink to this headline">¶</a></h5>
<p>A survival analysis displays time-to-event data.</p>
<ol class="arabic">
<li><p class="first">To perform a survival analysis:</p>
<ol class="arabic">
<li><p class="first">Run tranSMART, then click the <strong>Dataset Explorer</strong> tab.</p>
</li>
<li><p class="first">Define the cohorts you wish to analyze by dragging one or more
concepts from a study into empty subset definition boxes. For more
information, see <em>Populating the Study Groups</em> on page 35.</p>
</li>
<li><p class="first">Click the <strong>Advanced Workflow</strong> tab above Subset 1:</p>
<p><a class="reference internal" href="_images/image76.png"><img alt="image147" src="_images/image76.png" style="width: 5.25600in; height: 0.57813in;" /></a></p>
</li>
<li><p class="first">Select <strong>Survival Analysis</strong> from the <strong>Analysis</strong> dropdown menu:</p>
<p><a class="reference internal" href="_images/image98.png"><img alt="image148" src="_images/image98.png" style="width: 1.65380in; height: 2.16216in;" /></a></p>
<p>The Variable Selection section appears.</p>
</li>
<li><p class="first">Define the following variables:</p>
</li>
</ol>
</li>
</ol>
<table border="1" class="docutils">
<colgroup>
<col width="7%" />
<col width="5%" />
<col width="76%" />
<col width="12%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head">Variable</th>
<th class="head">Required?</th>
<th class="head">Definition</th>
<th class="head">Example</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>Time</td>
<td>Yes</td>
<td>A numeric field within tranSMART.</td>
<td><p class="first">Survival at Follow Up (Years)</p>
<p class="last"><a class="reference internal" href="_images/image99.png"><img alt="image149" src="_images/image99.png" style="width: 2.03200in; height: 0.48116in;" /></a></p>
</td>
</tr>
<tr class="row-odd"><td>Category</td>
<td>No</td>
<td><p class="first">A concept that is dragged into this input will dictate the groups into which the data will be split in order to compare their survival times.</p>
<p class="last">If this variable is continuous, it requires binning. For details, see <em>Data Binning Using Survival Analysis</em> on page 72.</p>
</td>
<td><p class="first">Cancer Stage</p>
<p class="last"><a class="reference internal" href="_images/image100.png"><img alt="image150" src="_images/image100.png" style="width: 1.44776in; height: 1.54428in;" /></a></p>
</td>
</tr>
<tr class="row-even"><td>Censoring Value</td>
<td>No</td>
<td>Specifies which patients had the event whose time is being measured. For example, if the Time variable selected is <strong>Overall Survival Time (Years)</strong>, an appropriate censoring variable is <strong>Patient Death</strong>.</td>
<td><p class="first">Dead</p>
<p class="last"><a class="reference internal" href="_images/image101.png"><img alt="image151" src="_images/image101.png" style="width: 1.79851in; height: 0.54705in;" /></a></p>
</td>
</tr>
</tbody>
</table>
<table border="1" class="docutils">
<colgroup>
<col width="4%" />
<col width="96%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image152" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">In this example, the data binning feature is not used. For future reference, data binning refers to a pre-processing technique used to reduce minor observation errors. Clusters of data are replaced by a value representative of that cluster (the central value). For information on data binning, see <em>Data Binning Using Survival Analysis</em> on page 72.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<ol class="arabic">
<li><p class="first">Click <strong>Run</strong>.</p>
<p>Your analysis appears below:</p>
<p><a class="reference internal" href="_images/image102.png"><img alt="image153" src="_images/image102.png" style="width: 6.00000in; height: 6.85064in;" /></a></p>
</li>
</ol>
</div>
<div class="section" id="table-with-fisher-test-analysis">
<h5>Table with Fisher Test Analysis<a class="headerlink" href="#table-with-fisher-test-analysis" title="Permalink to this headline">¶</a></h5>
<p>A Fisher Test analysis examines the significance of associated
variables.</p>
<ol class="arabic simple">
<li>To perform a table with fisher test analysis:<ol class="arabic">
<li>Run tranSMART, then click the <strong>Dataset Explorer</strong> tab.</li>
<li>Define the cohorts you wish to analyze by dragging one or more
concepts from a study into empty subset definition boxes. For more
information, see <em>Populating the Study Groups</em> on page 35.</li>
</ol>
</li>
</ol>
<table border="1" class="docutils">
<colgroup>
<col width="13%" />
<col width="87%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image154" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">Only one subset may be specified in this analysis. Information in Subset 2 will be ignored.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<ol class="arabic">
<li><p class="first">Click the <strong>Advanced Workflow</strong> tab:</p>
<p><a class="reference internal" href="_images/image76.png"><img alt="image155" src="_images/image76.png" style="width: 5.60000in; height: 0.61597in;" /></a></p>
</li>
<li><p class="first">Select <strong>Table with Fisher Test</strong> from the <strong>Analysis</strong> dropdown
menu:</p>
<p><a class="reference internal" href="_images/image103.png"><img alt="image156" src="_images/image103.png" style="width: 1.46847in; height: 1.92563in;" /></a></p>
<p>The Variable Selection section appears. You will need to define what
variables in the study are independent, and what variables are
dependent. Both variables should be categorical values (for example,
Tissue Type).</p>
</li>
</ol>
<table border="1" class="docutils">
<colgroup>
<col width="4%" />
<col width="96%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image157" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">In this example, the data binning feature is not used. For future reference, data binning refers to a pre-processing technique used to reduce minor observation errors. Clusters of data are replaced by a value representative of that cluster (the central value). For information on data binning, see <em>Data Binning</em> on page 70.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<ol class="arabic">
<li><p class="first">Define the variables.</p>
</li>
<li><p class="first">Click <strong>Run</strong>.</p>
<p>Your analysis appear below:</p>
<p><a class="reference internal" href="_images/image104.png"><img alt="image158" src="_images/image104.png" style="width: 4.30155in; height: 3.27042in;" /></a></p>
</li>
</ol>
</div>
</div>
<div class="section" id="data-binning">
<h4>Data Binning<a class="headerlink" href="#data-binning" title="Permalink to this headline">¶</a></h4>
<p>Data binning refers to a pre-processing technique used to reduce
observation errors and to allow continuous variables to become
categorical. Clusters of data are replaced by a value representative of
that cluster (the central value).</p>
<table border="1" class="docutils">
<colgroup>
<col width="5%" />
<col width="95%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image87.png"><img alt="image159" src="_images/image87.png" style="width: 0.31246in; height: 0.31246in;" /></a></th>
<th class="head">The data displayed after binning represents the data available in the study. If, for example, you have selected to bin based on date range (0-10 years of age), yet there is only data available for subjects eight years old and up, the bin will display the age range as 8-10.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<div class="section" id="data-binning-using-box-plot-with-anova">
<h5>Data Binning Using Box Plot with ANOVA<a class="headerlink" href="#data-binning-using-box-plot-with-anova" title="Permalink to this headline">¶</a></h5>
<p>When conducting a Box Plot with ANOVA analysis, at least one of the
variables selected should be a continuous variable (for example, age),
and the other should be a categorical value (for example, tumor stage).</p>
<p>A continuous variable can be viewed as a categorical value using the
binning feature, described below. Alternatively, binning can also be
used to categorize data. For example, if histological grade with values
such as Well Defined, Moderately Well Defined, and Poorly Defined are
selected, you can group Moderately Well Defined with Poorly Defined and
treat them as one group for the purposes of this analysis.</p>
<ol class="arabic">
<li><p class="first">To use the data binning feature with a box plot analysis:</p>
<ol class="arabic">
<li><p class="first">Run tranSMART, then click the <strong>Dataset Explorer</strong> tab.</p>
</li>
<li><p class="first">Define the cohorts you wish to analyze by dragging one or more
concepts from a study into empty subset definition boxes. For more
information, see <em>Populating the Study Groups</em> on page 35.</p>
</li>
<li><p class="first">Click the <strong>Advanced Workflow</strong> tab:</p>
<p><a class="reference internal" href="_images/image76.png"><img alt="image160" src="_images/image76.png" style="width: 5.60000in; height: 0.61597in;" /></a></p>
</li>
<li><p class="first">Select <strong>Box Plot with ANOVA</strong> from the <strong>Analysis</strong> dropdown
menu:</p>
<p><a class="reference internal" href="_images/image105.png"><img alt="image161" src="_images/image105.png" style="width: 1.66042in; height: 2.17986in;" /></a></p>
<p>The Variable Selection section appears. You will need to define
what variables in the study are independent, and what variables
are dependent. At least one of the variables should be continuous
(for example, Age), and one should be a categorical value (for
example, Tissue Type).</p>
</li>
</ol>
<ol class="arabic">
<li><p class="first">Define the variables.</p>
</li>
<li><p class="first">Under <strong>Binning</strong>, click <strong>Enable</strong>:</p>
<p><a class="reference internal" href="_images/image106.png"><img alt="image162" src="_images/image106.png" style="width: 0.81818in; height: 0.28283in;" /></a></p>
</li>
<li><p class="first">Define the following:</p>
</li>
</ol>
</li>
</ol>
<table border="1" class="docutils">
<colgroup>
<col width="5%" />
<col width="30%" />
<col width="65%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head">Field</th>
<th class="head">Description</th>
<th class="head">Comments</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>Variable</td>
<td>Select which variable should define the groups (Independent or Dependent) from the dropdown menu.</td>
<td>If the <em>independent variable</em> defines the groups, boxes will be plotted horizontally. If the <em>dependent variable</em> defines the groups, boxes will be plotted vertically</td>
</tr>
<tr class="row-odd"><td>Variable Type</td>
<td>Select whether the variable you have defined above is continuous or categorical from the dropdown menu.</td>
<td>A continuous variable can be turned into a categorical variable when you use the binning feature.</td>
</tr>
<tr class="row-even"><td>Number of Bins</td>
<td>Type the number of bins you would like data to be organized in.</td>
<td>This step may require trial and error based on how you wish to display data.</td>
</tr>
<tr class="row-odd"><td>Bin Assignments</td>
<td><p class="first">Select how you would like data to be binned from the dropdown menu.</p>
<p class="last"><strong>Note:</strong> This feature can only be used when the variable type selected above is continuous.</p>
</td>
<td><ul class="first last simple">
<li><strong>Evenly Distribute Population:</strong> assigns bins based on the underlying data. For example, if the majority of the subjects in the study were elderly, bins based on age could look like: [(1-40), (40-80), (81-85), (86-90), (90-92)].</li>
<li><strong>Evenly Spaced Bins:</strong> creates bins based on the overall range of the variable. For example, if the majority of the subjects in the study were elderly, bins based on age could look like: [(1-20), (21-40), (41-60), (61-80), (81-100)].</li>
</ul>
</td>
</tr>
<tr class="row-even"><td>Manual Binning</td>
<td><p class="first">Select the checkbox if you wish to bin manually.</p>
<p class="last"><strong>Note:</strong> This is the only binning method available if you are attempting to bin a categorical variable type.</p>
</td>
<td><p class="first">Complete the binning form that populates as a result of checking the <strong>Manual Binning</strong> box.</p>
<p>For continuous data:</p>
<p><a class="reference internal" href="_images/image107.png"><img alt="image163" src="_images/image107.png" style="width: 2.24724in; height: 0.69697in;" /></a></p>
<p>For categorical data:</p>
<p class="last"><a class="reference internal" href="_images/image108.png"><img alt="image164" src="_images/image108.png" style="width: 2.33117in; height: 0.60674in;" /></a></p>
</td>
</tr>
</tbody>
</table>
<ol class="arabic simple">
<li>Click <strong>Run</strong>.</li>
</ol>
</div>
<div class="section" id="data-binning-using-survival-analysis">
<h5>Data Binning Using Survival Analysis<a class="headerlink" href="#data-binning-using-survival-analysis" title="Permalink to this headline">¶</a></h5>
<p>Data binning is used in survival analyses if the variable you wish to
use is continuous (for example, age), but needs to be viewed as
categorical data. Alternatively, it can be used to regroup categorical
data. For example, if histological grade with values such as <em>Well
Defined</em>, <em>Moderately Well Defined</em>, and <em>Poorly Defined</em> are selected,
you can group <em>Moderately Well Defined</em> with <em>Poorly Defined</em> and treat
them as one group for the purposes of this analysis.</p>
<ol class="arabic">
<li><p class="first">To use the data binning feature with a survival analysis:</p>
<ol class="arabic">
<li><p class="first">Run tranSMART, then click the <strong>Dataset Explorer</strong> tab.</p>
</li>
<li><p class="first">Define the cohorts you wish to analyze by dragging one or more
concepts from a study into empty subset definition boxes. For more
information, see <em>Populating the Study Groups</em> on page 35.</p>
</li>
<li><p class="first">Click the <strong>Advanced Workflow</strong> tab:</p>
<p><a class="reference internal" href="_images/image76.png"><img alt="image165" src="_images/image76.png" style="width: 5.60000in; height: 0.61597in;" /></a></p>
</li>
</ol>
<ol class="arabic">
<li><p class="first">Select <strong>Survival Analysis</strong> from the <strong>Analysis</strong> dropdown menu:</p>
<p><a class="reference internal" href="_images/image109.png"><img alt="image166" src="_images/image109.png" style="width: 1.65347in; height: 2.16042in;" /></a></p>
<p>The Variable Selection section appears.</p>
</li>
<li><p class="first">Define the variables:</p>
</li>
</ol>
</li>
</ol>
<table border="1" class="docutils">
<colgroup>
<col width="14%" />
<col width="8%" />
<col width="65%" />
<col width="13%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head">Variable</th>
<th class="head">Required?</th>
<th class="head">Description</th>
<th class="head">Example</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>Time</td>
<td>Yes</td>
<td>A time variable used in the study.</td>
<td>Survival Time</td>
</tr>
<tr class="row-odd"><td>Category</td>
<td>No</td>
<td><p class="first">A variable that you wish to use to sort the cohorts.</p>
<p class="last">If the variable you wish to use is continuous (for example, age), the binning feature should be used.</p>
</td>
<td>Cancer Stage</td>
</tr>
<tr class="row-even"><td>Censoring Variable</td>
<td>No</td>
<td>A censoring variable (occurs when the value of a measurement/observation is partially known).</td>
<td>Survival (Censor)</td>
</tr>
</tbody>
</table>
<ol class="arabic">
<li><p class="first">Under <strong>Binning</strong>, click <strong>Enable</strong>:</p>
<p><a class="reference internal" href="_images/image106.png"><img alt="image167" src="_images/image106.png" style="width: 1.23549in; height: 0.42708in;" /></a></p>
</li>
<li><p class="first">Define the following:</p>
</li>
</ol>
<table border="1" class="docutils">
<colgroup>
<col width="7%" />
<col width="39%" />
<col width="55%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head">Field</th>
<th class="head">Description</th>
<th class="head">Comments</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>Variable Type</td>
<td>Select whether the variable you have defined above is continuous or categorical.</td>
<td>A continuous variable can be treated as a categorical variable when you use the binning feature.</td>
</tr>
<tr class="row-odd"><td>Number of Bins</td>
<td>Type the number of bins you would like data to be organized in.</td>
<td>This step may require trial and error based on how you wish to display data.</td>
</tr>
<tr class="row-even"><td>Bin Assignments</td>
<td><p class="first">Select how you would like data to be binned.</p>
<p class="last"><strong>Note:</strong> This feature can only be used when the variable type selected above is continuous.</p>
</td>
<td><ul class="first last">
<li><p class="first">Evenly Distribute Population: assigns bins based on the underlying data.</p>
<p>For example, if the majority of the subjects in the study were elderly, bins based on age could look like: [(1-40), (40-80), (81-85), (86-90), (90-92)].</p>
</li>
<li><p class="first">Evenly Spaced Bins: creates bins based on the overall range of the variable.</p>
<p>For example, if the majority of the subjects in the study were elderly, bins based on age could look like: [(1-20), (21-40), (41-60), (61-80), (81-100)].</p>
</li>
</ul>
</td>
</tr>
<tr class="row-odd"><td>Manual Binning</td>
<td><p class="first">Select the checkbox if you wish to bin manually.</p>
<p class="last"><strong>Note:</strong> This is the only binning method available if you are attempting to bin a categorical variable type.</p>
</td>
<td><p class="first">Complete the binning form that populates as a result of checking the <strong>Manual Binning</strong> box.</p>
<ul class="simple">
<li>For continuous data:</li>
</ul>
<p><a class="reference internal" href="_images/image107.png"><img alt="image168" src="_images/image107.png" style="width: 2.24724in; height: 0.69697in;" /></a></p>
<ul class="simple">
<li>For categorical data:</li>
</ul>
<p class="last"><a class="reference internal" href="_images/image108.png"><img alt="image169" src="_images/image108.png" style="width: 2.33117in; height: 0.60674in;" /></a></p>
</td>
</tr>
</tbody>
</table>
<ol class="arabic simple">
<li>Click <strong>Run</strong>.</li>
</ol>
</div>
</div>
<div class="section" id="high-dimensional-data">
<h4>High Dimensional Data<a class="headerlink" href="#high-dimensional-data" title="Permalink to this headline">¶</a></h4>
<p>The High Dimensional Data button available within the Advanced Workflow
section of Dataset Explorer allows you to specify additional inputs for
selected variables. These inputs help filter specific information of
value (such as platforms, samples, or timepoints).</p>
<table border="1" class="docutils">
<colgroup>
<col width="5%" />
<col width="95%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image87.png"><img alt="image170" src="_images/image87.png" style="width: 0.31250in; height: 0.31250in;" /></a></th>
<th class="head">The High Dimensional Data feature must be used when you perform an analysis using high dimensional data (such as SNP, gene expression, RBM, etc.) symbolized by the DNA icon (<a class="reference internal" href="_images/image110.png"><img alt="image171" src="_images/image110.png" style="width: 0.13542in; height: 0.13542in;" /></a>). Additionally, the High Dimensional Data feature cannot be used without high dimensional data.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<ol class="arabic">
<li><p class="first">To use the High Dimensional Data feature:</p>
<ol class="arabic">
<li><p class="first">Click the tranSMART <strong>Dataset Explorer</strong> tab to display the
Dataset Explorer window.</p>
</li>
<li><p class="first">In the left pane of the Dataset Explorer window, click the
<strong>Navigate Terms</strong> tab.</p>
<p>The navigation tree appears, showing the categories of available
studies:</p>
<p><a class="reference internal" href="_images/image71.png"><img alt="image172" src="_images/image71.png" style="width: 2.42708in; height: 0.68750in;" /></a></p>
</li>
<li><p class="first">Select the study of interest.</p>
</li>
<li><p class="first">Drag the study of interest into a subset definition box in Subset
1.</p>
</li>
<li><p class="first">Click the <strong>Advanced Workflow</strong> tab above Subset 1:</p>
<p><a class="reference internal" href="_images/image111.png"><img alt="image173" src="_images/image111.png" style="width: 4.73958in; height: 0.44461in;" /></a></p>
</li>
<li><p class="first">Select the analysis you wish to perform, and define at least one
variable using high dimensional data.</p>
</li>
<li><p class="first">Click the <strong>High Dimensional Data</strong> button.</p>
<p>The Compare Subsets-Pathway Selection dialog appears.</p>
</li>
<li><p class="first">Define the following available filters:</p>
</li>
</ol>
</li>
</ol>
<table border="1" class="docutils">
<colgroup>
<col width="8%" />
<col width="92%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image174" src="_images/image4.png" style="width: 0.37500in; height: 0.30208in;" /></a></th>
<th class="head">Dataset Explorer will attempt to pre-populate default values in the associated fields of the dialog based on the underlying data in the variable selection box.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="1" class="docutils">
<colgroup>
<col width="9%" />
<col width="91%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head">Filter</th>
<th class="head">Description</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>Platform</td>
<td>The platform type (for example, SNP, mRNA, etc.) used to collect biomarker data in the study.</td>
</tr>
<tr class="row-odd"><td>GPL Platform</td>
<td>The specific name of the platform used in the study.</td>
</tr>
<tr class="row-even"><td>Sample</td>
<td>The type of sample tested in the study.</td>
</tr>
<tr class="row-odd"><td>Tissue Type</td>
<td>The type of tissue tested in the study.</td>
</tr>
<tr class="row-even"><td>Timepoint</td>
<td>The time period when the sample was taken.</td>
</tr>
<tr class="row-odd"><td>Select a Gene/Pathway</td>
<td><p class="first">The gene, gene signature, or pathway of interest.</p>
<p class="last">If you would like to run the analysis on the entire chip, leave this field blank.</p>
</td>
</tr>
<tr class="row-even"><td>Select SNP Type</td>
<td><p class="first">Select the type of SNP data being used:</p>
<ul class="simple">
<li>Genotype – Use for categorical variables.</li>
<li>Copy Number – Use for continuous variables.</li>
</ul>
<p><strong>Note:</strong> This option is only available when you drag SNP data into the variable selection box.</p>
<p class="last"><strong>Note:</strong> Both Genotype and Copy Number data may not be available for all studies involving SNP data.</p>
</td>
</tr>
<tr class="row-odd"><td>Aggregate Probes?</td>
<td><p class="first">The checkbox can be selected if the variable chosen is either gene expression data or SNP copy number data.</p>
<p>If the checkbox is selected, the algorithm WGCNA (weighted correlation network analysis) is employed. For genes that are comprised of multiple probes, WGCNA selects the probe that best represents the overall expression level or copy number.</p>
<p class="last"><strong>Note:</strong> WGCNA was developed by the Department of Human Genetics at UCLA. For more information, see <a class="reference external" href="http://www.genetics.ucla.edu/labs/horvath/CoexpressionNetwork/">http://www.genetics.ucla.edu/labs/horvath/CoexpressionNetwork/</a>.</p>
</td>
</tr>
</tbody>
</table>
<ol class="arabic simple">
<li>Click <strong>Apply Selections</strong>.</li>
<li>Define any additional required variables, then click <strong>Run</strong>.</li>
</ol>
</div>
<div class="section" id="other-features">
<h4>Other Features<a class="headerlink" href="#other-features" title="Permalink to this headline">¶</a></h4>
<p>The sections below illustrate additional features in the Advanced
Workflow tab.</p>
<div class="section" id="save-to-pdf">
<h5>Save to PDF<a class="headerlink" href="#save-to-pdf" title="Permalink to this headline">¶</a></h5>
<p>The Save to PDF feature allows you to save analyses run through the
Advanced Workflow function within Dataset Explorer.</p>
<ol class="arabic">
<li><p class="first">To save advanced workflow analyses as a PDF file:</p>
<ol class="arabic">
<li><p class="first">Click the tranSMART <strong>Dataset Explorer</strong> tab to display the
Dataset Explorer window.</p>
</li>
<li><p class="first">In the left pane of the Dataset Explorer window, click the
<strong>Navigate Terms</strong> tab.</p>
<p>The navigation tree appears, showing the categories of available
studies:</p>
<p><a class="reference internal" href="_images/image71.png"><img alt="image175" src="_images/image71.png" style="width: 2.42537in; height: 0.68559in;" /></a></p>
</li>
<li><p class="first">Select the study of interest.</p>
</li>
<li><p class="first">Drag the study of interest into a subset definition box in Subset
1.</p>
</li>
<li><p class="first">Click the <strong>Advanced Workflow</strong> tab above Subset 1:</p>
<p><a class="reference internal" href="_images/image111.png"><img alt="image176" src="_images/image111.png" style="width: 4.73958in; height: 0.44461in;" /></a></p>
</li>
<li><p class="first">Select the analysis you wish to perform, and define the variables
accordingly.</p>
</li>
<li><p class="first">Click <strong>Run</strong>.</p>
<p>Your analysis appears below the variable selection panes.</p>
</li>
<li><p class="first">Click <strong>Save to PDF</strong>:</p>
<p><a class="reference internal" href="_images/image112.png"><img alt="image177" src="_images/image112.png" style="width: 6.00000in; height: 0.78397in;" /></a></p>
<p>The following dialog appears:</p>
<p><a class="reference internal" href="_images/image113.png"><img alt="image178" src="_images/image113.png" style="width: 3.34344in; height: 2.52731in;" /></a></p>
</li>
<li><p class="first">Select <strong>Open with</strong> or <strong>Save File</strong>, then click <strong>OK</strong>.</p>
</li>
</ol>
</li>
</ol>
</div>
<div class="section" id="download-raw-r-data">
<h5>Download Raw R Data<a class="headerlink" href="#download-raw-r-data" title="Permalink to this headline">¶</a></h5>
<p>Analyses run through the Advanced Workflow tool within Dataset Explorer
use R for computation. You are able to download raw R data for use in an
external tool.</p>
<table border="1" class="docutils">
<colgroup>
<col width="11%" />
<col width="89%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image179" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">For more information on The R Project for Statistical Computing, visit the following site: <em>www.r-project.org</em>.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<ol class="arabic">
<li><p class="first">To download advanced workflow analyses as raw R data:</p>
<ol class="arabic">
<li><p class="first">Click the tranSMART <strong>Dataset Explorer</strong> tab to display the
Dataset Explorer window.</p>
</li>
<li><p class="first">In the left pane of the Dataset Explorer window, click the
<strong>Navigate Terms</strong> tab.</p>
<p>The navigation tree appears, showing the categories of available
studies:</p>
<p><a class="reference internal" href="_images/image71.png"><img alt="image180" src="_images/image71.png" style="width: 2.42537in; height: 0.68559in;" /></a></p>
</li>
<li><p class="first">Select the study of interest.</p>
</li>
<li><p class="first">Drag the study of interest into a subset definition box in Subset
1.</p>
</li>
<li><p class="first">Click the <strong>Advanced Workflow</strong> tab above Subset 1:</p>
<p><a class="reference internal" href="_images/image111.png"><img alt="image181" src="_images/image111.png" style="width: 4.73958in; height: 0.44461in;" /></a></p>
</li>
<li><p class="first">Select the analysis you wish to perform, and define the variables
accordingly.</p>
</li>
<li><p class="first">Click <strong>Run</strong>.</p>
<p>Your analysis appears below the variable selection panes.</p>
</li>
<li><p class="first">Click <strong>Download raw R data</strong>:</p>
<p><a class="reference internal" href="_images/image114.png"><img alt="image182" src="_images/image114.png" style="width: 1.82847in; height: 0.40417in;" /></a></p>
<p>The following dialog appears:</p>
<p><a class="reference internal" href="_images/image115.png"><img alt="image183" src="_images/image115.png" style="width: 1.83117in; height: 1.38418in;" /></a></p>
</li>
<li><p class="first">Select whether you would like to open the file or save it to your
hard drive, then click <strong>OK</strong>.</p>
</li>
</ol>
</li>
</ol>
</div>
</div>
</div>
<div class="section" id="the-jobs-tab">
<h3>The Jobs Tab<a class="headerlink" href="#the-jobs-tab" title="Permalink to this headline">¶</a></h3>
<p>The Jobs tab allows you to review analyses you have run previously.</p>
<p><a class="reference internal" href="_images/image116.png"><img alt="image184" src="_images/image116.png" style="width: 3.04129in; height: 0.28122in;" /></a></p>
<p>Each advanced workflow analysis that you have run in the past seven days
is logged in the Jobs tab in a spreadsheet format.</p>
<p>The columns of information in the Jobs tab are described below:</p>
<table border="1" class="docutils">
<colgroup>
<col width="8%" />
<col width="92%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head">Column</th>
<th class="head">Description</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>Name</td>
<td><p class="first">The name of the analysis run. The format of the name is as follows:</p>
<p class="last"><a class="reference internal" href="_images/image117.png"><img alt="image185" src="_images/image117.png" style="width: 2.64935in; height: 0.77398in;" /></a></p>
</td>
</tr>
<tr class="row-odd"><td>Status</td>
<td><p class="first">The status of the analysis. Statuses are explained below:</p>
<ul class="last simple">
<li><strong>Completed</strong> – The job has finished and a visualization or analysis is available.</li>
<li><strong>Started</strong> – The job has been started and is still processing.</li>
<li><strong>Uploading File</strong> – You have selected to load additional data into your visualization, and the data is still in the process of uploading to tranSMART.</li>
<li><strong>Error</strong> – The job did not complete due to an error.</li>
<li><strong>Cancelled</strong> – The job was cancelled and will not complete.</li>
</ul>
</td>
</tr>
<tr class="row-even"><td>Run Time</td>
<td>The time the analysis took to process.</td>
</tr>
<tr class="row-odd"><td>Started On</td>
<td>The date and time that the analysis was started.</td>
</tr>
</tbody>
</table>
<table border="1" class="docutils">
<colgroup>
<col width="11%" />
<col width="89%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image186" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">Click the <strong>Refresh</strong> button to view any changes that have been made since the Jobs tab initially populated.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<div class="section" id="viewing-a-logged-job">
<h4>Viewing a Logged Job<a class="headerlink" href="#viewing-a-logged-job" title="Permalink to this headline">¶</a></h4>
<p>Each advanced analysis that you have run in the previous seven days will
be logged in the <strong>Jobs</strong> tab. You may view the visualization or
analysis again by selecting it from the list.</p>
<ol class="arabic">
<li><p class="first">To run a logged advanced workflow:</p>
<ol class="arabic">
<li><p class="first">Run tranSMART, then click the <strong>Dataset Explorer</strong> tab.</p>
</li>
<li><p class="first">In the right pane, click the <strong>Jobs</strong> tab:</p>
<p><a class="reference internal" href="_images/image118.png"><img alt="image187" src="_images/image118.png" style="width: 3.53125in; height: 0.28125in;" /></a></p>
</li>
<li><p class="first">Click the hyperlink of the analysis you are interested in viewing:</p>
<p><a class="reference internal" href="_images/image119.png"><img alt="image188" src="_images/image119.png" style="width: 5.48052in; height: 1.16567in;" /></a></p>
</li>
</ol>
</li>
</ol>
<p>Chapter&nbsp;4</p>
</div>
</div>
</div>
<div class="section" id="chapter-4-gene-signatures-and-gene-lists">
<h2>Chapter 4: Gene Signatures and Gene Lists<a class="headerlink" href="#chapter-4-gene-signatures-and-gene-lists" title="Permalink to this headline">¶</a></h2>
<p>The tranSMART gene signature wizard guides you through the process of
creating a gene signature or gene list. You specify whether the gene
signature or list is publicly available to other tranSMART users or is
reserved for your private use.</p>
<p>Once you create the gene signature or list, it can be used in tranSMART
searches to find clinical studies and experiments where the
differentially regulated genes overlap with the genes contained in the
gene signature or list. This will generate a set of hypotheses about
diseases or treatments that may have similar genes deregulated, and that
can help you develop a further set of experiments.</p>
<table border="1" class="docutils">
<colgroup>
<col width="13%" />
<col width="87%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image189" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">This chapter uses the term “gene signature” to refer to both gene signatures and gene lists.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<div class="section" id="creating-a-gene-signature">
<h3>Creating a Gene Signature<a class="headerlink" href="#creating-a-gene-signature" title="Permalink to this headline">¶</a></h3>
<p>There are two basic tasks involved in creating a gene signature:</p>
<ol class="arabic simple">
<li><ol class="first arabic">
<li>Add the list of genes for the gene signature to a text file.</li>
</ol>
</li>
</ol>
<p>Genes can be indicated by gene symbol or by their associated probe set
ID.</p>
<ol class="arabic simple">
<li>Use the gene signature wizard to define the information on which the
gene signature is based, such as species, source of data, and test
type, and also to import into the gene signature definition the text
file containing the genes.</li>
</ol>
<div class="section" id="step-1-adding-the-genes-to-a-text-file">
<h4>Step 1. Adding the Genes to a Text File<a class="headerlink" href="#step-1-adding-the-genes-to-a-text-file" title="Permalink to this headline">¶</a></h4>
<p>The gene signature wizard expects to import the genes for the gene
signature from a tab-separated text file. The file must contain one, and
possibly two, columns of information:</p>
<ul class="simple">
<li>First column – A list of gene symbols or probe set IDs.</li>
<li>Optional second column – The fold change ratios associated with the
gene symbols or probe set IDs.</li>
</ul>
<p>The fold change ratios can be either <strong>actual values</strong> (for example,
12.8 or -12.8) or one of the following <strong>composite values:</strong></p>
<ul class="simple">
<li><strong>-1</strong>. All down-regulated gene expressions.</li>
<li><strong>1</strong>. All up-regulated gene expressions.</li>
<li><strong>0</strong>. No change.</li>
</ul>
<p>The following table shows the different ways you can specify the genes
for your gene signature:</p>
<table border="1" class="docutils">
<colgroup>
<col width="37%" />
<col width="33%" />
<col width="30%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head">Contents of File</th>
<th class="head">Format</th>
<th class="head">Examples</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>Gene symbols only</td>
<td><em>GeneSymbol</em></td>
<td><p class="first">TCN1</p>
<p>IL1RN</p>
<p>KIAA1199</p>
<p class="last">G0S2</p>
</td>
</tr>
<tr class="row-odd"><td>Gene symbols,
actual fold change</td>
<td><em>GeneSymbol</em>&lt;tab&gt;*ActualFC*</td>
<td><p class="first">CXCL5 -19.19385797</p>
<p>IL8RB -18.21493625</p>
<p>FPR1 -17.6056338</p>
<p class="last">FCGR3A -15.69858713</p>
</td>
</tr>
<tr class="row-even"><td>Gene symbols, composite fold change</td>
<td><em>GeneSymbol</em>&lt;tab&gt;*CompositeFC*</td>
<td><p class="first">CXCL5 -1</p>
<p>IL8RB -1</p>
<p>MMP3 0</p>
<p class="last">SOD2 1</p>
</td>
</tr>
<tr class="row-odd"><td>Probe set IDs only</td>
<td><em>ProbesetID</em></td>
<td><p class="first">224301_x_at</p>
<p>1398191_at</p>
<p>Dr.2473.1.A1_at</p>
<p class="last">A_24_P93251</p>
</td>
</tr>
<tr class="row-even"><td>Probe set IDs,
actual fold change</td>
<td><em>ProbesetID</em>&lt;tab&gt;*ActualFC*</td>
<td><p class="first">224301_x_at -19.19385797</p>
<p>1398191_at -18.21493625</p>
<p>Dr.2473.1.A1_at -17.6056338</p>
<p class="last">A_24_P93251 -15.69858713</p>
</td>
</tr>
<tr class="row-odd"><td>Probe set IDs, composite fold change</td>
<td><em>ProbesetID</em>&lt;tab&gt;*CompositeFC*</td>
<td><p class="first">224301_x_at -1</p>
<p>1398191_at 0</p>
<p>Dr.2473.1.A1_at 1</p>
<p class="last">A_24_P93251 -1</p>
</td>
</tr>
</tbody>
</table>
<div class="section" id="using-transmart-to-select-genes">
<h5>Using tranSMART to Select Genes<a class="headerlink" href="#using-transmart-to-select-genes" title="Permalink to this headline">¶</a></h5>
<p>You can use the tranSMART Search tool to help you select the list of
genes for your gene signature. For example, suppose you are interested
in brain diseases, and want to create a gene signature consisting of
genes that were strongly up-regulated in an experiment involving
Alzheimer’s patients.</p>
<p>You can use tranSMART to select the genes for the gene signature as
follows:</p>
<ol class="arabic simple">
<li><ol class="first arabic">
<li>In tranSMART, click the <strong>Search</strong> tab to display the Search
window.</li>
<li>Type <strong>brain</strong> in the Search field:</li>
</ol>
</li>
</ol>
<p><a class="reference internal" href="_images/image120.png"><img alt="image190" src="_images/image120.png" style="width: 4.92126in; height: 0.88542in;" /></a></p>
<ol class="arabic simple">
<li>Click <strong>Disease&gt; Brain Diseases</strong>.</li>
</ol>
<p>In a few seconds, the search result appears.</p>
<ol class="arabic simple">
<li>Click the <strong>mRNA Analysis</strong> tab, then click the <strong>Study View</strong>
button:</li>
</ol>
<p><a class="reference internal" href="_images/image121.png"><img alt="image191" src="_images/image121.png" style="width: 6.00000in; height: 1.91987in;" /></a></p>
<p>tranSMART displays a list of all the experiments related to brain
diseases.</p>
<ol class="arabic simple">
<li>Scroll through the list of experiments until you find the one to use
as the basis of your gene signature.</li>
<li>Click the <strong>+</strong> icon (<a class="reference internal" href="_images/image29.png"><img alt="image192" src="_images/image29.png" style="width: 0.11761in;" /></a>) to the left of the experiment name:</li>
</ol>
<p><a class="reference internal" href="_images/image122.png"><img alt="image193" src="_images/image122.png" style="width: 4.29870in; height: 1.52521in;" /></a></p>
<p>A list of the analyses based on this experiment appears.</p>
<ol class="arabic simple">
<li>Click the <strong>Excel</strong> button for the analysis that you want to use for
your gene signature<strong>:</strong></li>
</ol>
<p><a class="reference internal" href="_images/image123.png"><img alt="image194" src="_images/image123.png" style="width: 6.00000in; height: 2.24551in;" /></a></p>
<p>This action exports the analysis information, including the gene
expression data, to a Microsoft Excel file.</p>
<ol class="arabic simple">
<li>Click <strong>Open</strong> in the File Download dialog.</li>
</ol>
<p>Excel starts up and displays the analysis data – for example:</p>
<p><a class="reference internal" href="_images/image124.png"><img alt="image195" src="_images/image124.png" style="width: 5.84416in; height: 0.99338in;" /></a></p>
<ol class="arabic simple">
<li>Remove any rows containing genes that you do not want to include in
the gene signature.</li>
<li>Remove the column headings and all data from the spreadsheet except
for the gene symbol and fold change ratio, then export the remaining
data to a tab-separated text file using the Excel <strong>Save as type</strong>
option <strong>Text (Tab delimited) (*.txt)</strong>.</li>
</ol>
<table border="1" class="docutils">
<colgroup>
<col width="50%" />
<col width="50%" />
</colgroup>
<tbody valign="top">
<tr class="row-odd"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="section" id="step-2-creating-the-gene-signature">
<h4>Step 2. Creating the Gene Signature<a class="headerlink" href="#step-2-creating-the-gene-signature" title="Permalink to this headline">¶</a></h4>
<ol class="arabic simple">
<li><ol class="first arabic">
<li>In tranSMART, click the <strong>Gene Signature/Lists</strong> tab.</li>
<li>Click the <strong>New Signature</strong> button.</li>
</ol>
</li>
</ol>
<p>The first page of the gene signature wizard appears:</p>
<p><a class="reference internal" href="_images/image125.png"><img alt="image196" src="_images/image125.png" style="width: 6.00000in; height: 1.93804in;" /></a></p>
<table border="1" class="docutils">
<colgroup>
<col width="14%" />
<col width="86%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image197" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">Required fields on gene signature wizard pages are marked with a red asterisk (*).</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<p>You can find additional information about the gene signature wizard by
clicking <strong>Information</strong> on any wizard page.</p>
<ol class="arabic simple">
<li>Specify a name (required) and an optional description for your gene
signature, then click Meta-Data to proceed to the next gene wizard
page.</li>
</ol>
<p>Gene Sig Pg.</p>
<p>The second page of the gene signature wizard appears:</p>
<p><a class="reference internal" href="_images/image126.png"><img alt="image198" src="_images/image126.png" style="width: 5.98958in; height: 5.07083in;" /></a></p>
<ol class="arabic simple">
<li>Specify values in the required fields <strong>Species</strong> and <strong>Technology</strong>
<strong>Platform</strong>, and also in any other relevant fields, then click Next
to proceed to the final gene signature wizard page:</li>
</ol>
<p><a class="reference internal" href="_images/image127.png"><img alt="image199" src="_images/image127.png" style="width: 6.00000in; height: 2.96058in;" /></a></p>
<p>Gene Sig Pg 3</p>
<ol class="arabic simple">
<li>Specify values in the required field <strong>P-value Cutoff</strong>.</li>
<li>In the section <strong>File Upload Information</strong>, describe the text file
you created in the section <em>Step 1. Adding the Genes to a Text File</em>
on page 81, using the required fields <strong>File Information</strong> and
<strong>Upload File</strong>:</li>
</ol>
<ul class="simple">
<li>In the <strong>File schema</strong> section of <strong>File Information</strong>, select <strong>Gene
Symbol &lt;tab&gt; Metric Indicator</strong> or <strong>Probe Set Symbol &lt;tab&gt; Metric
Indicator</strong>, depending on the method you chose to specify the genes.</li>
<li>In the <strong>Fold change metric</strong> section of <strong>File Information</strong>, select
one of the following choices from the dropdown:</li>
</ul>
<table border="1" class="docutils">
<colgroup>
<col width="23%" />
<col width="77%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head">Fold Change Metric Indicator</th>
<th class="head">Description</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>Actual fold change</td>
<td>The text file contains actual fold change values for each gene symbol or probe set ID.</td>
</tr>
<tr class="row-odd"><td>Not used</td>
<td>The text file contains gene symbols or probe set ID only. There are no associated fold change values.</td>
</tr>
<tr class="row-even"><td>-1 (down), 1 (up), 0 (optional for unchanged)</td>
<td>The fold change values are not actual values. They simply represent whether the gene expression was down-regulated (-1), up-regulated (1), or unchanged (0).</td>
</tr>
</tbody>
</table>
<ul class="simple">
<li>In <strong>Upload File</strong>, specify the path and name of the file that
contains the genes to import. Use the <strong>Browse</strong> button to select the
file from the navigation tree.</li>
</ul>
<ol class="arabic simple">
<li>Specify values in any other relevant fields on this gene wizard page,
then click <strong>Save</strong> to save the gene signature.</li>
</ol>
<p>The new gene signature appears in the <strong>Gene Signature List</strong> at the top
of the Gene Signature/List view:</p>
<p><a class="reference internal" href="_images/image128.png"><img alt="image200" src="_images/image128.png" style="width: 6.00000in; height: 0.82086in;" /></a></p>
<div class="section" id="making-a-new-gene-signature-public">
<h5>Making a New Gene Signature Public<a class="headerlink" href="#making-a-new-gene-signature-public" title="Permalink to this headline">¶</a></h5>
<p>By default, a newly created gene signature is private.</p>
<ol class="arabic simple">
<li>To make a gene signature public:<ol class="arabic">
<li>In the <strong>Gene Signature List</strong>, click the <strong>Select</strong> <strong>Action</strong>
dropdown to the right of the gene signature you just created.</li>
<li>Click <strong>Make Public</strong> in the dropdown list:</li>
</ol>
</li>
</ol>
<p><a class="reference internal" href="_images/image129.png"><img alt="image201" src="_images/image129.png" style="width: 5.98958in; height: 1.34375in;" /></a></p>
<p>After you click <strong>Make Public</strong>, the value in the <strong>Public</strong> column for
the gene signature changes from <strong>No</strong> to <strong>Yes</strong>:</p>
<p><a class="reference internal" href="_images/image130.png"><img alt="image202" src="_images/image130.png" style="width: 6.00000in; height: 0.80833in;" /></a></p>
<table border="1" class="docutils">
<colgroup>
<col width="12%" />
<col width="88%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image203" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">tranSMART users assigned the role ROLE_ADMIN have access to both public and private gene signatures.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div class="section" id="performing-actions-on-your-gene-signatures">
<h3>Performing Actions on Your Gene Signatures<a class="headerlink" href="#performing-actions-on-your-gene-signatures" title="Permalink to this headline">¶</a></h3>
<ol class="arabic simple">
<li>To edit or perform other actions on a gene signature in your gene
signature list:<ol class="arabic">
<li>In tranSMART, click the <strong>Gene Signature/Lists</strong> tab.</li>
</ol>
</li>
</ol>
<p>The <strong>Gene Signature List</strong> appears, containing all the genes you have
created:</p>
<p><a class="reference internal" href="_images/image131.png"><img alt="image204" src="_images/image131.png" style="width: 5.99937in; height: 0.81289in;" /></a></p>
<ol class="arabic simple">
<li>Click the <strong>Select Action</strong> dropdown for the gene signature you are
acting on. The dropdown contains all the actions you can perform on
the gene signature:</li>
</ol>
<table border="1" class="docutils">
<colgroup>
<col width="7%" />
<col width="93%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head">Action</th>
<th class="head">Description</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>Clone</td>
<td><p class="first">Create an exact duplicate of the gene signature definition (<em>except</em> for the text file containing the gene symbols and fold change values), and display the definition in the gene signature wizard.</p>
<p class="last">Cloning a gene signature helps you create a new gene signature with a similar definition to an existing one. However, it is expected you will import a different set of genes into the gene signature.</p>
</td>
</tr>
<tr class="row-odd"><td>Delete</td>
<td>Delete the gene signature.</td>
</tr>
<tr class="row-even"><td>Edit</td>
<td><p class="first">Open the gene signature in the gene signature wizard for editing.</p>
<p>The gene signature wizard displays all the information in the gene signature, including the reference to the text file containing the list of genes and fold change values. If you want to choose a different text file, click the following label:</p>
<p><a class="reference internal" href="_images/image132.png"><img alt="image205" src="_images/image132.png" style="width: 2.97500in; height: 0.22500in;" /></a></p>
<p class="last">To save any changes you make during editing, you must click the <strong>Save</strong> button on the third page of the wizard.</p>
</td>
</tr>
<tr class="row-odd"><td>Edit Items</td>
<td>Add, delete, or modify one or more genes in the text file containing the gene symbols and fold change values.</td>
</tr>
<tr class="row-even"><td>Excel Download</td>
<td><p class="first">Generate the entire contents of the gene signature, including the information in the text file containing the gene symbols and fold change values, to a Microsoft Excel spreadsheet.</p>
<p class="last">The gene signature definition and gene symbols/fold change values are written to separate spreadsheets.</p>
</td>
</tr>
<tr class="row-odd"><td>Make Public</td>
<td><p class="first">Make a private gene signature public.</p>
<p><strong>Note:</strong> To make a public gene signature private, edit the gene signature and set the <strong>Public?</strong> field to <strong>No</strong> on the first page of the gene signature wizard:</p>
<p class="last"><a class="reference internal" href="_images/image133.png"><img alt="image206" src="_images/image133.png" style="width: 2.00000in; height: 0.23333in;" /></a></p>
</td>
</tr>
</tbody>
</table>
</div>
<div class="section" id="performing-actions-on-other-users-signatures">
<h3>Performing Actions on Other Users’ Signatures<a class="headerlink" href="#performing-actions-on-other-users-signatures" title="Permalink to this headline">¶</a></h3>
<p>You can perform actions on gene signatures that other tranSMART users
have created. The gene signatures you can access and the actions you can
perform on them depend on the role assigned to your tranSMART user ID,
as follows:</p>
<table border="1" class="docutils">
<colgroup>
<col width="29%" />
<col width="71%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head">Role</th>
<th class="head">Authorized Actions</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>ROLE_ADMIN</td>
<td>All actions on all gene signatures, both public and private.</td>
</tr>
<tr class="row-odd"><td>ROLE_SPECTATOR
ROLE_STUDY_OWNER
ROLE_DATASET_EXPLORER_ADMIN</td>
<td>Only <strong>Clone</strong> and <strong>Excel</strong> <strong>Download</strong>, and only on public gene signatures.</td>
</tr>
</tbody>
</table>
<ol class="arabic simple">
<li>To edit or perform actions on a gene signature other than your own:<ol class="arabic">
<li>In tranSMART, click the <strong>Gene Signature/Lists</strong> tab.</li>
<li>Click <strong>Public Signatures</strong> to open the list of public gene
signatures:</li>
</ol>
</li>
</ol>
<p><a class="reference internal" href="_images/image134.png"><img alt="image207" src="_images/image134.png" style="width: 5.98958in; height: 1.33333in;" /></a></p>
<table border="1" class="docutils">
<colgroup>
<col width="11%" />
<col width="89%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image4.png"><img alt="image208" src="_images/image4.png" style="width: 0.37000in; height: 0.30000in;" /></a></th>
<th class="head">tranSMART users assigned the role ROLE_ADMIN will see <strong>Other Signatures</strong> instead of <strong>Public Signatures</strong>.</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<ol class="arabic simple">
<li>Click the <strong>Select Action</strong> dropdown for the gene signature you want
to act on.</li>
<li>Select the action you want to perform on the gene signature.</li>
</ol>
</div>
<div class="section" id="viewing-a-gene-signature-definition">
<h3>Viewing a Gene Signature Definition<a class="headerlink" href="#viewing-a-gene-signature-definition" title="Permalink to this headline">¶</a></h3>
<p>You can view the definition of a gene signature, including its list of
genes and fold change values, for any gene signature you are authorized
to access.</p>
<p>To view a gene signature definition, click the <strong>Detail</strong> icon
(<a class="reference internal" href="_images/image135.png"><img alt="image209" src="_images/image135.png" style="width: 0.18403in; height: 0.13542in;" /></a>) next to the gene signature name.</p>
<p>The Gene Signature Detail dialog appears, containing the gene signature
definition:</p>
<p><a class="reference internal" href="_images/image136.png"><img alt="image210" src="_images/image136.png" style="width: 6.00000in; height: 2.64583in;" /></a></p>
<p>Chapter&nbsp;5</p>
</div>
</div>
<div class="section" id="chapter-5-other-tasks">
<h2>Chapter 5: Other Tasks<a class="headerlink" href="#chapter-5-other-tasks" title="Permalink to this headline">¶</a></h2>
<div class="section" id="other-tasks">
<h3>Other Tasks<a class="headerlink" href="#other-tasks" title="Permalink to this headline">¶</a></h3>
<p>In addition to Search, Dataset Explorer, and Gene Signature/List, the
tranSMART toolbar includes the following dropdown menu and tool tabs:</p>
<ul>
<li><p class="first"><strong>Utilities</strong></p>
<p>Options under this menu include Help (takes you to the online help
manual), Contact Us (opens email for questions, problem reports, or
any other feedback about the tranSMART application), About (version
information for tranSMART application), and access to Login and
Logout</p>
</li>
<li><p class="first"><strong>Admin</strong></p>
</li>
</ul>
<p>Perform user administration functions such as creating users, assigning
roles to users, and enabling or disabling user accounts.</p>
<p>This tool tab is visible only to users who are assigned the role
ROLE_ADMIN.</p>
<p>Appendix&nbsp;A</p>
</div>
</div>
<div class="section" id="appendix-a-how-tea-scores-are-calculated">
<h2>Appendix A: How TEA Scores Are Calculated<a class="headerlink" href="#appendix-a-how-tea-scores-are-calculated" title="Permalink to this headline">¶</a></h2>
<p>This appendix summarizes the operations tranSMART performs to calculate
the overall TEA score for an experiment, and the data inputs that the
calculation requires. Pseudocode representations of the operations being
performed are included where they may clarify the operation.</p>
<div class="section" id="data-inputs-to-the-tea-algorithm">
<h3><strong>Data Inputs to the TEA algorithm</strong><a class="headerlink" href="#data-inputs-to-the-tea-algorithm" title="Permalink to this headline">¶</a></h3>
<p>One of the following:</p>
<ul class="simple">
<li>A gene signature or gene list containing any number of genes, with a
binary up- or down-regulation flag based on fold change.</li>
<li>A pathway containing any number of genes.</li>
</ul>
<p>And:</p>
<ul class="simple">
<li>A gene search result list for each signature, list, or pathway gene.
Result lists contain experimental comparisons.</li>
</ul>
</div>
<div class="section" id="operations">
<h3>Operations<a class="headerlink" href="#operations" title="Permalink to this headline">¶</a></h3>
<ol class="arabic">
<li><ol class="first arabic simple">
<li>Compute the average fold change and standard deviation for all
genes in the comparison.</li>
<li>Compute a normalized p-value (NPV) for each gene in the
comparison, based on its fold change (fc) value, and the above
average (ave) and standard deviation (std) values. Use a normal
distribution function (CDF):</li>
</ol>
<blockquote>
<div><p>if (fc &gt; 0)</p>
<p>NPV = 1.0 - CDF(fc, ave, std)</p>
<p>else</p>
<p>NPV = 1.0 - CDF(-fc, ave, std)</p>
<p>if NPV &lt; 1.0e-15, set to 1.0e-15</p>
</div></blockquote>
</li>
</ol>
<ol class="arabic simple">
<li>For each gene in the gene signature, list, or pathway, search against
experimental comparisons and extract those comparisons where the
gene’s normalized p-value is less than 0.05. This returns a
comparison list.</li>
<li>Iterate through the comparison list. For each comparison (C), add the
normalized p-value to one of two arrays of sums (pv_sum), as
follows:</li>
</ol>
<ul class="simple">
<li>For gene signatures and gene lists, add the gene’s normalized p-value
to:<ul>
<li>pv_sum(C, up) if the gene’s fold change in the signature (svc)
<em>and</em> in the comparison (cfc) are co-regulated.</li>
<li>pv_sum(C, down) if the gene’s fold change in the signature <em>and</em>
in the comparison are anti-regulated.</li>
</ul>
</li>
<li>For pathways, add the gene’s normalized p-value to:<ul>
<li>pv_sum(C, up) if the gene’s comparison fold change (cfc) is
up-regulated.</li>
<li>pv_sum(C, down) if the gene’s comparison fold change is
down-regulated.</li>
</ul>
</li>
</ul>
<p>Also, use the logarithm of the normalized p-value to make the final TEA
score more human readable:</p>
<blockquote>
<div><p>if (gene_signature OR gene_list)</p>
<p>if ( (sfc &gt; 0 AND cfc &gt; 0) OR (sfc &lt; 0 AND cfc &lt; 0) )</p>
<p>pv_sum(C, up) += -Math.log(NPV)</p>
<p>pv_count(C, up)++</p>
<p>else</p>
<p>pv_sum(C, down) += -Math.log(NPV)</p>
<p>pv_count(C, down)++</p>
<p>if (gene_pathway)</p>
<p>if (cfc &gt; 0)</p>
</div></blockquote>
<p>pv_sum(C, up) += -Math.log(NPV)</p>
<p>pv_count(C, up)++</p>
<p>else</p>
<blockquote>
<div><p>pv_sum(C, down) += -Math.log(NPV)</p>
<p>pv_count(C, down)++</p>
</div></blockquote>
<ol class="arabic">
<li><p class="first">Compute the min-LogP average (pv_ave) for each sum:</p>
<blockquote>
<div><p>pv_ave(C, up) = Math.exp(-pv_sum(C,up) / pv_count(C,up) )</p>
<p>pv_ave(C, down) = Math.exp(-pv_sum(C,down) / pv_count(C,down) )</p>
</div></blockquote>
</li>
</ol>
<ol class="arabic simple">
<li>Compute a TEA score (pv_tea) for each min-LogP average through a
binomial distribution function:</li>
</ol>
<p>pv_tea(C, up) = 1.0 - Binom( N, pv_count(C,up), pv_ave(C,up) )</p>
<p>pv_tea(C, down) = 1.0 - Binom( N, pv_count(C,down), pv_ave(C,down) )</p>
</div>
<div class="section" id="result">
<h3>Result<a class="headerlink" href="#result" title="Permalink to this headline">¶</a></h3>
<ul class="simple">
<li><strong>TEA score</strong>: For gene signatures, lists, and pathways, the final
TEA score is the more significant pv_tea value (the lower of the two
pv_tea values).</li>
<li>A gene signature or list is determined to be co-regulated or
anti-regulated as follows:<ul>
<li><strong>Co-regulated</strong>: The more significant pv_tea value was derived
from the sums associated with co-regulated fold change values
(pv_sum(C, up)).</li>
<li><strong>Anti-regulated</strong>: The more significant pv_tea value was derived
from the sums associated with anti-regulated fold change values
(pv_sum(C, down)).</li>
</ul>
</li>
<li>A pathway is determined to be up-regulated or down-regulated as
follows:<ul>
<li><strong>Up-regulated</strong>: The more significant pv_tea value was derived
from the sums associated with up-regulated fold change values
(pv_sum(C, up)).</li>
<li><strong>Down-regulated</strong>: The more significant pv_tea value was derived
from the sums associated with down-regulated fold change values
(pv_sum(C, down)).</li>
</ul>
</li>
</ul>
<p>Appendix&nbsp;B</p>
</div>
</div>
<div class="section" id="appendix-b-rules-for-loading-omicsoft-data">
<h2>Appendix B: Rules for Loading OmicSoft Data<a class="headerlink" href="#appendix-b-rules-for-loading-omicsoft-data" title="Permalink to this headline">¶</a></h2>
<div class="section" id="rules-for-loading-omicsoft-data">
<h3>Rules for Loading OmicSoft Data<a class="headerlink" href="#rules-for-loading-omicsoft-data" title="Permalink to this headline">¶</a></h3>
<p>The following rules determine whether OmicSoft data is loaded into
tranSMART databases:</p>
<p><strong>Rules for loading data into table BIO_ASSAY_ANALYSIS_DATA</strong></p>
<p>Load the data if one of the following two criteria is satisfied:</p>
<ul class="simple">
<li>fold_change_ratio is &gt;= 1.0 OR &lt;=-1.0</li>
</ul>
<p>AND</p>
<p>preferred_pvalue is null OR &lt;= 0.1</p>
<ul class="simple">
<li>fold_change_ratio is null</li>
</ul>
<p>AND</p>
<p>At least one of the following values is not null:</p>
<ul class="simple">
<li>r_value</li>
</ul>
<p>Pearson product-moment correlation coefficient.</p>
<ul class="simple">
<li>rho_value</li>
</ul>
<p>Spearman rank correlation coefficient</p>
<ul class="simple">
<li>cut_value</li>
<li>results_value</li>
</ul>
<p><strong>Rules for loading data into table BIO_ASSAY_ANALYSIS_DATA_TEA</strong></p>
<p>Load the data if:</p>
<p>TEA_NORMALIZED_PVALUE &lt;= 0.05</p>
<p>For information about the TEA algorithm, see <em>Appendix A: How TEA Scores
Are Calculated</em>.</p>
<p>Appendix&nbsp;C</p>
</div>
</div>
<div class="section" id="appendix-c-glossary-of-terms">
<h2>Appendix C: Glossary of Terms<a class="headerlink" href="#appendix-c-glossary-of-terms" title="Permalink to this headline">¶</a></h2>
<div class="section" id="glossary-of-terms">
<h3>Glossary of Terms<a class="headerlink" href="#glossary-of-terms" title="Permalink to this headline">¶</a></h3>
<p>aggregate probes</p>
<p>Used in Dataset Explorer, the Aggregate Probes checkbox allows you to
group probes used in high-dimensional data samples to form a total
quantity that analyses will be performed on.</p>
<p>analysis of variance (anova)</p>
<p>Analysis of Variance (ANOVA) is a statistical method used in Dataset
Explorer to make concurrent comparisons between two or more means in a
box plot.</p>
<p>analysis view</p>
<p>Used in the Search tool, the Analysis View option displays the
statistically significant analyses from your search filter(s).</p>
<p>anti-regulation</p>
<p>An analysis of a statistically significant experiment returned from a
search against a gene signature or list is designated as <em>co-regulated</em>
or <em>anti-regulated</em>.</p>
<p>array data</p>
<p>See: <a class="reference external" href="#Microarray">Microarray</a></p>
<p>arrayexpress</p>
<p>Database of gene expression and other microarray data at the European
Bioinformatics Institute (EBI).</p>
<p>See <a class="reference external" href="http://www.ebi.ac.uk/arrayexpress">http://www.ebi.ac.uk/arrayexpress</a> for details.</p>
<p>binomial distribution</p>
<p>Graph that displays the discrete probability distribution of obtaining
<em>n</em> successes out of N Bernoulli trials.</p>
<p>See <a class="reference external" href="http://mathworld.wolfram.com/BinomialDistribution.html">http://mathworld.wolfram.com/BinomialDistribution.html</a> for details.</p>
<p>biomarker</p>
<p>Short for Biological Marker, a biomarker is a key molecular or cellular
event that links a specific environmental exposure to a health outcome.</p>
<p>box plot</p>
<p>Also known as a Box and Whisker Plot, a box plot is a histogram-like
method of displaying data. Box plots are useful when conveying location
and variation information in datasets.</p>
<p>categorical variable</p>
<p>Also known as a nominal value, a categorical variable is one that has
two or more categories, but with no intrinsic ordering to the
categories. An example of a categorical value is hair color – there is
no way to order these variables from highest to lowest.</p>
<p>censoring value</p>
<p>Used in Survival Analyses. The Censoring Value specifies which patients
had the event whose time is being measured. For example, if the Time
variable selected is Overall Survival Time (Years), an appropriate
censoring variable is Patient Death.</p>
<p>chi squared</p>
<p>Let the probabilities of various classes in a distribution be
<a class="reference internal" href="_images/image137.png"><img alt="image211" src="_images/image137.png" style="width: 0.14444in; height: 0.14444in;" /></a>, <a class="reference internal" href="_images/image138.png"><img alt="image212" src="_images/image138.png" style="width: 0.14444in; height: 0.14444in;" /></a>, ..., <a class="reference internal" href="_images/image139.png"><img alt="image213" src="_images/image139.png" style="width: 0.14444in; height: 0.14444in;" /></a>, with observed frequencies
<a class="reference internal" href="_images/image140.png"><img alt="image214" src="_images/image140.png" style="width: 0.17083in; height: 0.14444in;" /></a>, <a class="reference internal" href="_images/image141.png"><img alt="image215" src="_images/image141.png" style="width: 0.17083in; height: 0.14444in;" /></a>, ..., <a class="reference internal" href="_images/image142.png"><img alt="image216" src="_images/image142.png" style="width: 0.17083in; height: 0.14444in;" /></a>. The quantity</p>
<table border="1" class="docutils">
<colgroup>
<col width="78%" />
<col width="22%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head"><a class="reference internal" href="_images/image143.png"><img alt="image217" src="_images/image143.png" style="width: 1.28958in; height: 0.48681in;" /></a></th>
<th class="head">&nbsp;</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>&nbsp;</td>
<td>&nbsp;</td>
</tr>
</tbody>
</table>
<p>is therefore a measure of the deviation of a sample from expectation,
where <a class="reference internal" href="_images/image144.png"><img alt="image218" src="_images/image144.png" style="width: 0.10556in; height: 0.14444in;" /></a>is the sample size.</p>
<p>cohort</p>
<p>A group of subjects who have shared a specific event or characteristic.</p>
<p>continuous variable</p>
<p>Continuous variables have an infinite number of values between two
points. For example, age or temperature.</p>
<p>co-regulation</p>
<p>An analysis of a statistically significant experiment returned from a
search against a gene signature or list is designated as <em>co-regulated</em>
or <em>anti-regulated</em>.</p>
<p>correlation analysis</p>
<p>A type of Regression Analysis, correlation analysis measures the
correlation coefficient – the linear association between two variables.
Values of the correlation coefficient are always between -1 and +1. A
correlation coefficient of +1 indicates that two variables are perfectly
related in a positive linear sense, while a correlation coefficient of
-1 indicates that two variables are perfectly related in a negative
linear sense.</p>
<p>cox coefficient</p>
<p>The Cox coefficient refers to the coefficients in a Cox regression model
(also known as the proportional hazards model for survival-time). The
analysis investigates the effects of one or more variables upon the time
a specified event takes to happen. The cox coefficient relates to a
hazard; a positive coefficient indicates a worse prognosis, while a
negative coefficient indicates a protective effect of the variable.</p>
<p>data binning</p>
<p>Defers to a data pre-processing technique used to reduce observation
errors and to allow continuous variables to become categorical. Clusters
of data are replaced by a value representative of that cluster (often
but not necessarily, the central value).</p>
<p>data warehouse</p>
<p>A database used for reporting and analysis.</p>
<p>dataset</p>
<p>Collection of data, most commonly presented in a tabular form where each
column represents a specific variable, and each row represents a value
for that variable.</p>
<p>dataset explorer</p>
<p>Dataset Explorer lets you compare data generated for test subjects in
two different study groups, based on criteria and points of comparison
that you specify. Dataset Explorer is useful to help you test a
hypothesis that involves the criteria and points of comparison that you
select.</p>
<p>dependent variable</p>
<p>In an experiment, the dependent variable is the response that is
measured.</p>
<p>differential modulation</p>
<p>down-regulation</p>
<p>An analysis of a statistically significant experiment returned from a
search against a pathway is designated as <em>up-regulated</em> or
<em>down-regulated</em>.</p>
<p>entrez gene</p>
<p>Reference sequences for a wide range of species. For details, see
<a class="reference external" href="http://www.ncbi.nlm.nih.gov/gene/">http://www.ncbi.nlm.nih.gov/gene/</a>.</p>
<p>entrez global</p>
<p>Federated search engine that allows users to search various health
sciences databases at the National Center for Biotechnology Information
(NCBI) website.</p>
<p>See <a class="reference external" href="http://www.ncbi.nlm.nih.gov/gqueryfor">http://www.ncbi.nlm.nih.gov/gqueryfor</a> details.</p>
<p>fold change ratio</p>
<p>A number describing how much a quantity changes going from an initial to
a final value. An initial value of 50 and a final value of 100
corresponds to a fold change of 2 (a two-fold increase).</p>
<p>gene</p>
<p>Stretches of DNA and RNA that code for a polypeptide or for an RNA chain
– contains hereditary molecular information.</p>
<p>gene chip</p>
<p>See: <a class="reference external" href="#Microarray">Microarray</a></p>
<p>gene expression</p>
<p>The flow of genetic information from gene to protein; the process, or
the regulation of the process, by which the effects of a gene are
manifested; the manifestation of a heritable trait in an individual
carrying the gene or genes that determine it.</p>
<p>gene expression omnibus</p>
<p>GEO is an international public repository that archives and freely
distributes microarray, next-generation sequencing, and other forms of
high-throughput functional genomics data submitted by the research
community. For more information, see <a class="reference external" href="http://www.ncbi.nlm.nih.gov/geo">http://www.ncbi.nlm.nih.gov/geo</a>.</p>
<p>gene set enrichment analysis (gsea)</p>
<p>Computational method that determines whether an a priori defined set of
genes shows statistically significant, concordant differences between
two biological states (for example, phenotypes).</p>
<p>See <a class="reference external" href="http://www.broadinstitute.org/gsea/index.jsp">http://www.broadinstitute.org/gsea/index.jsp</a> for details.</p>
<p>gene signature</p>
<p>A group of genes whose combined expression pattern is uniquely
characteristic of a medical condition or other clinical outcome of
interest.</p>
<p>gene symbol</p>
<p>A unique abbreviation of a gene name consisting of italicized uppercase
Latin letters and Arabic numbers. We use Entrez as the full list of
genes (related to but not identical to HUGO)</p>
<p>See <a class="reference external" href="http://www.genenames.org/">http://www.genenames.org/</a> for details.</p>
<p>genecards</p>
<p>Database that offers information about human genes (and mouse
homologues).</p>
<p>See <a class="reference external" href="http://www.genecards.org">http://www.genecards.org</a> for details.</p>
<p>google scholar</p>
<p>Google application that provides a search of scholarly literature across
multiple disciplines and sources.</p>
<p>See <a class="reference external" href="http://scholar.google.com">http://scholar.google.com</a> for details.</p>
<p>gpl platform</p>
<p>A Platform record is composed of a summary description of the array or
sequencer and, for array-based Platforms, a data table defining the
array template. Each Platform record is assigned a unique and stable GEO
accession number (GPLxxx). A Platform may reference many Samples that
have been submitted by multiple submitters.</p>
<p>heatmap</p>
<p>Display of differential expression. Individual values contained in the
matrix are represented by colors.</p>
<p>hierarchical clustering</p>
<p>Hierarchical clustering is a type of clustering analysis whose goal is
to organize data so that the objects in the same cluster are more
similar to each other than to those in other clusters.</p>
<p>high dimensional data</p>
<p>Datasets where the intersection of a subject and measurement is
comprised of hundreds or thousands of points. For example, in a low
dimensional data measurement such as height the intersection of subject
and measurement is one number (ex. 180 cm) whereas in a high dimensional
data measurement such as gene expression in a lymph node the measurement
is 50,000 individual probe expression values.</p>
<p>histogram</p>
<p>A visual representation of the distribution of data values within a
dataset.</p>
<p>homology</p>
<p>The basis for comparative biology – where organs/structures from one
organism are compared to a similar organ/structure in a different
organism.</p>
<p>in vitro study</p>
<p>Those that are conducted using components of an organism that have been
isolated from their usual biological surroundings.</p>
<p>in vivo studies</p>
<p>Experimentation using a whole, living organism.</p>
<p>independent variable</p>
<p>In an experiment, the independent variable is the variable that is
manipulated.</p>
<p>job</p>
<p>In tranSMART, a job refers to a command you have given Dataset Explorer
to process or export data. Jobs and job-related events can be found
within the <strong>Jobs</strong> tab in Dataset Explorer.</p>
<p>kendall correlation</p>
<p>Kendall&#8217;s rank correlation provides a distribution-free test of
independence and a measure of the strength of dependence between two
variables.</p>
<p>k-means clustering</p>
<p>The K-Means clustering heatmap clusters genes and/or samples into a
specified number of clusters. The result is <em>k</em> clusters, each centered
around a randomly-selected data point.</p>
<p>line graph</p>
<p>Line graphs illustrate the temporal relationship between two major
variables.</p>
<p>marker selection</p>
<p>Marker Selection is a display of the top differentially expressed genes
between two specified cohorts.</p>
<p>mesh ontology</p>
<p>MeSH is the National Library of Medicine&#8217;s controlled vocabulary
thesaurus. It consists of sets of terms naming descriptors in a
hierarchical structure that permits searching at various levels of
specificity.</p>
<p>microarray</p>
<p>A two-dimensional array on a chip or solid surface that assays large
amounts of DNA material.</p>
<p>mrna analysis</p>
<p>Assays that quantify the expression levels of all mRNA molecules in an
experiment.</p>
<p>navigation tree</p>
<p>The Window’s Explorer-like, hierarchical representation of study data
that has been loaded into Dataset Explorer.</p>
<p>ncbi</p>
<p>The National Center for Biotechnology Information.</p>
<p>See <a class="reference external" href="http://">http://</a><a class="reference external" href="http://www.ncbi.nlm.nih.gov/">www.ncbi.nlm.nih.gov/</a>
for details.</p>
<p>numeric-node</p>
<p>Used in Dataset Explorer, numeric-nodes are indicated by the (<strong>123</strong>)
symbol, numeric nodes indicate that the data values associated with the
concept are only numeric (for example, age values, date values, etc.).
For more information, see <a class="reference external" href="#ContinuousVariable">Continuous Variable</a>.</p>
<p>ontology</p>
<p>A hierarchical description of the concepts and relationships that can
exist for an agent or a community of agents.</p>
<p>orthogonal component</p>
<p>When performing statistical analysis, independent variables that affect
a particular dependent variable are said to be orthogonal if they are
uncorrelated, since the covariance forms an inner product.</p>
<p>pathology</p>
<p>The study of diagnosis and disease.</p>
<p>pathway</p>
<p>A group of genes interacting to form an aggregate biological function.</p>
<p>pearson correlation</p>
<p>Obtained by dividing the covariance of the two variables by the product
of their standard deviations.</p>
<p>principal component analysis</p>
<p>A Principal Component Analysis (PCA) is commonly used as a tool in
exploratory data analysis. Data is split into orthogonal components, and
the genes/probes that contribute the most variance to the components are
displayed.</p>
<p>probe set</p>
<p>A probe set is a collection of probes designed to interrogate a given
sequence.</p>
<p>probe set id</p>
<p>A probe set ID is used to refer to a probe set, which looks like the
following:</p>
<p>12345_at or 12345_a_at or 12345_s_at or 12345_x_at</p>
<p>The last three characters (_at) identify the probe set strand.</p>
<p>p-value</p>
<p>The number corresponding probability that the occurrences of your
experiment and analysis did not happen by chance. P-value cutoffs are
often 0.05 or 0.01 – when the value is under the threshold, the result
is said to be statistically significant.</p>
<p>r</p>
<p>R is a language and environment for statistical computing and graphics.</p>
<p>See <a class="reference external" href="http://www.r-project.org">http://www.r-project.org</a> for details.</p>
<p>rbm data</p>
<p>Rules Based Medicine. They provide an array measurement of metabolites.</p>
<p>regression algorithms</p>
<p>Algorithms that are particularly suited for mining data sets that have
high dimensionality (many attributes), including transactional and
unstructured data.</p>
<p>rho-value</p>
<p>Also known as Spearman’s rho, the rho-value is a non-parametric measure
of statistical dependence between two variables. See: <a class="reference external" href="#SpearmanCorrelation">Spearman
Correlation</a>.</p>
<p>r-value</p>
<p>The value assigned to a correlation coefficient.</p>
<p>sample explorer</p>
<p>Sample Explorer lets you search for tissue and blood samples of interest
so that you can learn more about the samples; for example, you can look
up sample IDs and locate the study that produced the samples in the
Dataset Explorer.</p>
<p>scatter plot</p>
<p>Type of graph that uses Cartesian coordinates to display values for two
variables for a set of data.</p>
<p>search filter</p>
<p>A biomedical concept used to define search criteria in the Search tool.</p>
<p>search string</p>
<p>A sequence of biomedical concepts used to define search criteria in the
Search tool.</p>
<p>slope</p>
<p>The steepness of the line of best fit in a graph (∆y/∆x).</p>
<p>snp data</p>
<p>Single Nucleotide Polymorphism. DNA sequence data marking variation
occurring when a single nucleotide — A, T, C or G — in the genome
differs.</p>
<p>spearman correlation</p>
<p>The Spearman&#8217;s rank-order correlation is the nonparametric version of
the Pearson product-moment correlation. Spearman&#8217;s correlation
coefficient, (, also signified by rho-value) measures the strength of
association between two ranked variables.</p>
<p>statistical significance</p>
<p>Results of analyses on data that are statistically significant indicate
a confidence level that the results did not happen by chance.</p>
<p>study group</p>
<p>The subjects in a study grouped together due to some common attribute of
interest (for example, a study can have two study groups: normal and
control).</p>
<p>subset</p>
<p>A smaller grouping of participants in a study. See <a class="reference external" href="#cohort">cohort</a>.</p>
<p>survival analysis</p>
<p>Assessment of the amount of time that a person or population lives after
a particular intervention or condition.</p>
<p>t statistic</p>
<p>Ratio of the departure of an estimated parameter from its notional value
and its standard error.</p>
<p>table with fisher test</p>
<p>Examines the significance of associated categorical variables.</p>
<p>tea analyses</p>
<p>Target Enrichment Analysis (TEA) measures the enrichment of a gene
signature, gene list, or pathway in a microarray expression experiment.</p>
<p>tea p-value</p>
<p>These normalized p‑values are intermediate values in the TEA
calculation. To be considered a statistically significant analysis, an
analysis must have at least one matching biomarker with a TEA p-Value of
less than 0.05.</p>
<p>tea score</p>
<p>text-node</p>
<p>Indicated by the (<strong>abc</strong>) symbol, text nodes indicate that the data
values associated with the concept are only textual (for example, race
or gender). For more information, see <a class="reference external" href="#CategoricaVariable">Categorical
Variable</a>.</p>
<p>tissue type</p>
<p>The specific type of tissue that has been used in the experiment (for
example, breast tissue, lung tissue, etc.)</p>
<p>up-regulation</p>
<p>An analysis of a statistically significant experiment returned from a
search against a pathway is designated as <em>up-regulated</em> or
<em>down-regulated</em>.</p>
<p>x-axis</p>
<p>The horizontal axis of a two-dimensional Cartesian coordinate system.</p>
<p>y-axis</p>
<p>The vertical axis of a two-dimensional Cartesian coordinate system.</p>
</div>
</div>
</div>


          </div>
        </div>
      </div>
      <div class="sphinxsidebar" role="navigation" aria-label="main navigation">
        <div class="sphinxsidebarwrapper">
  <h3><a href="index.html">Table Of Contents</a></h3>
  <ul>
<li><a class="reference internal" href="#">**</a><ul>
<li><a class="reference internal" href="#chapter-1-getting-started-with-transmart">Chapter 1: Getting Started with tranSMART</a><ul>
<li><a class="reference internal" href="#feature-overview">Feature Overview</a><ul>
<li><a class="reference internal" href="#search-tool">Search Tool</a><ul>
<li><a class="reference internal" href="#search-results">Search Results</a></li>
</ul>
</li>
<li><a class="reference internal" href="#dataset-explorer">Dataset Explorer</a></li>
<li><a class="reference internal" href="#gene-signature-wizard">Gene Signature Wizard</a></li>
</ul>
</li>
<li><a class="reference internal" href="#logging-in">Logging In</a></li>
<li><a class="reference internal" href="#tools">Tools</a></li>
<li><a class="reference internal" href="#opening-a-particular-tool-at-login">Opening a Particular Tool at Login</a></li>
</ul>
</li>
<li><a class="reference internal" href="#chapter-2-search-tool">Chapter 2: Search Tool</a><ul>
<li><a class="reference internal" href="#search-tasks">Search Tasks</a><ul>
<li><a class="reference internal" href="#defining-a-search-filter">Defining a Search Filter</a><ul>
<li><a class="reference internal" href="#type-the-filter-name">Type the Filter Name</a></li>
<li><a class="reference internal" href="#browse-for-a-filter-name">Browse for a Filter Name</a></li>
<li><a class="reference internal" href="#use-a-saved-search-filter">Use a Saved Search Filter</a></li>
</ul>
</li>
<li><a class="reference internal" href="#building-a-search-string">Building a Search String</a><ul>
<li><a class="reference internal" href="#rules-for-building-a-search-string">Rules for Building a Search String</a></li>
<li><a class="reference internal" href="#instructions-for-building-a-search-string">Instructions for Building a Search String</a></li>
</ul>
</li>
<li><a class="reference internal" href="#saving-a-search-filter-or-search-string">Saving a Search Filter or Search String</a><ul>
<li><a class="reference internal" href="#editing-and-deleting-saved-filters">Editing and Deleting Saved Filters</a></li>
</ul>
</li>
<li><a class="reference internal" href="#working-with-search-results">Working with Search Results</a><ul>
<li><a class="reference internal" href="#clinical-trials-tab">Clinical Trials Tab</a></li>
<li><a class="reference internal" href="#mrna-analysis-tab">mRNA Analysis Tab</a><ul>
<li><a class="reference internal" href="#show-filters">Show Filters</a></li>
<li><a class="reference internal" href="#analysis-view">Analysis View</a></li>
<li><a class="reference internal" href="#study-view">Study View</a></li>
<li><a class="reference internal" href="#export-results-in-analysis-view-or-study-view">Export Results in Analysis View or Study View</a></li>
<li><a class="reference internal" href="#export-information-about-a-particular-analysis">Export Information about a Particular Analysis</a></li>
<li><a class="reference internal" href="#mouse-gene-homology-in-search-results">Mouse Gene Homology in Search Results</a></li>
<li><a class="reference internal" href="#additional-resources">Additional Resources</a></li>
</ul>
</li>
<li><a class="reference internal" href="#literature-tab">Literature Tab</a></li>
<li><a class="reference internal" href="#documents-tab">Documents Tab</a></li>
</ul>
</li>
</ul>
</li>
<li><a class="reference internal" href="#tea-analyses">TEA Analyses</a><ul>
<li><a class="reference internal" href="#tea-indicators-applied-to-individual-biomarkers">TEA Indicators Applied to Individual Biomarkers</a></li>
<li><a class="reference internal" href="#tea-indicators-applied-to-an-analysis">TEA Indicators Applied to an Analysis</a><ul>
<li><a class="reference internal" href="#what-the-tea-score-means">What the TEA Score Means</a></li>
<li><a class="reference internal" href="#what-co-anti-regulation-and-up-down-regulation-mean">What Co-/Anti-Regulation and Up-/Down-Regulation Mean</a></li>
</ul>
</li>
<li><a class="reference internal" href="#tea-indicators-applied-to-an-individual-gene">TEA Indicators Applied to an Individual Gene</a></li>
</ul>
</li>
</ul>
</li>
<li><a class="reference internal" href="#chapter-3-dataset-explorer">Chapter 3: Dataset Explorer</a><ul>
<li><a class="reference internal" href="#overview-of-the-ui">Overview of the UI</a></li>
<li><a class="reference internal" href="#using-dataset-explorer">Using Dataset Explorer</a><ul>
<li><a class="reference internal" href="#public-and-private-studies">Public and Private Studies</a></li>
<li><a class="reference internal" href="#selecting-the-study">Selecting the Study</a><ul>
<li><a class="reference internal" href="#search-terms-tab">Search Terms Tab</a><ul>
<li><a class="reference internal" href="#selecting-and-opening-a-study-in-a-search-result">Selecting and Opening a Study in a Search Result</a></li>
</ul>
</li>
<li><a class="reference internal" href="#navigate-terms-tab">Navigate Terms Tab</a></li>
<li><a class="reference internal" href="#branches-and-leaves-of-the-navigation-tree">Branches and Leaves of the Navigation Tree</a></li>
</ul>
</li>
<li><a class="reference internal" href="#populating-the-study-groups">Populating the Study Groups</a><ul>
<li><a class="reference internal" href="#selecting-criteria-for-the-study-groups">Selecting Criteria for the Study Groups</a><ul>
<li><a class="reference internal" href="#visual-aids-to-help-you-select-the-criteria">Visual Aids to Help You Select the Criteria</a></li>
</ul>
</li>
<li><a class="reference internal" href="#specifying-a-numeric-value">Specifying a Numeric Value</a><ul>
<li><a class="reference internal" href="#viewing-the-numeric-values-associated-with-a-concept">Viewing the Numeric Values Associated with a Concept</a></li>
</ul>
</li>
<li><a class="reference internal" href="#saving-comparison-definitions">Saving Comparison Definitions</a></li>
<li><a class="reference internal" href="#joining-multiple-criteria-for-a-subset-definition">Joining Multiple Criteria for a Subset Definition</a></li>
<li><a class="reference internal" href="#modifying-or-deleting-criteria">Modifying or Deleting Criteria</a></li>
</ul>
</li>
<li><a class="reference internal" href="#generating-summary-statistics">Generating Summary Statistics</a><ul>
<li><a class="reference internal" href="#significance-tests">Significance Tests</a></li>
</ul>
</li>
<li><a class="reference internal" href="#defining-points-of-comparison">Defining Points of Comparison</a><ul>
<li><a class="reference internal" href="#results-of-a-comparison">Results of a Comparison</a></li>
</ul>
</li>
<li><a class="reference internal" href="#printing-or-saving-the-contents-of-the-results-analysis-view">Printing or Saving the Contents of the Results/Analysis View</a></li>
<li><a class="reference internal" href="#copying-individual-charts-in-the-results-analysis-view">Copying Individual Charts in the Results/Analysis View</a></li>
</ul>
</li>
<li><a class="reference internal" href="#viewing-a-study">Viewing a Study</a></li>
<li><a class="reference internal" href="#exporting-dataset-explorer-findings">Exporting Dataset Explorer Findings</a></li>
<li><a class="reference internal" href="#generating-advanced-analyses-and-visualizations">Generating Advanced Analyses and Visualizations</a><ul>
<li><a class="reference internal" href="#generating-heatmaps">Generating Heatmaps</a><ul>
<li><a class="reference internal" href="#standard-heatmap">Standard Heatmap</a></li>
<li><a class="reference internal" href="#hierarchical-clustering">Hierarchical Clustering</a></li>
<li><a class="reference internal" href="#k-means-clustering">K-Means Clustering</a></li>
<li><a class="reference internal" href="#marker-selection">Marker Selection</a></li>
</ul>
</li>
<li><a class="reference internal" href="#generating-advanced-analyses">Generating Advanced Analyses</a><ul>
<li><a class="reference internal" href="#box-plot-with-anova">Box Plot with ANOVA</a></li>
<li><a class="reference internal" href="#principal-component-analysis">Principal Component Analysis</a></li>
<li><a class="reference internal" href="#scatter-plot-with-linear-regression">Scatter Plot with Linear Regression</a></li>
<li><a class="reference internal" href="#survival-analysis">Survival Analysis</a></li>
<li><a class="reference internal" href="#table-with-fisher-test-analysis">Table with Fisher Test Analysis</a></li>
</ul>
</li>
<li><a class="reference internal" href="#data-binning">Data Binning</a><ul>
<li><a class="reference internal" href="#data-binning-using-box-plot-with-anova">Data Binning Using Box Plot with ANOVA</a></li>
<li><a class="reference internal" href="#data-binning-using-survival-analysis">Data Binning Using Survival Analysis</a></li>
</ul>
</li>
<li><a class="reference internal" href="#high-dimensional-data">High Dimensional Data</a></li>
<li><a class="reference internal" href="#other-features">Other Features</a><ul>
<li><a class="reference internal" href="#save-to-pdf">Save to PDF</a></li>
<li><a class="reference internal" href="#download-raw-r-data">Download Raw R Data</a></li>
</ul>
</li>
</ul>
</li>
<li><a class="reference internal" href="#the-jobs-tab">The Jobs Tab</a><ul>
<li><a class="reference internal" href="#viewing-a-logged-job">Viewing a Logged Job</a></li>
</ul>
</li>
</ul>
</li>
<li><a class="reference internal" href="#chapter-4-gene-signatures-and-gene-lists">Chapter 4: Gene Signatures and Gene Lists</a><ul>
<li><a class="reference internal" href="#creating-a-gene-signature">Creating a Gene Signature</a><ul>
<li><a class="reference internal" href="#step-1-adding-the-genes-to-a-text-file">Step 1. Adding the Genes to a Text File</a><ul>
<li><a class="reference internal" href="#using-transmart-to-select-genes">Using tranSMART to Select Genes</a></li>
</ul>
</li>
<li><a class="reference internal" href="#step-2-creating-the-gene-signature">Step 2. Creating the Gene Signature</a><ul>
<li><a class="reference internal" href="#making-a-new-gene-signature-public">Making a New Gene Signature Public</a></li>
</ul>
</li>
</ul>
</li>
<li><a class="reference internal" href="#performing-actions-on-your-gene-signatures">Performing Actions on Your Gene Signatures</a></li>
<li><a class="reference internal" href="#performing-actions-on-other-users-signatures">Performing Actions on Other Users’ Signatures</a></li>
<li><a class="reference internal" href="#viewing-a-gene-signature-definition">Viewing a Gene Signature Definition</a></li>
</ul>
</li>
<li><a class="reference internal" href="#chapter-5-other-tasks">Chapter 5: Other Tasks</a><ul>
<li><a class="reference internal" href="#other-tasks">Other Tasks</a></li>
</ul>
</li>
<li><a class="reference internal" href="#appendix-a-how-tea-scores-are-calculated">Appendix A: How TEA Scores Are Calculated</a><ul>
<li><a class="reference internal" href="#data-inputs-to-the-tea-algorithm"><strong>Data Inputs to the TEA algorithm</strong></a></li>
<li><a class="reference internal" href="#operations">Operations</a></li>
<li><a class="reference internal" href="#result">Result</a></li>
</ul>
</li>
<li><a class="reference internal" href="#appendix-b-rules-for-loading-omicsoft-data">Appendix B: Rules for Loading OmicSoft Data</a><ul>
<li><a class="reference internal" href="#rules-for-loading-omicsoft-data">Rules for Loading OmicSoft Data</a></li>
</ul>
</li>
<li><a class="reference internal" href="#appendix-c-glossary-of-terms">Appendix C: Glossary of Terms</a><ul>
<li><a class="reference internal" href="#glossary-of-terms">Glossary of Terms</a></li>
</ul>
</li>
</ul>
</li>
</ul>
<div class="relations">
<h3>Related Topics</h3>
<ul>
  <li><a href="index.html">Documentation overview</a><ul>
      <li>Previous: <a href="index.html" title="previous chapter">Welcome to tranSMART User Guide&#8217;s documentation!</a></li>
  </ul></li>
</ul>
</div>
  <div role="note" aria-label="source link">
    <h3>This Page</h3>
    <ul class="this-page-menu">
      <li><a href="_sources/tmug.txt"
            rel="nofollow">Show Source</a></li>
    </ul>
   </div>
<div id="searchbox" style="display: none" role="search">
  <h3>Quick search</h3>
    <form class="search" action="search.html" method="get">
      <input type="text" name="q" />
      <input type="submit" value="Go" />
      <input type="hidden" name="check_keywords" value="yes" />
      <input type="hidden" name="area" value="default" />
    </form>
</div>
<script type="text/javascript">$('#searchbox').show(0);</script>
        </div>
      </div>
      <div class="clearer"></div>
    </div>
    <div class="footer">
      &copy;2016, Paul Agapow.
      
      |
      Powered by <a href="http://sphinx-doc.org/">Sphinx 1.4.1</a>
      &amp; <a href="https://github.com/bitprophet/alabaster">Alabaster 0.7.8</a>
      
      |
      <a href="_sources/tmug.txt"
          rel="nofollow">Page source</a>
    </div>

    

    
  </body>
</html>