<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">


<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    
    <title>Appendix: Glossary of Terms &mdash; tranSMART User Guide 0.1 documentation</title>
    
    <link rel="stylesheet" href="../_static/alabaster.css" type="text/css" />
    <link rel="stylesheet" href="../_static/pygments.css" type="text/css" />
    
    <script type="text/javascript">
      var DOCUMENTATION_OPTIONS = {
        URL_ROOT:    '../',
        VERSION:     '0.1',
        COLLAPSE_INDEX: false,
        FILE_SUFFIX: '.html',
        HAS_SOURCE:  true
      };
    </script>
    <script type="text/javascript" src="../_static/jquery.js"></script>
    <script type="text/javascript" src="../_static/underscore.js"></script>
    <script type="text/javascript" src="../_static/doctools.js"></script>
    <script type="text/javascript" src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
    <link rel="top" title="tranSMART User Guide 0.1 documentation" href="../index.html" />
    <link rel="next" title="Appendix: Rules for Loading OmicSoft Data" href="loading-omicsoft-data.html" />
    <link rel="prev" title="Other Tasks" href="../maincontent/other-tasks.html" />
   
  <link rel="stylesheet" href="../_static/custom.css" type="text/css" />
  
  <meta name="viewport" content="width=device-width, initial-scale=0.9, maximum-scale=0.9" />

  </head>
  <body role="document">
  

    <div class="document">
      <div class="documentwrapper">
        <div class="bodywrapper">
          <div class="body" role="main">
            
  <div class="section" id="appendix-glossary-of-terms">
<h1>Appendix: Glossary of Terms<a class="headerlink" href="#appendix-glossary-of-terms" title="Permalink to this headline">¶</a></h1>
<dl class="docutils">
<dt>aggregate probes</dt>
<dd>Used in Dataset Explorer, the Aggregate Probes checkbox allows you to
group probes used in high-dimensional data samples to form a total
quantity that analyses will be performed on.</dd>
<dt>analysis of variance (anova)</dt>
<dd>Analysis of Variance (ANOVA) is a statistical method used in Dataset
Explorer to make concurrent comparisons between two or more means in a
box plot.</dd>
<dt>analysis view</dt>
<dd>Used in the Search tool, the Analysis View option displays the
statistically significant analyses from your search filter(s).</dd>
<dt>anti-regulation</dt>
<dd>An analysis of a statistically significant experiment returned from a
search against a gene signature or list is designated as <em>co-regulated</em>
or <em>anti-regulated</em>.</dd>
<dt>array data</dt>
<dd>See: <a class="reference external" href="#Microarray">Microarray</a></dd>
<dt>arrayexpress</dt>
<dd>Database of gene expression and other microarray data at the European
Bioinformatics Institute (EBI). See <a class="reference external" href="http://www.ebi.ac.uk/arrayexpress">http://www.ebi.ac.uk/arrayexpress</a> for details.</dd>
<dt>binomial distribution</dt>
<dd>Graph that displays the discrete probability distribution of obtaining
<em>n</em> successes out of N Bernoulli trials. See <a class="reference external" href="http://mathworld.wolfram.com/BinomialDistribution.html">http://mathworld.wolfram.com/BinomialDistribution.html</a> for details.</dd>
<dt>biomarker</dt>
<dd>Short for Biological Marker, a biomarker is a key molecular or cellular
event that links a specific environmental exposure to a health outcome.</dd>
<dt>box plot</dt>
<dd>Also known as a Box and Whisker Plot, a box plot is a histogram-like
method of displaying data. Box plots are useful when conveying location
and variation information in datasets.</dd>
<dt>categorical variable</dt>
<dd>Also known as a nominal value, a categorical variable is one that has
two or more categories, but with no intrinsic ordering to the
categories. An example of a categorical value is hair color – there is
no way to order these variables from highest to lowest.</dd>
<dt>censoring value</dt>
<dd>Used in Survival Analyses. The Censoring Value specifies which patients
had the event whose time is being measured. For example, if the Time
variable selected is Overall Survival Time (Years), an appropriate
censoring variable is Patient Death.</dd>
</dl>
<dl class="docutils">
<dt>chi squared</dt>
<dd><p class="first">Let the probabilities of various classes in a distribution be
<span class="math">\(p_1, p_2 ... p_k\)</span>, with observed frequencies
<span class="math">\(m_1, m_2 ... m_k\)</span>. The quantity:</p>
<p><a href="#id1"><span class="problematic" id="id2">|image217|</span></a></p>
<p class="last">is therefore a measure of the deviation of a sample from expectation,
where <span class="math">\(N_is\)</span> is the sample size.</p>
</dd>
<dt>cohort</dt>
<dd>A group of subjects who have shared a specific event or characteristic.</dd>
<dt>continuous variable</dt>
<dd>Continuous variables have an infinite number of values between two
points. For example, age or temperature.</dd>
<dt>co-regulation</dt>
<dd>An analysis of a statistically significant experiment returned from a
search against a gene signature or list is designated as <em>co-regulated</em>
or <em>anti-regulated</em>.</dd>
<dt>correlation analysis</dt>
<dd>A type of Regression Analysis, correlation analysis measures the
correlation coefficient – the linear association between two variables.
Values of the correlation coefficient are always between -1 and +1. A
correlation coefficient of +1 indicates that two variables are perfectly
related in a positive linear sense, while a correlation coefficient of
-1 indicates that two variables are perfectly related in a negative
linear sense.</dd>
<dt>cox coefficient</dt>
<dd>The Cox coefficient refers to the coefficients in a Cox regression model
(also known as the proportional hazards model for survival-time). The
analysis investigates the effects of one or more variables upon the time
a specified event takes to happen. The cox coefficient relates to a
hazard; a positive coefficient indicates a worse prognosis, while a
negative coefficient indicates a protective effect of the variable.</dd>
<dt>data binning</dt>
<dd>Defers to a data pre-processing technique used to reduce observation
errors and to allow continuous variables to become categorical. Clusters
of data are replaced by a value representative of that cluster (often
but not necessarily, the central value).</dd>
<dt>data warehouse</dt>
<dd>A database used for reporting and analysis.</dd>
<dt>dataset</dt>
<dd>Collection of data, most commonly presented in a tabular form where each
column represents a specific variable, and each row represents a value
for that variable.</dd>
<dt>dataset explorer</dt>
<dd>Dataset Explorer lets you compare data generated for test subjects in
two different study groups, based on criteria and points of comparison
that you specify. Dataset Explorer is useful to help you test a
hypothesis that involves the criteria and points of comparison that you
select.</dd>
<dt>dependent variable</dt>
<dd>In an experiment, the dependent variable is the response that is
measured.</dd>
<dt>differential modulation</dt>
<dd><p class="first">down-regulation</p>
<p class="last">An analysis of a statistically significant experiment returned from a
search against a pathway is designated as <em>up-regulated</em> or
<em>down-regulated</em>.</p>
</dd>
<dt>entrez gene</dt>
<dd>Reference sequences for a wide range of species. For details, see
<a class="reference external" href="http://www.ncbi.nlm.nih.gov/gene/">http://www.ncbi.nlm.nih.gov/gene/</a>.</dd>
<dt>entrez global</dt>
<dd>Federated search engine that allows users to search various health
sciences databases at the National Center for Biotechnology Information
(NCBI) website. See <a class="reference external" href="http://www.ncbi.nlm.nih.gov/gqueryfor">http://www.ncbi.nlm.nih.gov/gqueryfor</a> details.</dd>
<dt>fold change ratio</dt>
<dd>A number describing how much a quantity changes going from an initial to
a final value. An initial value of 50 and a final value of 100
corresponds to a fold change of 2 (a two-fold increase).</dd>
<dt>gene</dt>
<dd>Stretches of DNA and RNA that code for a polypeptide or for an RNA chain
– contains hereditary molecular information.</dd>
<dt>gene chip</dt>
<dd>See: <a class="reference external" href="#Microarray">Microarray</a></dd>
<dt>gene expression</dt>
<dd>The flow of genetic information from gene to protein; the process, or
the regulation of the process, by which the effects of a gene are
manifested; the manifestation of a heritable trait in an individual
carrying the gene or genes that determine it.</dd>
<dt>gene expression omnibus</dt>
<dd>GEO is an international public repository that archives and freely
distributes microarray, next-generation sequencing, and other forms of
high-throughput functional genomics data submitted by the research
community. For more information, see <a class="reference external" href="http://www.ncbi.nlm.nih.gov/geo">http://www.ncbi.nlm.nih.gov/geo</a>.</dd>
<dt>gene set enrichment analysis (gsea)</dt>
<dd>Computational method that determines whether an a priori defined set of
genes shows statistically significant, concordant differences between
two biological states (for example, phenotypes). See <a class="reference external" href="http://www.broadinstitute.org/gsea/index.jsp">http://www.broadinstitute.org/gsea/index.jsp</a> for details.</dd>
<dt>gene signature</dt>
<dd>A group of genes whose combined expression pattern is uniquely
characteristic of a medical condition or other clinical outcome of
interest.</dd>
<dt>gene symbol</dt>
<dd>A unique abbreviation of a gene name consisting of italicized uppercase
Latin letters and Arabic numbers. We use Entrez as the full list of
genes (related to but not identical to HUGO). See <a class="reference external" href="http://www.genenames.org/">http://www.genenames.org/</a> for details.</dd>
<dt>genecards</dt>
<dd>Database that offers information about human genes (and mouse
homologues). See <a class="reference external" href="http://www.genecards.org">http://www.genecards.org</a> for details.</dd>
<dt>google scholar</dt>
<dd>Google application that provides a search of scholarly literature across
multiple disciplines and sources.See <a class="reference external" href="http://scholar.google.com">http://scholar.google.com</a> for details.</dd>
<dt>gpl platform</dt>
<dd>A Platform record is composed of a summary description of the array or
sequencer and, for array-based Platforms, a data table defining the
array template. Each Platform record is assigned a unique and stable GEO
accession number (GPLxxx). A Platform may reference many Samples that
have been submitted by multiple submitters.</dd>
<dt>heatmap</dt>
<dd>Display of differential expression. Individual values contained in the
matrix are represented by colors.</dd>
<dt>hierarchical clustering</dt>
<dd>Hierarchical clustering is a type of clustering analysis whose goal is
to organize data so that the objects in the same cluster are more
similar to each other than to those in other clusters.</dd>
<dt>high dimensional data</dt>
<dd>Datasets where the intersection of a subject and measurement is
comprised of hundreds or thousands of points. For example, in a low
dimensional data measurement such as height the intersection of subject
and measurement is one number (ex. 180 cm) whereas in a high dimensional
data measurement such as gene expression in a lymph node the measurement
is 50,000 individual probe expression values.</dd>
<dt>histogram</dt>
<dd>A visual representation of the distribution of data values within a
dataset.</dd>
<dt>homology</dt>
<dd>The basis for comparative biology – where organs/structures from one
organism are compared to a similar organ/structure in a different
organism.</dd>
<dt>in vitro study</dt>
<dd>Those that are conducted using components of an organism that have been
isolated from their usual biological surroundings.</dd>
<dt>in vivo studies</dt>
<dd>Experimentation using a whole, living organism.</dd>
<dt>independent variable</dt>
<dd>In an experiment, the independent variable is the variable that is
manipulated.</dd>
<dt>job</dt>
<dd>In tranSMART, a job refers to a command you have given Dataset Explorer
to process or export data. Jobs and job-related events can be found
within the <strong>Jobs</strong> tab in Dataset Explorer.</dd>
<dt>kendall correlation</dt>
<dd>Kendall&#8217;s rank correlation provides a distribution-free test of
independence and a measure of the strength of dependence between two
variables.</dd>
<dt>k-means clustering</dt>
<dd>The K-Means clustering heatmap clusters genes and/or samples into a
specified number of clusters. The result is <em>k</em> clusters, each centered
around a randomly-selected data point.</dd>
<dt>line graph</dt>
<dd>Line graphs illustrate the temporal relationship between two major
variables.</dd>
<dt>marker selection</dt>
<dd>Marker Selection is a display of the top differentially expressed genes
between two specified cohorts.</dd>
<dt>mesh ontology</dt>
<dd>MeSH is the National Library of Medicine&#8217;s controlled vocabulary
thesaurus. It consists of sets of terms naming descriptors in a
hierarchical structure that permits searching at various levels of
specificity.</dd>
<dt>microarray</dt>
<dd>A two-dimensional array on a chip or solid surface that assays large
amounts of DNA material.</dd>
<dt>mrna analysis</dt>
<dd>Assays that quantify the expression levels of all mRNA molecules in an
experiment.</dd>
<dt>navigation tree</dt>
<dd>The Window’s Explorer-like, hierarchical representation of study data
that has been loaded into Dataset Explorer.</dd>
<dt>ncbi</dt>
<dd>The National Center for Biotechnology Information. See <a class="reference external" href="http://">http://</a><a class="reference external" href="http://www.ncbi.nlm.nih.gov/">www.ncbi.nlm.nih.gov/</a>
for details.</dd>
<dt>numeric-node</dt>
<dd>Used in Dataset Explorer, numeric-nodes are indicated by the (<strong>123</strong>)
symbol, numeric nodes indicate that the data values associated with the
concept are only numeric (for example, age values, date values, etc.).
For more information, see <a class="reference external" href="#ContinuousVariable">Continuous Variable</a>.</dd>
<dt>ontology</dt>
<dd>A hierarchical description of the concepts and relationships that can
exist for an agent or a community of agents.</dd>
<dt>orthogonal component</dt>
<dd>When performing statistical analysis, independent variables that affect
a particular dependent variable are said to be orthogonal if they are
uncorrelated, since the covariance forms an inner product.</dd>
<dt>pathology</dt>
<dd>The study of diagnosis and disease.</dd>
<dt>pathway</dt>
<dd>A group of genes interacting to form an aggregate biological function.</dd>
<dt>pearson correlation</dt>
<dd>Obtained by dividing the covariance of the two variables by the product
of their standard deviations.</dd>
<dt>principal component analysis</dt>
<dd>A Principal Component Analysis (PCA) is commonly used as a tool in
exploratory data analysis. Data is split into orthogonal components, and
the genes/probes that contribute the most variance to the components are
displayed.</dd>
<dt>probe set</dt>
<dd>A probe set is a collection of probes designed to interrogate a given
sequence.</dd>
<dt>probe set id</dt>
<dd>A probe set ID is used to refer to a probe set, which looks like the
following: 12345_at or 12345_a_at or 12345_s_at or 12345_x_at
The last three characters (_at) identify the probe set strand.</dd>
<dt>p-value</dt>
<dd>The number corresponding probability that the occurrences of your
experiment and analysis did not happen by chance. P-value cutoffs are
often 0.05 or 0.01 – when the value is under the threshold, the result
is said to be statistically significant.</dd>
<dt>r</dt>
<dd>R is a language and environment for statistical computing and graphics. See <a class="reference external" href="http://www.r-project.org">http://www.r-project.org</a> for details.</dd>
<dt>rbm data</dt>
<dd>Rules Based Medicine. They provide an array measurement of metabolites.</dd>
<dt>regression algorithms</dt>
<dd>Algorithms that are particularly suited for mining data sets that have
high dimensionality (many attributes), including transactional and
unstructured data.</dd>
<dt>rho-value</dt>
<dd>Also known as Spearman’s rho, the rho-value is a non-parametric measure
of statistical dependence between two variables. See: <a class="reference external" href="#SpearmanCorrelation">Spearman
Correlation</a>.</dd>
<dt>r-value</dt>
<dd>The value assigned to a correlation coefficient.</dd>
<dt>sample explorer</dt>
<dd>Sample Explorer lets you search for tissue and blood samples of interest
so that you can learn more about the samples; for example, you can look
up sample IDs and locate the study that produced the samples in the
Dataset Explorer.</dd>
<dt>scatter plot</dt>
<dd>Type of graph that uses Cartesian coordinates to display values for two
variables for a set of data.</dd>
<dt>search filter</dt>
<dd>A biomedical concept used to define search criteria in the Search tool.</dd>
<dt>search string</dt>
<dd>A sequence of biomedical concepts used to define search criteria in the
Search tool.</dd>
<dt>slope</dt>
<dd>The steepness of the line of best fit in a graph (∆y/∆x).</dd>
<dt>snp data</dt>
<dd>Single Nucleotide Polymorphism. DNA sequence data marking variation
occurring when a single nucleotide — A, T, C or G — in the genome
differs.</dd>
<dt>spearman correlation</dt>
<dd>The Spearman&#8217;s rank-order correlation is the nonparametric version of
the Pearson product-moment correlation. Spearman&#8217;s correlation
coefficient, (, also signified by rho-value) measures the strength of
association between two ranked variables.</dd>
<dt>statistical significance</dt>
<dd>Results of analyses on data that are statistically significant indicate
a confidence level that the results did not happen by chance.</dd>
<dt>study group</dt>
<dd>The subjects in a study grouped together due to some common attribute of
interest (for example, a study can have two study groups: normal and
control).</dd>
<dt>subset</dt>
<dd>A smaller grouping of participants in a study. See <a class="reference external" href="#cohort">cohort</a>.</dd>
<dt>survival analysis</dt>
<dd>Assessment of the amount of time that a person or population lives after
a particular intervention or condition.</dd>
<dt>t statistic</dt>
<dd>Ratio of the departure of an estimated parameter from its notional value
and its standard error.</dd>
<dt>table with fisher test</dt>
<dd>Examines the significance of associated categorical variables.</dd>
<dt>tea analyses</dt>
<dd>Target Enrichment Analysis (TEA) measures the enrichment of a gene
signature, gene list, or pathway in a microarray expression experiment.</dd>
<dt>tea p-value</dt>
<dd>These normalized p‑values are intermediate values in the TEA
calculation. To be considered a statistically significant analysis, an
analysis must have at least one matching biomarker with a TEA p-Value of
less than 0.05.</dd>
<dt>text-node</dt>
<dd>Indicated by the (<strong>abc</strong>) symbol, text nodes indicate that the data
values associated with the concept are only textual (for example, race
or gender). For more information, see <a class="reference external" href="#CategoricaVariable">Categorical
Variable</a>.</dd>
<dt>tissue type</dt>
<dd>The specific type of tissue that has been used in the experiment (for
example, breast tissue, lung tissue, etc.)</dd>
<dt>up-regulation</dt>
<dd>An analysis of a statistically significant experiment returned from a
search against a pathway is designated as <em>up-regulated</em> or
<em>down-regulated</em>.</dd>
<dt>x-axis</dt>
<dd>The horizontal axis of a two-dimensional Cartesian coordinate system.</dd>
<dt>y-axis</dt>
<dd>The vertical axis of a two-dimensional Cartesian coordinate system.</dd>
</dl>
</div>


          </div>
        </div>
      </div>
      <div class="sphinxsidebar" role="navigation" aria-label="main navigation">
        <div class="sphinxsidebarwrapper"><div class="relations">
<h3>Related Topics</h3>
<ul>
  <li><a href="../index.html">Documentation overview</a><ul>
      <li>Previous: <a href="../maincontent/other-tasks.html" title="previous chapter">Other Tasks</a></li>
      <li>Next: <a href="loading-omicsoft-data.html" title="next chapter">Appendix: Rules for Loading OmicSoft Data</a></li>
  </ul></li>
</ul>
</div>
  <div role="note" aria-label="source link">
    <h3>This Page</h3>
    <ul class="this-page-menu">
      <li><a href="../_sources/appendices/glossary.txt"
            rel="nofollow">Show Source</a></li>
    </ul>
   </div>
<div id="searchbox" style="display: none" role="search">
  <h3>Quick search</h3>
    <form class="search" action="../search.html" method="get">
      <input type="text" name="q" />
      <input type="submit" value="Go" />
      <input type="hidden" name="check_keywords" value="yes" />
      <input type="hidden" name="area" value="default" />
    </form>
</div>
<script type="text/javascript">$('#searchbox').show(0);</script>
        </div>
      </div>
      <div class="clearer"></div>
    </div>
    <div class="footer">
      &copy;2016, Paul Agapow.
      
      |
      Powered by <a href="http://sphinx-doc.org/">Sphinx 1.4.1</a>
      &amp; <a href="https://github.com/bitprophet/alabaster">Alabaster 0.7.8</a>
      
      |
      <a href="../_sources/appendices/glossary.txt"
          rel="nofollow">Page source</a>
    </div>

    

    
  </body>
</html>